---
layout: indexed_content
title: '27th ICM: May 20-22 2014, Mishima, Japan'
start_date: 2014-05-10
end_date: 2014-05-22
place: Mishima, Japan
lang: en
category: insdc_meeting
related_pages:
  - title: INSDC
    url: /about/insdc-e.html
  - title: About DDBJ Center  Major Activities
    url: /about/index-e.html#activity
---

## 2014  <a name="2014"></a>

## 27th ICM: May 20-22 2014, Mishima, Japan

[International Nucleotide Sequence Database Collaboration
(INSDC)](https://www.insdc.org/), consisted of DDBJ,
[EBI](https://www.ebi.ac.uk/) and [NCBI](https://www.ncbi.nlm.nih.gov/),
hold the international collaborators meeting every year.  
In 2014, the meeting was [held at DDBJ](/news/en/2014-05-20-e.html), 20-22
May, to discuss practical matters to maintain and update nucleotide
sequence data archives; [DDBJ](/index-e.html),
[ENA](https://www.ebi.ac.uk/ena/),
[GenBank](https://www.ncbi.nlm.nih.gov/genbank/index.html), 
Sequence Read Archive (SRA) and Trace Archive.  
The outcomes of the meeting are summarized below.

### The Items; Discussed and To Be Studied

BioSample database  
:  The BioSample database contains descriptions of biological source materials used in experimental assays.  
    DDBJ has also started accepting [BioSample](/biosample/index-e.html) submissions in early 2014.  
     Following the meetings on [2012](/activities/insdc_meeting/2012-e.html) and [2013](/activities/insdc_meeting/2013-e.html), 
    we discussed action items to collect and to share BioSample data at INSDC.

Issues about lots of genome submissions, [WGS](/ddbj/wgs-e.html) and so on  
Data Exchange  
:  We discussed new data formats and an effective way to exchange data each other.

[Assembly](https://www.ncbi.nlm.nih.gov/assembly/) (Genome Collection)  
:  Following the meeting on [2012](/activities/insdc_meeting/2012-e.html), we continues collaboration with Assembly activities to collect genome sequences.

About /[protein_id](/ddbj/qualifiers-e.html#protein_id)  
:  Currently, we have so many submissions of similar genomes derived from multiple strains of a species etc. In those cases, there are many orthologs in CDS features, so, many /protein\_ids seem to be redundantly assigned for them. We discussed possibilities about new ways of /protein\_id assignment.

### Changes in SRA XML schema

SRA XML schema version 2.0 continues to be discussed for refactoring SRA metadata with [BioProject](/bioproject/index-e.html) and [BioSample](/biosample/index-e.html) data.

### Forthcoming changes in The DDBJ/EMBL/GenBank Feature Table: Definition  <a name="2014-ft"></a>

The following items will be applied from October 2014 with the next revision of [Feature Table Definition](/ddbj/feature-table-e.html), if not otherwise specified.

-   A new feature, [regulatory](/ddbj/features-e.html#regulatory), and a
    new qualifier,
    /[regulatory_class](/ddbj/qualifiers-e.html#regulatory_class), will 
    be available from December 2014.  
    Following features will be replaced by the new feature; -35_signal, -10_signal, CAAT_signal, GC_signal, 
    TATA_signal, polyA_signal, attenuator, terminator, promoter, enhancer, RBS.
-   A base code for 'dihydrouridine' of [Modified Base Abbreviations](/ddbj/code-e.html#nucleotide-2) and
    /[mod_base](/ddbj/qualifiers-e.html#mod_base) qualifier, "d" will be replaced by "dhu".  
-   To be sure using 
    [prim_transcript](/ddbj/feature-table-e.html#prim_transcript) and 
    [precursor_RNA](/ddbj/feature-table-e.html#precursor_RNA) for RNA 
    transcripts other than mRNA, their definitions will be updated appropriately. 
-   [Since 2013](/activities/insdc_meeting/2013-e.html), a new qualifier, 
    /type_material, is  considered to specify type strains, type specimens and so on.  
    It is not decided in details and applicable period of the qualifier.  
