---
layout: indexed_content
title: '26th: May 21-23 2013, Hinxton, UK'
start_date: 2013-05-21
end_date: 2013-05-23
place: Hinxton, UK
lang: en
category: icm
related_pages:
  - title: DDBJ Publications Archive
    url: /activities/archives-e.html
  - title: DDBJ staff papers (2009-)
    url: /activities/index-e.html
  - title: DDBJ staff presentations (2009-)
    url: /activities/index-e.html
  - title: DDBJ staff presentations (2001-2008)
    url: /activities/past-presentations-e.html
  - title: DDBJ staff papers（1996-2008）
    url: /activities/past-papers-e.html
  - title: DDBJ Annual Reports
    url: /activities/index-e.html
  - title: Patent column from DDBJ
    url: /activities/patent-e.html
  - title: DDBJ Mail Magazine
    url: /subscribe-ddbj-e.html
---

## 2013 <a name="2013"></a>

### 26th: May 21-23 2013, Hinxton, UK

[International Nucleotide Sequence Database Collaboration
(INSDC)](http://www.insdc.org/), consisted of DDBJ,
[EBI](https://www.ebi.ac.uk/) and [NCBI](https://www.ncbi.nlm.nih.gov/),
hold the international collaborators meeting every year.  
In 2013, the meeting was held at EBI in UK, 21-23 May, to discuss
practical matters to maintain and update nucleotide sequence data
archives; [DDBJ](/index-e.html),
[EMBL-Bank](https://www.ebi.ac.uk/embl/),
[GenBank](https://www.ncbi.nlm.nih.gov/genbank/index.html), Sequence
Read Archive (SRA) and Trace Archive.  
The outcomes of the meeting are summarized below.

#### The Items; Discussed and To Be Studied

BioSample database  
:  The BioSample database contains descriptions of biological source materials used in experimental assays. The purpose of the BioSample database is to provide unified storage and access to information about biological samples. These samples may have investigation information stored in other databases (e.g. nucleotide sequence, expression).  
  [Following the meeting on 2012](/activities/icm/2012-e), we discussed action items to collect and to share BioSample data at INSDC.  
  DDBJ started to accept [BioSample](/biosample/index-e.html) submissions in 2014.

Strain level taxonomy ID assignment for microorganism genome submission  
:  All [organism names](/ddbj/organism-e.html) that are represented in the sequence data of INSDC are registered to the taxonomy database.  
[Since 2009](/activities/icm/2009-e), taxonomy database has considered to terminate assignment of strain level taxonomy ID for microorganism genomes.  
From 2014, we will provide BioSample data instead of strain level taxonomy ID, and will terminate to assign strain level taxonomy ID for microorganism genomes   
We reported in detail about this issue in [an academic paper](http://standardsingenomics.org/index.php/sigen/article/view/sigs.4851102)

#### Changes related to INSDC submission

Relaxation rules to accept [WGS](/ddbj/wgs-e.html) and scaffold data  
:  Heretofore, INSDC accepted sequences of overlapping reads (not including any sequencing gaps) as [WGS](/ddbj/wgs-e.html) entries and accepted [AGP format](https://www.ncbi.nlm.nih.gov/projects/genome/assembly/agp/AGP_Specification.shtml) to indicate scaffolds (including sequencing gaps) as [CON](/ddbj/con-e.html) entries.  
  Recently, the policy seemed to be out of date, because some of software tools for genome assemble support to output scaffold only in sequences, not in AGP format. So, we decided to accept sequences of scaffolds with gap n's.   
  See also [INSDC standards for genome assembly submission](/ddbj/assembly-e.html)

Accepting submission of scaffolded [TSA](/ddbj/tsa-e.html) data  
:  Recently, paired-end sequencing is fairly common not only for genomes but also for transcriptomes and some of the RNAseq assembly software packages have added scaffolding. So, we started accepting these scaffolded assemblies as TSA records with [assembly\_gap](/ddbj/features-e.html#assembly_gap) features and /[linkage\_evidence](/ddbj/qualifiers-e.html#linkage_evidence)="paired-ends" or some.

Update guidelines for [TPA](/ddbj/tpa-e.html) submission  
:  Guidelines for TPA submission will be updated to cope with the current status of data submission.See also
[TPA Submission Guidelines](/ddbj/tpa-table-e.html).   
Major modification points are follows;
  1. TPA is renamed from "Third Party Annotation" to "Third Party Data".
  2. Specify to accept not only annotation but also assemble for TPA.
  3. A new subcategory, "TPA:specialist\_db" will be added in TPA to accept submissions from expert databases.

#### Changes in SRA XML schema

[SRA XML schema version 1.5](https://www.ncbi.nlm.nih.gov/viewvc/v1/trunk/sra/doc/SRA_1-5/) has been applied. The modification points are elimination and consolidation of redundant description items.

SRA XML schema version 2.0 continues to be discussed for refactoring SRA
metadata with [BioProject](/bioproject/index-e.html) and
[BioSample](/biosample/index-e.html) data.

We decided to allow SRA accessions to have variable lengths after 6
digits have been used up, e.g. SRR1000000 would follow SRR999999.

#### Forthcoming changes in the DDBJ/EMBL/GenBank Feature Table: Definition <a name="2013-ft"></a>

The following items will be applied from October 2013 with the revision
of [Feature Table Definition](/ddbj/feature-table-e.html), if not otherwise
specified.

-   It is reconfirmed that [5'UTR](/ddbj/features-e.html#5UTR) and
    [3'UTR](/ddbj/features.html#3UTR) features can be used for RNA viral
    genome. Their definitions will be updated appropriately
-   It is reconfirmed that [5'UTR](/ddbj/features-e.html#5UTR) and
    [3'UTR](/ddbj/features.html#3UTR) features can be used for RNA viral
    genome.  
    <span class="red">It will be applied from December 2013</span>
    1. Time (with time zone): in [the ISO standard
        format](https://www.w3.org/TR/NOTE-datetime)  
        i.e. "2007-04-05T14:30Z
    2. Range: in the format delimited by "/"  
        i.e. "2007-03-01T13:00Z/2008-05-11T15:30Z""
-   A new value, "lncRNA", will be legal for
    /[ncRNA\_class](/ddbj/qualifiers-e.html#ncRNA_class) qualifier.
-   The qualifier,
    /[estimated\_length](/ddbj/qualifiers-e.html#estimated_length), will
    be modified to allow different lengths for unknown length gaps.
-   A new qualifier, /type\_material, will be considered to specify type
    strains, type specimens and so on.  
    It is not decided in details and applicable period of the qualifier.
