---
layout: indexed_content
title: '31st ICM: May 15-17 2018, Bethesda, USA'
start_date: 2018-05-15
end_date: 2018-05-17
place: Bethesda, USA
lang: en
category: icm
related_pages:
  - title: DDBJ Publications Archive
    url: /activities/archives-e.html
  - title: DDBJ staff papers (2009-)
    url: /activities/index-e.html
  - title: DDBJ staff presentations (2009-)
    url: /activities/index-e.html
  - title: DDBJ staff presentations (2001-2008)
    url: /activities/past-presentations-e.html
  - title: DDBJ staff papers（1996-2008）
    url: /activities/past-papers-e.html
  - title: DDBJ Annual Reports
    url: /activities/index-e.html
  - title: Patent column from DDBJ
    url: /activities/patent-e.html
  - title: DDBJ Mail Magazine
    url: /subscribe-ddbj-e.html
---

## 2018  <a name="2018"></a>

### 31st ICM: May 15-17 2018, Bethesda, USA

[International Nucleotide Sequence Database Collaboration
(INSDC)](http://www.insdc.org/), consisted of DDBJ Center,
[EBI](https://www.ebi.ac.uk/) and [NCBI](https://www.ncbi.nlm.nih.gov/),
hold the international collaborators meeting every year.  
In 2018, the meeting was held at NCBI, 15-17 May, to discuss practical
matters to maintain and update nucleotide sequence data archives;
[DDBJ](/index-e.html)、[ENA](https://www.ebi.ac.uk/ena/)、[GenBank](https://www.ncbi.nlm.nih.gov/genbank/index.html)
and Sequence Read Archive (SRA).  
The outcomes of the meeting are summarized below.

#### The Items; Discussed and To Be Studied

Format expansions for accession numbers and for /protein\_id.  
: By the end of 2018, INSDC will expand the accession number formats as
follows;
  -   For bulk sequence data, [WGS](/ddbj/wgs-e.html),
      [TSA](/ddbj/tsa-e.html) and [TLS](/ddbj/tls-e.html), currently using
      a "4+2+6/7/8" format ( four-letter prefix and two digits for set
      version followed by six digits), the new format will be a six-letter
      prefix and a two-digit set version number followed by 7, 8, or 9
      digits ("6+2+7/8/9" format); for example, AAAAAA020000001.
  -   For conventional sequence data, currently using a "2+6" format
      (two-letter prefix followed by six digits) and a "1+5" format, the
      new format will be two-letter prefix followed by eight digits ("2+8"
      format).
  -   For /protein\_id, currently using a "3+5" format (three-letter
      prefix followed by five digits), the new format will be three-letter
      prefix followed by seven digits ("3+7" format).

International Protein Nomenclature Guidelines are introduced.  
: INSDC agreed to recommend [the international protein nomenclature guidelines](https://www.ncbi.nlm.nih.gov/genome/doc/internatprot_nomenguide/) to submitters of DDBJ, ENA and GenBank. The guidelines were revised and reorganized from previous guidelines of [UniProt](https://insideuniprot.blogspot.com/2018/07/new-guidelines-to-help-with-protein.html)t and NCBI by NCBI, EBI, the Protein Information Resource and the Swiss Institute for Bioinformatics. The guidelines are exclusively focused on nomenclature. The future changes of guidelines can be tracked in [GitHub](https://github.com/ncbi/International-Protein-Nomenclature-Guidelines).See also [NCBI Insights](https://ncbiinsights.ncbi.nlm.nih.gov/2018/08/01/new-international-protein-naming-guidelines-promote-clarity-consistency/) and [Inside UniProt](https://insideuniprot.blogspot.com/2018/07/new-guidelines-to-help-with-protein.html).

Increasing metagenomic data  
: Assembling/analyzing methods for metagenomic approaches are progressed and data submissions of them are increased. We discussed to accommodate with metagenomic data. Both BioSample and flatfile will be expanded to deal with minimum information about a single amplified genome (MISAG)
and a metagenome-assembled genome (MIMAG) as requested by the Genomic Standards Consortium.  
See [Nat. Biotechnol.
35:725-731 (2017)](https://www.nature.com/articles/nbt.3893) about MIMAG and MISAG in detail.

#### Forthcoming changes in The DDBJ/ENA/GenBank Feature Table: Definition  <a name="2018-ft"></a>

The following items will be applied after October 2018 with the next revision of [Feature Table Definition](/ddbj/feature-table-e.html), if not otherwise specified.

-   A new qualifier, [/metagenome\_source](/ddbj/qualifiers-e.html#metagenome_source), will be available for [source](/ddbj/features-e.html#source) feature.   In cases of metagenomic data, the /metagenome\_source is mandatory when [/organism](/ddbj/qualifiers-e.html#organism) does not show ”xxx metagenome” or the like. The /metagenome\_source must contain [the word "metagenome" and must exist in the taxonomy database](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Undef&id=408169).   /metagenome\_source requires [/environmental\_sample](/ddbj/qualifiers-e.html#environmental_sample) qualifiers.
-   Definition of [/EC\_number](/ddbj/qualifiers-e.html#EC_number) qualifier qualifier will be slightly changed.

#### Data Access Policy

For COP14 and the Nagoya Protocol  
:  DDBJ reported the results of the Conference of the Parties Convention on Biological Diversity 14 (COP14) for their opinion on whether “digital sequence information” should be included in the Nagoya Protocol. [The Science Council of Japan](http://www.scj.go.jp/ja/info/kohyo/pdf/kohyo-24-t258e.pdf) and [the International Chamber of Commerce](https://cdn.iccwbo.org/content/uploads/sites/3/2017/10/ICC-digital-sequence-information-and-the-nagoya-protocol.pdf) do not favor this action.  
  We, INSDC, have [an existing policy](http://www.insdc.org/policy.html) of free and unrestricted access to all of the data records.

For GDPR  
:  In May 2018, the European General Data Protection Regulations (GDPR) have come into force. ENA presented the work and the changes that arise from these with respect to ENA operations.

NGS Quality Scores  
:  Most of SRA storage is taken up by quality scores, however, the sequencing technologies are going to natural progression and precisions of base calling do not impact to assemblies or analyses. Besides, to save storage of users during analyses and to transfer SRA data from submitters and to users more easier, we consider that it is preferable to remove quality scores from SRA data. We would like to discuss about how to accept future NGS data with major research communities including vendors, tool maintainers and users.
