---
layout: indexed_content
title: '32nd ICM: May 15-17 2019, Hinxton, UK'
start_date: 2019-05-15
end_date: 2019-05-17
place: Hinxton, UK
lang: en
category: icm
related_pages:
  - title: DDBJ Publications Archive
    url: /activities/pub-archives-e.html
  - title: DDBJ staff papers (2009-)
    url: /activities/index-e.html
  - title: DDBJ staff presentations (2009-)
    url: /activities/index-e.html
  - title: DDBJ staff presentations (2001-2008)
    url: /activities/past-presentations-e.html
  - title: DDBJ staff papers（1996-2008）
    url: /activities/past-papers-e.html
  - title: DDBJ Annual Reports
    url: /activities/index-e.html
  - title: Patent column from DDBJ
    url: /activities/patent-e.html
  - title: DDBJ Mail Magazine
    url: /activities/subscribe-e.html
---

## 2019 <a name="2019"></a>

### 32nd ICM: May 15-17 2019, Hinxton, UK

[International Nucleotide Sequence Database Collaboration
(INSDC)](http://www.insdc.org/), consisted of DDBJ Center,
[EBI](https://www.ebi.ac.uk/) and [NCBI](https://www.ncbi.nlm.nih.gov/),
hold the international collaborators meeting every year.  
In 2019, the meeting was held at EBI, 15-17 May, to discuss practical
matters to maintain and update nucleotide sequence data archives;
[DDBJ](/index-e.html)、[ENA](https://www.ebi.ac.uk/ena/)、[GenBank](https://www.ncbi.nlm.nih.gov/genbank/index.html)
and Sequence Read Archive (SRA).  
The outcomes of the meeting are summarized below.

#### The Items Discussed and To Be Studied

Classification of metagenome data  
:  The methods to assemble and to analyse metagenomic sequences have been developed, and the metagenomic data submissions are increasing. We classified metagenomic data into following three levels,
    1. primary metagenome assemblies with environmental samples
    2. binned metagenome assemblies with binned samples
    3. MAGs with MIMAG samples (See [Nat. Biotechnol.
        35:725-731 (2017)](https://www.nature.com/articles/nbt.3893))
We will accept 1) and 2) into SRA Analysis objects and 3) into flat files.

"Organism name" for negative controls  
:  For control experiment, submitters like to use non-organism for BioSample. So, [blank sample](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Info&id=2582415) is added to taxonomy database as an "organism name" for non-organism.

Genome sequences with suspicious descriptions of origin species  
:  We discussed how to deal with cases that Average Nucleotide Identity analysis indicates the correspondence between the genome sequence and the described species name is suspicious.

#### Discussion Items Related to SRA Data <a name="2019-sra"></a>

  -   INSDC partners confirmed, in cases of analyses by single-cell sequencing or others, we can accept descriptions of a representative sample instead of huge numbers of individual descriptions for BioSample. Descriptions for individual samples are accepted into GEO, ArrayExpress, and GEA, if necessary.
  -   In cases of genome submission using PacBio, we will ask submitters to provide motif summary files for methylation analyses.
  -   Currently, SRA Analysis object is not always shared among INSDC members. We discussed how to share SRA Analysis object.

#### Forthcoming changes in The DDBJ/ENA/GenBank Feature Table: Definition <a name="2019-ft"></a>

Some parts of applied in advance to [/gap\_type](/ddbj/qualifiers-e.html#gap_type) and [/linkage\_evidence](/ddbj/qualifiers-e.html#linkage_evidence) qualifiers, [AGP
Specification](https://www.ncbi.nlm.nih.gov/assembly/agp/AGP_Specification/) will be revised to [version 2.1](https://www.ncbi.nlm.nih.gov/assembly/agp AGP_Specification/proposed-version-2-1/).
Related to the revision, controlled vocabularies (CV) for
[/linkage\_evidence](/ddbj/qualifiers-e.html#linkage_evidence) qualifier will be modified. 'proximity ligation' will be added to CV and 'strobe' will be no longer used.  
The modifications will be applied after October 2019 with the next revision of [Feature Table Definition](/ddbj/full_index-e.html).

#### Data Access Policy

Preprint and data publication  
:  Recently, many researchers disclose their results via preprint servers and INSDC accession numbers are often described in preprints. INSDC members agreed to release the sequence data when INSDC accession numbers are found in preprints. Related to this, we revised [INSDC Status Document](/insdc-status-e.html).

Access and Benefit-Sharing (ABS)  
:  The UN has established a Preparatory Committee that is active in developing a resolution around Biodiversity Beyond National Jurisdiction ([BBNJ](https://www.un.org/bbnj/)). This committee will investigate about systems for ABS. INSDC members have also been asked for their opinions from various people concerned. Related to the issue on [2018](#2018), the Conference of the Parties Convention on Biological Diversity 14 (COP14) and Nagoya protocol, INSDC will state about importance of [our policy](/insdc-e.html#policy), free and unrestricted access to all of the data records.

NCBI shifting SRA data to cloud  
:  We discussed the handling tasks regarding the policy of [NCBI shifting SRA data to cloud](https://www.nih.gov/news-events/news-releases/nih-makes-strides-accelerate-discoveries-cloud).