---
layout: indexed_content
title: UME User’s manual
category: services
---

<div id="page" class="top">

<div id="header" class="clearfix">

<div id="header-top">

[![](/images/parts/ddbj-logo-white.svg)](/index-e.html "DNA Data Bank of Japan")

<div id="header-top-left">

<span class="resources-btn">Services**</span>

</div>

<div id="header-top-right">

  - [Login & Submit](https://ddbj.nig.ac.jp/D-way)
  - [Policies and Disclaimers](/policies-e.html)
  - [Contact](/contact-e.html)
  - [Japanese](/ddbj/ume.html)

</div>

</div>

<div class="resources-contents">

<div class="boxContainer">

<div class="resources-box">

  - ### ![](/images/logos/service-org.svg)Search

  - [getentry](http://getentry.ddbj.nig.ac.jp/top-e.html "Retrieve sequence records by accession numbers")

  - [ARSA](http://ddbj.nig.ac.jp/arsa/ "Retrieve sequence and annotation by keywords")

  - [DRA
    Search](http://ddbj.nig.ac.jp/DRASearch "Search and retrieve sequencing data.")

  - [TXSearch](http://ddbj.nig.ac.jp/tx_search/?lang=en "Search unified taxonomy database")

  - [BLAST](http://blast.ddbj.nig.ac.jp/blastn?lang=en "Search nucleotide sequences by sequence similarity")

<!-- end list -->

  - ### ![](/images/logos/wrench.svg)Analysis

  - [Vector Screening
    System](http://ddbj.nig.ac.jp/vecscreen/ "Screen vector sequences")

  - [ClustalW](http://clustalw.ddbj.nig.ac.jp/index.php?lang=en "Multiple alignment and tree-making")

  - [WABI (Web API for
    Biology)](/search/help/wabi/wabi-e.html "Web API for using the search services of DDB")

  - [DDBJ FTP
    Site](ftp://ftp.ddbj.nig.ac.jp/ "Download public data by ftp")

</div>

<div class="resources-box">

  - ### ![](/images/logos/database-org.svg)Databases

  - [Annotated/Assembled Sequences
    (DDBJ)](/ddbj/index-e.html "Annotated/Assembled Sequences (DDBJ)")

  - [Sequence Read Archive
    (DRA)](/dra/index-e.html "Sequence Read Archive")

  - [Genomic Expression Archive
    (GEA)](/gea/index-e.html "Genomic Expression Archive")

  - [BioProject](/bioproject/index-e.html "BioProject")

  - [BioSample](/biosample/index-e.html "BioSample")

  - [Japanese Genotype-phenotype Archive
    (JGA)](/jga/index-e.html "Japanese Genotype-phenotype Archive")

  - [Submission portal
    D-way](https://ddbj.nig.ac.jp/D-way "Submission portal D-way")

<!-- end list -->

  - ### ![](/images/logos/sc-org.svg)NIG SuperComputer

  - [NIG
    SuperComputer](https://sc.ddbj.nig.ac.jp/en "NIG SuperComputer")

</div>

<div class="resources-box dbcls">

  - ### ![](/images/logos/dbcls-s.svg)DBCLS Services

  - [AOE](http://aoe.dbcls.jp/en "Statistics and trends of gene expression data")

  - [CRISPRdirect](https://crispr.dbcls.jp/ "Designing CRISPR/Cas9 guide RNA with reduced off-target sites")

  - [DBCLS
    SRA](http://sra.dbcls.jp/ "Statistics and trends of SRA data")

  - [Gendoo](http://gendoo.dbcls.jp/ "Functional profiling of gene and disease features for omics analysis")

  - [GGGenome](https://gggenome.dbcls.jp/en/ "A ultrafast sequence search")

  - [GGRNA](https://ggrna.dbcls.jp/en/ "A Google-like, ultrafast search engine for genes and transcripts")

  - [RefEx](http://refex.dbcls.jp/index.php?lang=en "A web tool for a comfortable search of reference data for gene expression analysis")

</div>

</div>

</div>

<div id="header-middle" class="clearfix">

<div class="page-title">

[DDBJ Annotated/Assembled Sequences](/ddbj/index-e.html)

</div>

<div class="search-box">

<div id="search-box-flex">

<div class="search-options-wrapper">

BI-DDBJ Web Sites Annotated/Assembled Sequences Sequence Read Archive

</div>

</div>

</div>

</div>

<div id="header-bottom">

  - [Home](/ddbj/index-e.html)
  - [Submission![](/images/parts/arrow_down.svg)](/ddbj/submission-e.html)
      - [Before Submission](submission-e.html)
      - [Web submission](websub-e.html)
      - [Mass submission](mss-e.html)
      - [Data Update](updt-e.html)
  - [Search![](/images/parts/arrow_down.svg)](http://ddbj.nig.ac.jp/arsa/?lang=en)
      - [getentry](http://getentry.ddbj.nig.ac.jp/top-e.html)
      - [ARSA](http://ddbj.nig.ac.jp/arsa/?lang=en)
  - [Flat File![](/images/parts/arrow_down.svg)](/ddbj/flat-file-e.html)
      - [Feature key](/ddbj/features-e.html)
      - [Qualifier key](/ddbj/qualifiers-e.html)
      - [Organism qualifier](/ddbj/organism-e.html)
      - [Description of Location](/ddbj/location-e.html)
      - [Protein Coding Sequence](/ddbj/cds-e.html)
      - [The Genetic Codes](/ddbj/geneticcode-e.html)
      - [Codes Used in Sequence Description](/ddbj/code-e.html)
      - [Example of Submission](/ddbj/example-e.html)
  - [Data
    Categories![](/images/parts/arrow_down.svg)](/ddbj/data-categories-e.html)
      - [Data Submission from Genome Project](/ddbj/genome-e.html)
      - [WGS](/ddbj/wgs-e.html)
      - [CON](/ddbj/con-e.html)
      - [GSS](/ddbj/gss-e.html)
      - [HTG](/ddbj/htg-e.html)
      - [Submission of environmental
        sequences](/ddbj/environmental-e.html)
      - [ENV](/ddbj/env-e.html)
      - [TLS](/ddbj/tls-e.html)
      - [Data Submission from Transcriptome
        Project](/ddbj/transcriptome-e.html)
      - [TSA](/ddbj/tsa-e.html)
      - [EST](/ddbj/est-e.html)
      - [HTC](/ddbj/htc-e.html)
      - [Third Party Data (TPA)](/ddbj/tpa-e.html)
  - [FAQ](/ddbj/faq-e.html)
  - [Other![](/images/parts/arrow_down.svg)](/)
      - [Patent](/ddbj/pat-data-e.html)
      - [MGA](/ddbj/mga-e.html)

</div>

</div>

<div id="main" class="clearfix">

<div id="breadcrumb" class="clearfix">

  - [HOME](/ddbj/index-e.html "DDBJ Annotated/Assembled Sequences top page")
  - \>
  - [MSS - Mass Submission System](/ddbj/mss-e.html)
  - \>
  - UME User’s manual

</div>

<div id="primary">

<div id="page_main">

<div id="book">

# UME User’s manual

<div class="section book">

UME (Utilities for MSS file Error check) is a stand-alone tool to check
syntax and format of [Sequence file](/ddbj/file-format-e.html#sequence)
and [Annotation file](/ddbj/file-format-e.html#annotation) required for
using Mass Submission System of DDBJ and to check amino acid translation
of [CDS feature](/ddbj/cds-e.html) based on the description of each
file. UME includes both Parser and transChecker.

<div class="section chapter">

## Basic function

<div class="main_table format nowrap last-wrap" data-"="">

| Name                          | Functions                                                                                                                                                                               |
| ----------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| [Parser](#validation-2)       | Verification of the syntax and format for [Sequence file](/ddbj/file-format-e.html#sequence) and [Annotation file](/ddbj/file-format-e.html#annotation).                                |
| [transChecker](#validation-3) | Validation of the amino acid translation for [CDS (protein-coding sequence)](/ddbj/cds-e.html) feature.                                                                                 |
| [FFconv](#validation-4)       | Making simulated format of [DDBJ flat file](/ddbj/flat-file-e.html) from [Sequence file](/ddbj/file-format-e.html#sequence) and [Annotation file](/ddbj/file-format-e.html#annotation). |
| [AGPParser](#agpparser)       | For the submission of contigs ([CON entries](/ddbj/con-e.html)), [AGP file](/ddbj/file-format-e.html#agp) and [Annotation file](/ddbj/file-format-e.html#annotation).                   |

</div>

</div>

<div class="section chapter">

## Install

Download UME package from [Validation tool for MSS data
file](/ddbj/mss-tool-e.html).  
Uncompress the downloaded file to extract its contents into the UME
folder.

  - Windows: Uncompress UME\_win.zip  
    In case of Windows, you can likely explore the files packed in zip
    format, directly, like as a folder. However, to run UME, all files
    in the UME package must be uncompressed.
  - unix/macOS: Uncompress UME\_unix.zip  
    Extract the zipped file using a decompression program or the
    following command.
    ``` code
    $ gunzip UME_unix.zip
    ```
  - macOS: Regarding the available file names  
    DO NOT use multibyte character(s) for any file or folder name of
    [Sequence file](/ddbj/file-format-e.html#sequence), [Annotation
    file](/ddbj/file-format-e.html#annotation), or [AGP
    file](/ddbj/file-format-e.html#agp) when you use UME on some
    versions of macOS. UME may not work when multibyte character is
    included in the file or folder name.

</div>

<div class="section chapter">

## Initialization

<div class="section section">

### Starting UME

You can run UME application by double clicking the UME icon (Depending
on your PC environment, some extension of UME file would be displayed)
in the UME folder.

On macOS, you may not be able to run UME by double clicking the icon.  
In that case, please run UME by typing commands on Terminal window like
as follows.

``` code
$ cd UME    # Move to directory that you have expanded UME package. 
$ ./UME.sh
```

![](/images/ddbj/UME1-e.png "Starting UME")

  - lib folder  
    Directory which includes library files used by UME. If you move this
    directory or any file in it, UME would not properly work.
  - resource folder  
    Directory which includes resource files used by UME. If you move
    this directory or any file in it, UME would not properly work.
  - sample folder  
    Including sample files for DDBJ submission.
  - UME file  
    Start UME application by double-clicking this file.
  - license file  
    A text file for license agreement. Please read at first. The
    contents of the file are the same as [End-user license
    agreement](/ddbj/mss-tool-e.html#license).

</div>

<div class="section section">

### Initial setting of UME

At the first time, you run UME application, change the maximum size of
RAM for UME to optimize run-time environments. The default configuration
of it is 512 Mbytes. [![](/images/ddbj/UME2.png)](/images/ddbj/UME2.png)

  - (1) Click Config in menu bar.
  - (2) Select the appropriate size of Max memory for runtime from
    pull-down menu.

</div>

</div>

<div class="section chapter">

## Verification of submission files

<div class="section section">

### Check panel: basic verification of submission files

You can move to this panel by clicking the Check tab enclosed in the red
square.

![](/images/ddbj/UME3.png
"Check panel: basic verification of submission files")

  - (1) Specify the subjects, both [Annotation
    file](/ddbj/file-format-e.html#annotation) and [Sequence
    file](/ddbj/file-format-e.html#sequence).
  - (2) By clicking Execute button, verify the files specified at step
    (1) by Parser.
  - (3) By clicking Execute button, validate amino acid translation of
    the files specified at step (1) by transChecker.
  - (4) By clicking Execute button, process the files specified at step
    (1) by FFconv.
  - (5) By clicking Exit button, Close UME.

</div>

<div class="section section">

### Parser: verification of syntax and format

Execute Parser through the following steps.

![](/images/ddbj/UME5.png "Parser: verification of syntax and format")

Click Check tab and move to Check panel.

  - (1) Specify the subjects, both [Annotation
    file](/ddbj/file-format-e.html#annotation) and [Sequence
    file](/ddbj/file-format-e.html#sequence).
  - (2) By clicking Execute button, process the files specified at step
    (1) by Parser.
  - (3) When you check the box, Save log in file, the result log of
    Parser is saved as a text file.
  - (4) The window of Parser log is opened.
  - (5) The progress of Parser is indicated.
  - (6) The result log of Parser is displayed.
  - (7) By clicking Save button, the contents of (6) are saved as a text
    file.
  - (8) By clicking Close button, the window, Parser result, is closed.

<div class="attention no_color">

When an error occurs, the parser outputs an error message.  
Please reffer: [Parser Error Eessages](/ddbj/validator-e.html#parser),
in detail.

</div>

</div>

<div class="section section">

### transChecker: validation of amino acid translation

transChecker can translate nucleotide sequences of [Sequence
file](/ddbj/file-format-e.html#sequence) into amino acid sequences
corresponding to [CDS features](/ddbj/cds-e.html) specified at
Annotation file.  
Before using transChecker, both two files should be checked by
[Parser](#validation-2), and remove all of Parser errors.  
Please note that when some Parser error is found, transChecker is
terminated without any result. Execute transChecker through the
following steps.

![](/images/ddbj/UME6.png "transChecker")

Click Check tab to move to Check panel.

  - (1) Specify the subjects, both [Annotation
    file](/ddbj/file-format-e.html#annotation) and [Sequence
    file](/ddbj/file-format-e.html#sequence).
  - (2) By clicking Execute button, process the files specified at step
    (1) by transChecker.
  - (3) When you check the box, Save log in file, the execution log and
    error messages of transChecker are saved as a text file.
  - (4) When you check the box, Save translated sequences in file the
    amino acid sequences translated by transChecker are saved as a text
    file in either of two formats.  
    You can specify the format for translated amino acid sequences from
    following two types;
    <div class="news_post_container glossary">
    |                                                                            |                                                                      |
    | -------------------------------------------------------------------------- | -------------------------------------------------------------------- |
    | [FASTA-like format](/ddbj/transchecker.html#output-1-1)                    | in case of selecting Translate in FASTA format                       |
    | [Alignment with nucleotide sequence](/ddbj/transchecker-e.html#output-1-2) | in case of selecting Translate in alignment with nucleotide sequence |
    </div>
  - (5) The window of transChecker is opened.
  - (6) The progress of transChecker is indicated.
  - (7) The execution log and error messages of transChecker are
    displayed.
  - (8) By clicking Save button, the contents of (7) are saved as a text
    file.
  - (9) The amino acid sequences translated by transChecker are
    displayed in the format selected at (4).
  - (10) By clicking Save button, the contents of (9) are saved as a
    text file.
  - (11) By clicking Close button, the window, transChecker result, is
    closed.

<div class="attention no_color">

When an error occurs, the transChecker outputs an error message.  
Please reffer: [transChecker Error
Eessages](/ddbj/validator-e.html#transchecker), in detail.

</div>

</div>

<div class="section section">

### FFconv: confirmation by making simulated DDBJ flat file

FFconv can make simulated format of [DDBJ flat
file](/ddbj/flat-file-e.html) from [Annotation
file](/ddbj/file-format-e.html#annotation) and [Sequence
file](/ddbj/file-format-e.html#sequence).

Before using FFconv, both two files should be checked by , and remove
all of Parser errors. Please note that when some Parser error is found,
FFconv is terminated without any result.

Execute FFconv through the following steps.

![](/images/ddbj/UME7.png "FFconv")

Click Check tab to move to Check panel.

  - (1) Specify the subjects, both [Annotation
    file](/ddbj/file-format-e.html#annotation) and [Sequence
    file](/ddbj/file-format-e.html#sequence).
  - (2) By clicking Execute button, make simulated format of DDBJ flat
    file from the files specified at step (1) by FFconv.
  - (3) When you check the box, Save flat file in file, the result of
    FFconv, simulated format of DDBJ flat file, is saved as a text file.
  - (4) The window of FFconv result is opened.
  - (5) The progress of FFconv is indicated.
  - (6) The execution log of FFconv is displayed.
  - (7) By clicking Save button, the contents of (6) are saved as a text
    file.
  - (8) The result of FFconv, simulated format of DDBJ flat file, is
    displayed.
  - (9) By clicking Save button, the contents of (8) are saved as a text
    file.
  - (10) By clicking Close button, the window, FFconv result, is closed.

</div>

</div>

<div class="section chapter">

## Verification of contig submission files

<div class="section section">

### Contig panel: verification of AGP file for contig submission

You can move to this panel by clicking the Contig tab enclosed in the
red square.

![](/images/ddbj/UME4.png "Verification of contig submission files")

  - (1) Specify the subject, [Annotation
    file](/ddbj/file-format-e.html#annotation).
  - (2) Specify the subject, [AGP file](/ddbj/file-format-e.html#agp).
  - (3) Specify the subject, Multi-fasta file for components in AGP
    file.
  - (4) By clicking Execute button, process the files specified at steps
    (1) to (2) by AGPParser.
  - (5) By clicking Exit t button, Close UME.

</div>

<div class="section section">

### Executing AGPParser

Basically, AGPParser checks the format of [AGP
file](/ddbj/file-format-e.html#agp).AGP file. When you specify
[Annotation file](/ddbj/file-format-e.html#annotation) and/or
multi-fasta file including all of sequences cited by AGP file, AGPParser
also checks the consistency between the two or three files.

![](/images/ddbj/UME8.png "Executing AGPParser")

Click Contig tab to move to Contig pannel.

  - (1) Specify the subject, [Annotation
    file.](/ddbj/file-format-e.html#annotation) Not required.
  - (2) Specify the subject, [AGP
    file](/ddbj/file-format-e.html#agp).Required.
  - (3) Specify the subject, the multi-fasta file including all of
    sequences cited by [Sequence
    file](/ddbj/file-format-e.html#sequence). Not required.
  - (4) By clicking Execute button,AGPParser checks the specified file
    at the steps (1) to (3).
  - (5) When you check the box, Construct sequence from AGP and
    multi-fasta files, AGPParser constructs a text file for nucleotide
    sequence(s) on accordance with the specification of the two files.
  - (6) Check the button, Save log in file,then the result log of
    AGPParser is saved as a text file.
  - (7) The window of AGPParser log is opened.
  - (8) The progress of AGPParser is indicated.
  - (9) The result log of AGPParser is displayed.
  - (10) By clicking Save button, the contents of (9) are saved as a
    text file.
  - (11) By clicking Close button, the window of AGPParser log is
    closed.

<div class="attention no_color">

When an error occurs, the AGPParser outputs an error message.  
Please reffer: [AGPParser Error
Eessages](/ddbj/validator-e.html#agpparser), in detail.

</div>

</div>

</div>

</div>

</div>

</div>

</div>

<div id="side_navigation">

<div class="side_nav_menu handbook_contents handbook ume">

<div id="in_this_page">

## In this page

</div>

</div>

<div class="side_nav_menu handbook_related">

<div id="related_pages">

## Related pages

  - [Submission File Format](/ddbj/file-format-e.html)
  - [Validation tools for MSS data files](/ddbj/mss-tool-e.html)
  - [Parser User’s Manual](/ddbj/parser-e.html)
  - [transChecker User’s Manual](/ddbj/transchecker-e.html)
  - [Validator error message](/ddbj/validator-e.html)
  - [Application form for MSS](/ddbj/mass-form-e.html)

</div>

</div>

</div>

</div>

<div id="footer_original" class="clearfix">

  - [Policies and
    Disclaimers](/policies-e.html "Policies and Disclaimers")
  - [News](/news/en/index-e.html "News archives")
  - [FAQs](/faq/en/index-e.html "Frequently asked questions")
  - [Sitemap](/sitemap-e.html "Sitemap")
  - [Calendar](/calendar-e.html "DDBJ Center Calendar")
  - [Address](/access-e.html "DDBJ Center address")
  - [Contact](/contact-e.html "Contact to DDBJ Center")
  - Last modified: 2018-04-18

</div>

</div>
