---
layout: indexed_content
title: Feature key
pathname: features-e
category: flatfile
---

<div id="page" class="top">

<div id="header" class="clearfix">

<div id="header-top">

[![](/images/parts/ddbj-logo-white.svg)](/index-e.html "DNA Data Bank of Japan")

<div id="header-top-left">

<span class="resources-btn">Services**</span>

</div>

<div id="header-top-right">

  - [Login & Submit](https://ddbj.nig.ac.jp/D-way)
  - [Policies and Disclaimers](/policies-e.html)
  - [Contact](/contact-e.html)
  - [Japanese](/ddbj/features.html)

</div>

</div>

<div class="resources-contents">

<div class="boxContainer">

<div class="resources-box">

  - ### ![](/images/logos/service-org.svg)Search

  - [getentry](http://getentry.ddbj.nig.ac.jp/top-e.html "Retrieve sequence records by accession numbers")

  - [ARSA](http://ddbj.nig.ac.jp/arsa/ "Retrieve sequence and annotation by keywords")

  - [DRA
    Search](http://ddbj.nig.ac.jp/DRASearch "Search and retrieve sequencing data.")

  - [TXSearch](http://ddbj.nig.ac.jp/tx_search/?lang=en "Search unified taxonomy database")

  - [BLAST](http://blast.ddbj.nig.ac.jp/blastn?lang=en "Search nucleotide sequences by sequence similarity")

<!-- end list -->

  - ### ![](/images/logos/wrench.svg)Analysis

  - [Vector Screening
    System](http://ddbj.nig.ac.jp/vecscreen/ "Screen vector sequences")

  - [ClustalW](http://clustalw.ddbj.nig.ac.jp/index.php?lang=en "Multiple alignment and tree-making")

  - [WABI (Web API for
    Biology)](/search/help/wabi/wabi-e.html "Web API for using the search services of DDB")

  - [DDBJ FTP
    Site](ftp://ftp.ddbj.nig.ac.jp/ "Download public data by ftp")

</div>

<div class="resources-box">

  - ### ![](/images/logos/database-org.svg)Databases

  - [Annotated/Assembled Sequences
    (DDBJ)](/ddbj/index-e.html "Annotated/Assembled Sequences (DDBJ)")

  - [Sequence Read Archive
    (DRA)](/dra/index-e.html "Sequence Read Archive")

  - [Genomic Expression Archive
    (GEA)](/gea/index-e.html "Genomic Expression Archive")

  - [BioProject](/bioproject/index-e.html "BioProject")

  - [BioSample](/biosample/index-e.html "BioSample")

  - [Japanese Genotype-phenotype Archive
    (JGA)](/jga/index-e.html "Japanese Genotype-phenotype Archive")

  - [Submission portal
    D-way](https://ddbj.nig.ac.jp/D-way "Submission portal D-way")

<!-- end list -->

  - ### ![](/images/logos/sc-org.svg)NIG SuperComputer

  - [NIG
    SuperComputer](https://sc.ddbj.nig.ac.jp/en "NIG SuperComputer")

</div>

<div class="resources-box dbcls">

  - ### ![](/images/logos/dbcls-s.svg)DBCLS Services

  - [AOE](http://aoe.dbcls.jp/en "Statistics and trends of gene expression data")

  - [CRISPRdirect](https://crispr.dbcls.jp/ "Designing CRISPR/Cas9 guide RNA with reduced off-target sites")

  - [DBCLS
    SRA](http://sra.dbcls.jp/ "Statistics and trends of SRA data")

  - [Gendoo](http://gendoo.dbcls.jp/ "Functional profiling of gene and disease features for omics analysis")

  - [GGGenome](https://gggenome.dbcls.jp/en/ "A ultrafast sequence search")

  - [GGRNA](https://ggrna.dbcls.jp/en/ "A Google-like, ultrafast search engine for genes and transcripts")

  - [RefEx](http://refex.dbcls.jp/index.php?lang=en "A web tool for a comfortable search of reference data for gene expression analysis")

</div>

</div>

</div>

<div id="header-middle" class="clearfix">

<div class="page-title">

[DDBJ Annotated/Assembled Sequences](/ddbj/index-e.html)

</div>

<div class="search-box">

<div id="search-box-flex">

<div class="search-options-wrapper">

BI-DDBJ Web Sites Annotated/Assembled Sequences Sequence Read Archive

</div>

</div>

</div>

</div>

<div id="header-bottom">

  - [Home](/ddbj/index-e.html)
  - [Submission![](/images/parts/arrow_down.svg)](/ddbj/submission-e.html)
      - [Before Submission](submission-e.html)
      - [Web submission](websub-e.html)
      - [Mass submission](mss-e.html)
      - [Data Update](updt-e.html)
  - [Search![](/images/parts/arrow_down.svg)](http://ddbj.nig.ac.jp/arsa/?lang=en)
      - [getentry](http://getentry.ddbj.nig.ac.jp/top-e.html)
      - [ARSA](http://ddbj.nig.ac.jp/arsa/?lang=en)
  - [Flat File![](/images/parts/arrow_down.svg)](/ddbj/flat-file-e.html)
      - [Feature key](/ddbj/features-e.html)
      - [Qualifier key](/ddbj/qualifiers-e.html)
      - [Organism qualifier](/ddbj/organism-e.html)
      - [Description of Location](/ddbj/location-e.html)
      - [Protein Coding Sequence](/ddbj/cds-e.html)
      - [The Genetic Codes](/ddbj/geneticcode-e.html)
      - [Codes Used in Sequence Description](/ddbj/code-e.html)
      - [Example of Submission](/ddbj/example-e.html)
  - [Data
    Categories![](/images/parts/arrow_down.svg)](/ddbj/data-categories-e.html)
      - [Data Submission from Genome Project](/ddbj/genome-e.html)
      - [WGS](/ddbj/wgs-e.html)
      - [CON](/ddbj/con-e.html)
      - [GSS](/ddbj/gss-e.html)
      - [HTG](/ddbj/htg-e.html)
      - [Submission of environmental
        sequences](/ddbj/environmental-e.html)
      - [ENV](/ddbj/env-e.html)
      - [TLS](/ddbj/tls-e.html)
      - [Data Submission from Transcriptome
        Project](/ddbj/transcriptome-e.html)
      - [TSA](/ddbj/tsa-e.html)
      - [EST](/ddbj/est-e.html)
      - [HTC](/ddbj/htc-e.html)
      - [Third Party Data (TPA)](/ddbj/tpa-e.html)
  - [FAQ](/ddbj/faq-e.html)
  - [Other![](/images/parts/arrow_down.svg)](/)
      - [Patent](/ddbj/pat-data-e.html)
      - [MGA](/ddbj/mga-e.html)

</div>

</div>

<div id="main" class="clearfix">

<div id="breadcrumb" class="clearfix">

  - [HOME](/ddbj/index-e.html "DDBJ Annotated/Assembled Sequences top page")
  - \>
  - [DDBJ flat file format](/ddbj/flat-file-e.html)
  - \>
  - Feature Key

</div>

<div id="primary">

<div id="page_main">

# Feature Key

<div id="description_container">

<div id="description">

The feature keys used and recommended for DDBJ submissions are as
follows.  
For further information of INSDC feature keys, read [Feature Table
Definition](/ddbj/full_index-e.html): [7.2 Appendix II: Feature keys
reference](/ddbj/full_index-e.html#7.2).

## Feature/Qualifier Usage Matrix

The chart below explains recommended combinations of feature and
qualifier keys for DDBJ submissions.

[Feature/Qualifier usage matrix ![](/images/parts/pdf.png
"pdficon")](/files/pdf/ddbj/fq-e.pdf) (November 9, 2016)

For more detail of available combinations of feature and qualifier keys
in INSDC entries, read: [Feature Table
Definition](/ddbj/full_index-e.html): [7.2 Appendix II: Feature keys
reference](/ddbj/full_index-e.html#7.2).

## Definision of Feature key

  - assembly\_gap<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#assembly_gap)</span>  
    gap between two components of a [genome](/ddbj/genome-e.html) or
    [transcriptome](/ddbj/transcriptome-e.html) assembly
  - C\_region<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#C_region)</span>  
    Constant region of immunoglobulin light and heavy chains, and T-cell
    receptor alpha, beta, and gamma chains.  
    Includes one or more exons depending on the particular chain.
  - CDS<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#CDS)</span>  
    coding sequence; sequence of nucleotides that corresponds with the
    sequence of amino acids in a protein (location includes stop
    codon).  
    See also [the page about CDS feature](/ddbj/cds-e.html).
  - centromere<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#centromere)</span>  
    region of biological interest identified as a centromere and which
    has been experimentally characterized
  - D-loop<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#D-loop)</span>  
    displacement loop; a region within mitochondrial DNA in which a
    short stretch of RNA is paired with one strand of DNA, displacing
    the original partner DNA strand in this region;  
    also used to describe the displacement of a region of one strand of
    duplex DNA by a single stranded invader in the reaction catalyzed by
    RecA protein
  - D\_segment<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#D_segment)</span>  
    Diversity segment of immunoglobulin heavy chain, and T-cell receptor
    beta chain.
  - exon<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#exon)</span>  
    region of genome that codes for portion of spliced
    [mRNA](/ddbj/full_index-e.html#mRNA),
    [rRNA](/ddbj/full_index-e.html#rRNA) and
    [tRNA](/ddbj/full_index-e.html#tRNA)
  - gap<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#gap)</span>  
    gap in the sequence; sequencing gap other than
    [assembly\_gap](#assembly_gap)
  - intron<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#intron)</span>  
    a segment of DNA that is transcribed, but removed from within the
    transcript by splicing together the sequences
    ([exon](/ddbj/full_index-e.html#exon)s) on either side of it
  - J\_segment<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#J_segment)</span>  
    Joining segment of immunoglobulin light and heavy chains, and T-cell
    receptor alpha, beta, and gamma chains.
  - mat\_peptide<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#mat_peptide)</span>  
    mature peptide or protein coding sequence; coding sequence for the
    mature or final peptide or protein product following
    post-translational modification.  
    The location does not include the stop codon (unlike the
    corresponding [CDS](/ddbj/full_index-e.html#CDS))
  - misc\_binding<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#misc_binding)</span>  
    site in nucleic acid which covalently or non-covalently binds
    another moiety that cannot be described by any other binding key
    ([primer\_bind](/ddbj/full_index-e.html#primer_bind) or
    [protein\_bind](/ddbj/full_index-e.html#protein_bind))
  - misc\_difference<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#misc_difference)</span>  
    feature sequence is different from that presented in the entry and
    cannot be described by any other "difference" key;  
    [variation](/ddbj/full_index-e.html#variation),
    [modified\_base](/ddbj/full_index-e.html#modified_base)  
    Comment  
    The misc\_difference feature should be used to describe variability
    that arises as a result of genetic manipulation (e.g. site directed
    mutagenesis).
  - misc\_feature<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#misc_feature)</span>  
    region of biological interest which cannot be described by any other
    feature key; a new or rare feature.
  - misc\_RNA<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#misc_RNA)</span>  
    any transcript or RNA product that cannot be defined by other RNA
    keys  
    [prim\_transcript](/ddbj/full_index-e.html#prim_transcript),
    [precursor\_RNA](/ddbj/full_index-e.html#precursor_RNA),
    [mRNA](/ddbj/full_index-e.html#mRNA),
    [5'UTR](/ddbj/full_index-e.html#5UTR),
    [3'UTR](/ddbj/full_index-e.html#3UTR),
    [exon](/ddbj/full_index-e.html#exon),
    [CDS](/ddbj/full_index-e.html#CDS),
    [sig\_peptide](/ddbj/full_index-e.html#sig_peptide),
    [transit\_peptide](/ddbj/full_index-e.html#transit_peptide),
    [mat\_peptide](/ddbj/full_index-e.html#mat_peptide),
    [intron](/ddbj/full_index-e.html#intron),
    [polyA\_site](/ddbj/full_index-e.html#polyA_site),
    [ncRNA](/ddbj/full_index-e.html#ncRNA),
    [rRNA](/ddbj/full_index-e.html#rRNA),
    [tRNA](/ddbj/full_index-e.html#tRNA),
    [tmRNA](/ddbj/full_index-e.html#tmRNA)
  - misc\_structure<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#misc_structure)</span>  
    any secondary or tertiary nucleotide structure or conformation that
    cannot be described by other "Structure" keys;  
    [stem\_loop](/ddbj/full_index-e.html#stem_loop),
    [D-loop](/ddbj/full_index-e.html#D-loop)
  - mobile\_element<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#mobile_element)</span>  
    region of genome sequence containing mobile element
  - modified\_base<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#modified_base)</span>  
    the indicated nucleotide is a modified nucleotide and should be
    substituted for by the indicated molecule (given in the
    [mod\_base](/ddbj/qualifiers-e.html#mod_base) qualifier value)
  - mRNA<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#mRNA)</span>  
    messenger RNA; includes 5'untranslated region
    ([5'UTR](/ddbj/full_index-e.html#5UTR)), coding sequences
    ([CDS](/ddbj/full_index-e.html#CDS),
    [exon](/ddbj/full_index-e.html#exon)) and 3'untranslated region
    ([3'UTR](/ddbj/full_index-e.html#3UTR))
  - ncRNA<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#ncRNA)</span>  
    a non-protein-coding gene, other than ribosomal RNA
    ([rRNA](/ddbj/full_index-e.html#rRNA)) and transfer RNA
    ([tRNA](/ddbj/full_index-e.html#tRNA)), the functional molecule of
    which is the RNA transcript
  - operon<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#operon)</span>  
    region containing polycistronic transcript including a cluster of
    genes that are under the control of the same regulatory
    sequences/promoter and in the same biological pathway
  - oriT<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#oriT)</span>  
    iorigin of transfer; region of a DNA molecule where transfer is
    initiated during the process of conjugation or mobilization
  - precursor\_RNA<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#precursor_RNA)</span>  
    any RNA species that is not yet the mature RNA product
  - primer\_bind<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#primer_bind)</span>  
    Non-covalent primer binding site for initiation of replication,
    transcription, or reverse transcription. Includes site(s) for
    synthetic e.g., PCR primer elements
  - propeptide<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#propeptide)</span>  
    propeptide coding sequence; coding sequence for the domain of a
    proprotein that is cleaved to form the mature protein product.
  - protein\_bind<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#protein_bind)</span>  
    non-covalent protein binding site on nucleic acid
  - regulatory<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#regulatory)</span>  
    any region of sequence that functions in the regulation of
    transcription or translation  
    Since December 2014, the following old features have been merged
    into this feature.
  - repeat\_region<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#repeat_region)</span>  
    region of genome containing repeating units
  - rep\_origin<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#rep_origin)</span>  
    origin of replication; starting site for duplication of nucleic acid
    to give two identical copies
  - rRNA<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#rRNA)</span>  
    mature ribosomal RNA; the RNA component of the ribonucleoprotein
    particle (ribosome) which assembles amino acids into proteins
  - sig\_peptide<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#sig_peptide)</span>  
    signal peptide coding sequence; coding sequence for an N-terminal
    domain of a secreted protein;  
    this domain is involved in attaching nascent polypeptide to the
    membrane; leader sequence
  - source<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#source)</span>  
    identifies the biological source of the specified span of the
    sequence.  
    This key is mandatory. Every entry will have, as a minimum, a single
    source key spanning the entire sequence.  
    More than one source key per sequence is permissible
  - stem\_loop<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#stem_loop)</span>  
    hairpin; a double-helical region formed by base-pairing between
    adjacent (inverted) complementary sequences in a single strand of
    RNA or DNA
  - telomere<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#telomere)</span>  
    region of biological interest identified as a telomere and which has
    been experimentally characterized
  - tmRNA<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#tmRNA)</span>  
    transfer messenger RNA; tmRNA acts as a tRNA first, and then as an
    mRNA that encodes a peptide tag;  
    the ribosome translates this mRNA region of tmRNA and attaches the
    encoded peptide tag to the C-terminus of the unfinished protein;  
    this attached tag targets the protein for destruction or
    proteolysis;
  - transit\_peptide<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#transit_peptide)</span>  
    transit peptide coding sequence; coding sequence for an N-terminal
    domain of a nuclear-encoded organellar protein;  
    this domain is involved in post-translational import of the protein
    into the organelle
  - tRNA<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#tRNA)</span>  
    mature transfer RNA, a small RNA molecule (75-85 bases long) that
    mediates the translation of a nucleic acid sequence into an amino
    acid sequence
  - unsure<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#unsure)</span>  
    A small region of sequenced bases, generally 10 or fewer in its
    length, which could not be confidently identified.
  - V\_region<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#V_region)</span>  
    Variable region of immunoglobulin light and heavy chains, and T-cell
    receptor alpha, beta, and gamma chains. Codes for the variable amino
    terminal portion. Can be made up from V\_segments, D\_segments,
    N\_regions and J\_segments.
  - V\_segment<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#V_segment)</span>  
    Variable segment of immunoglobulin light and heavy chains, and
    T-cell receptor alpha, beta, and gamma chains. Codes for most of the
    variable region (V\_region) and the last few amino acids of the
    leader peptide
  - variation<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#variation)</span>  
    a related strain contains stable mutations from the same gene (e.g.,
    RFLPs, polymorphisms, etc.) which differ from the presented sequence
    at this location (and possibly others).
  - 3'UTR<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#3UTR)</span>  
    1\) region at the 3' end of a mature transcript (following the stop
    codon) that is not translated into a protein;  
    2\) region at the 3' end of an RNA virus (following the last stop
    codon) that is not translated into a protein;
  - 5'UTR<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#5UTR)</span>  
    1\) region at the 5' end of a mature transcript (preceding the
    initiation codon) that is not translated into a protein;  
    2\) region at the 5' end of an RNA virus genome (preceding the first
    initiation codon) that is not translated into a protein;

</div>

</div>

</div>

</div>

<div id="side_navigation">

<div class="side_nav_menu handbook_contents page-noside">

<div id="in_this_page">

## In this page

</div>

</div>

</div>

</div>

<div id="footer_original" class="clearfix">

  - [Policies and
    Disclaimers](/policies-e.html "Policies and Disclaimers")
  - [News](/news/en/index-e.html "News archives")
  - [FAQs](/faq/en/index-e.html "Frequently asked questions")
  - [Sitemap](/sitemap-e.html "Sitemap")
  - [Calendar](/calendar-e.html "DDBJ Center Calendar")
  - [Address](/access-e.html "DDBJ Center address")
  - [Contact](/contact-e.html "Contact to DDBJ Center")
  - Last modified: 2019-03-26

</div>

</div>
