---
layout: indexed_content
title: Organism qualifier
pathname: organism-e
category: flatfile
---

<div id="page" class="top">

<div id="header" class="clearfix">

<div id="header-top">

[![](/images/parts/ddbj-logo-white.svg)](/index-e.html "DNA Data Bank of Japan")

<div id="header-top-left">

<span class="resources-btn">Services**</span>

</div>

<div id="header-top-right">

  - [Login & Submit](https://ddbj.nig.ac.jp/D-way)
  - [Policies and Disclaimers](/policies-e.html)
  - [Contact](/contact-e.html)
  - [Japanese](/ddbj/organism.html)

</div>

</div>

<div class="resources-contents">

<div class="boxContainer">

<div class="resources-box">

  - ### ![](/images/logos/service-org.svg)Search

  - [getentry](http://getentry.ddbj.nig.ac.jp/top-e.html "Retrieve sequence records by accession numbers")

  - [ARSA](http://ddbj.nig.ac.jp/arsa/ "Retrieve sequence and annotation by keywords")

  - [DRA
    Search](http://ddbj.nig.ac.jp/DRASearch "Search and retrieve sequencing data.")

  - [TXSearch](http://ddbj.nig.ac.jp/tx_search/?lang=en "Search unified taxonomy database")

  - [BLAST](http://blast.ddbj.nig.ac.jp/blastn?lang=en "Search nucleotide sequences by sequence similarity")

<!-- end list -->

  - ### ![](/images/logos/wrench.svg)Analysis

  - [Vector Screening
    System](http://ddbj.nig.ac.jp/vecscreen/ "Screen vector sequences")

  - [ClustalW](http://clustalw.ddbj.nig.ac.jp/index.php?lang=en "Multiple alignment and tree-making")

  - [WABI (Web API for
    Biology)](/search/help/wabi/wabi-e.html "Web API for using the search services of DDB")

  - [DDBJ FTP
    Site](ftp://ftp.ddbj.nig.ac.jp/ "Download public data by ftp")

</div>

<div class="resources-box">

  - ### ![](/images/logos/database-org.svg)Databases

  - [Annotated/Assembled Sequences
    (DDBJ)](/ddbj/index-e.html "Annotated/Assembled Sequences (DDBJ)")

  - [Sequence Read Archive
    (DRA)](/dra/index-e.html "Sequence Read Archive")

  - [Genomic Expression Archive
    (GEA)](/gea/index-e.html "Genomic Expression Archive")

  - [BioProject](/bioproject/index-e.html "BioProject")

  - [BioSample](/biosample/index-e.html "BioSample")

  - [Japanese Genotype-phenotype Archive
    (JGA)](/jga/index-e.html "Japanese Genotype-phenotype Archive")

  - [Submission portal
    D-way](https://ddbj.nig.ac.jp/D-way "Submission portal D-way")

<!-- end list -->

  - ### ![](/images/logos/sc-org.svg)NIG SuperComputer

  - [NIG
    SuperComputer](https://sc.ddbj.nig.ac.jp/en "NIG SuperComputer")

</div>

<div class="resources-box dbcls">

  - ### ![](/images/logos/dbcls-s.svg)DBCLS Services

  - [AOE](http://aoe.dbcls.jp/en "Statistics and trends of gene expression data")

  - [CRISPRdirect](https://crispr.dbcls.jp/ "Designing CRISPR/Cas9 guide RNA with reduced off-target sites")

  - [DBCLS
    SRA](http://sra.dbcls.jp/ "Statistics and trends of SRA data")

  - [Gendoo](http://gendoo.dbcls.jp/ "Functional profiling of gene and disease features for omics analysis")

  - [GGGenome](https://gggenome.dbcls.jp/en/ "A ultrafast sequence search")

  - [GGRNA](https://ggrna.dbcls.jp/en/ "A Google-like, ultrafast search engine for genes and transcripts")

  - [RefEx](http://refex.dbcls.jp/index.php?lang=en "A web tool for a comfortable search of reference data for gene expression analysis")

</div>

</div>

</div>

<div id="header-middle" class="clearfix">

<div class="page-title">

[DDBJ Annotated/Assembled Sequences](/ddbj/index-e.html)

</div>

<div class="search-box">

<div id="search-box-flex">

<div class="search-options-wrapper">

BI-DDBJ Web Sites Annotated/Assembled Sequences Sequence Read Archive

</div>

</div>

</div>

</div>

<div id="header-bottom">

  - [Home](/ddbj/index-e.html)
  - [Submission![](/images/parts/arrow_down.svg)](/ddbj/submission-e.html)
      - [Before Submission](submission-e.html)
      - [Web submission](websub-e.html)
      - [Mass submission](mss-e.html)
      - [Data Update](updt-e.html)
  - [Search![](/images/parts/arrow_down.svg)](http://ddbj.nig.ac.jp/arsa/?lang=en)
      - [getentry](http://getentry.ddbj.nig.ac.jp/top-e.html)
      - [ARSA](http://ddbj.nig.ac.jp/arsa/?lang=en)
  - [Flat File![](/images/parts/arrow_down.svg)](/ddbj/flat-file-e.html)
      - [Feature key](/ddbj/features-e.html)
      - [Qualifier key](/ddbj/qualifiers-e.html)
      - [Organism qualifier](/ddbj/organism-e.html)
      - [Description of Location](/ddbj/location-e.html)
      - [Protein Coding Sequence](/ddbj/cds-e.html)
      - [The Genetic Codes](/ddbj/geneticcode-e.html)
      - [Codes Used in Sequence Description](/ddbj/code-e.html)
      - [Example of Submission](/ddbj/example-e.html)
  - [Data
    Categories![](/images/parts/arrow_down.svg)](/ddbj/data-categories-e.html)
      - [Data Submission from Genome Project](/ddbj/genome-e.html)
      - [WGS](/ddbj/wgs-e.html)
      - [CON](/ddbj/con-e.html)
      - [GSS](/ddbj/gss-e.html)
      - [HTG](/ddbj/htg-e.html)
      - [Submission of environmental
        sequences](/ddbj/environmental-e.html)
      - [ENV](/ddbj/env-e.html)
      - [TLS](/ddbj/tls-e.html)
      - [Data Submission from Transcriptome
        Project](/ddbj/transcriptome-e.html)
      - [TSA](/ddbj/tsa-e.html)
      - [EST](/ddbj/est-e.html)
      - [HTC](/ddbj/htc-e.html)
      - [Third Party Data (TPA)](/ddbj/tpa-e.html)
  - [FAQ](/ddbj/faq-e.html)
  - [Other![](/images/parts/arrow_down.svg)](/)
      - [Patent](/ddbj/pat-data-e.html)
      - [MGA](/ddbj/mga-e.html)

</div>

</div>

<div id="main" class="clearfix">

<div id="breadcrumb" class="clearfix">

  - [HOME](/ddbj/index-e.html "DDBJ Annotated/Assembled Sequences top page")
  - \>
  - [DDBJ flat file format](/ddbj/flat-file-e.html)
  - \>
  - Organism qualifier

</div>

<div id="primary">

<div id="page_main">

<div id="book">

# Organism qualifier

<div class="section book">

Every value of organism qualifier must be a "scientific name" ranked as
species or lower taxa in taxonomy database.

<div class="section chapter">

## taxonomy database

All organisms that are represented in the sequence data of
DDBJ/EMBL-Bank/GenBank are registered to the taxonomy database.  
For construction of the nucleotide sequence database, it is important to
manage the organism names for the data and also necessary to unify the
diverse names of the organisms.

The taxonomy database is used as the reference database for the unified
organism names.

The primary purpose of the taxonomy database is to unify descriptions of
organism names.  
Consequently, the taxonomy database is not an authoritative source for
nomenclature or classification.  
A taxonomic name may differ from the submitter's proposal or widely used
taxonomic name because only the organism name in the taxonomy database
which is managed by GenBank can be used for the entry.  
Please refer to [description of the taxonomy
database](https://www.ncbi.nlm.nih.gov/Taxonomy/taxonomyhome.html/index.cgi?chapter=howcite).

DDBJ provides a web service called
[TXSearch](http://ddbj.nig.ac.jp/tx_search/?lang=en) to retrieve
organism names in the taxonomy database. This would be helpful as a
reference of taxonomic names when you submit nucleotide sequences to
DDBJ.  
Please note, even though the organism name has already been registered
into the taxonomy database, when the data with the organism name have
not yet been open to public, you can not retrieve the name at
TXSearch.  
<span class="red">Notice)</span> During submission via Nucleotide
Sequence Submission System, please select "Not found in taxonomy
database, but already registered in other sequence data " for
["Category" of organism name](#deasy) to [indicate corresponding to the
case](#private).

To submit nucleotide sequence data to DDBJ or to search taxonomy
database, <span class="red">please make sure that the organism name has
no misspelling. </span>

</div>

<div class="section chapter">

## General rule to describe organism names

In general, an organism should be called its scientific name of species,
however, when the species is not identified or not defined, it would be
some tentative name instead of the scientific name of the species.

So, <span class="red">do NOT inappropriately select any organism name
existing in taxonomy database. </span> Only when you can identify
organism name from which your sequence has been obtained with no doubt
AND the organism name has been already registered to taxonomy database,
you can select the organism name existing in taxonomy database

  - The name of "unidentified organism" or "novel species" should be
    described as a tentative name
  - Sequence similarity of a marker gene is not absolute benchmark in
    phylogenetic relationship.
  - Identical sequences does NOT mean that they are derived from same
    species.

If the organism name that you submit is not in the database, the name
should be newly registered to the taxonomy database through DDBJ.  
Such new organism names will be open to the public on taxonomy database
after the release of corresponding sequence data from DDBJ.  

Before the release of corresponding sequence data from DDBJ, the
organism name can not be available on taxonomy database.  
Please do not hesitate to contact us when you like to update the
organism name of your sequence data, once submitted to taxonomy
database.  
See also the page, [Data Updates/Correction: after getting your
accession number](/ddbj/updt-e.html), when you like to update your data.

In principle, the organism name is required to be one of "scientific
name" in taxonomy database. If you have any questions or comments for
the classification of "scientific name", "synonyms", lineages, etc. on
the taxonomy database, DDBJ can ask managers of taxonomy database any
modification based on evidence papers or references from you. In this
regard, however, in case of different opinions in phylogenetic lineages,
the claim would be often rejected. When you find misspelled name in
taxonomy database, do not hesitate to [contact
us](/contact-e.html#to-ddbj) to correct it.

</div>

<div class="section chapter">

## "Category" of organism name

During submission through [Nucleotide Sequence Submission
System](/ddbj/websub-e.html) you have to select "Category" of organism
name. Here shows a flowchart for judging "Category" of organism on
Nucleotide Sequence Submission System

![](/images/ddbj/organism-e.png)

  - a. artificial constructed or synthesized sequence  
    Select "Artificially constructed or synthesized sequence" in the
    menu of \[Category\]. After referring to [4. Artificially
    constructed sequence](#syn), enter the name in the value of
    \[Scientific name\], appropriately.
  - b. virus  
    After finding the organism name from which your sequence was
    obtained, please select the name.  
    In cases of [pathogenic virus](#virus) , select "Pathogenic viruses"
    in the menu box of \[Category\].  
    Please include strain, serotype, collection\_date, country
    (collection site), host etc. in the values of corresponding
    qualifiers, respectively.
  - c. trinominal name  
    Select "Valid subspecies or variety not registered in taxonomy
    database" in the menu box of \[Category\].  
    After referring to [trinominal name](#trinominal), please include
    subspecies, variety, or strain etc. in the value of \[Scientific
    name\].
  - d. found in taxonomy database  
    After finding the organism name from which your sequence was
    obtained, please select the name.  
    You do not have to select any of \[Category\].
  - e. direct molecular isolation from a bulk environmental DNA sample  
    Select "Obtained by direct molecular isolation from environmental
    sample" in the menu box of \[Category\].  
    After referring to [3. Environmental samples](#env), enter an
    organism name in the \[Scientific name\] box, appropriately.
  - f. unidentified organism or novel species in proposing state  
    In case of a tentative name for an unidentified organism, select
    "Species is not identified".  
    In case of a novel species in proposing state, select "Species is
    not identified" in the menu box of \[Category\].  
    After referring to [2. In case of unidentified species
    names](#unidentified), enter an organism name in the \[Scientific
    name\] box, appropriately.
  - g. not found in taxonomy database, but already submitted other
    sequence data of the organism  
    Select "The name is valid but not registered in taxonomy database"
    in the menu box of \[Category\].  
    Enter the same organism name as previously submitted in the
    \[Scientific name\] box.
  - h. not found in taxonomy database, but validly identified the
    scientific name of species  
    Select "The name is valid but not registered in taxonomy database"
    in the menu box of \[Category\]. After referring to [1. For
    identified species](#species), enter an organism name in the
    \[Scientific name\] box, appropriately.

</div>

<div class="section chapter">

## Details to describe organism names

Though there are still some exceptions, followings are how to describe
organism names for DDBJ submission. If the application of the organism
name to taxonomy database is required, during your sequence submission,
please let us know [reference information](#items)

### 1\. For identified species

In principle, "organism name" is required to be a binomial name, i.e.
the genus name and the species epithet, from which the sequence is
obtained.  
The species name should be described following international code of
nomenclatures, such as International Code of Zoological Nomenclature
(ICZN), International Code of Botanical Nomenclature (ICBN),
International Code of Nomenclature of Bacteria.

Example

  - Homo sapiens

#### trinomial name

In cases of using trinomial name or the like, the name of subspecies,
variety, serotype or strain should be included in the organism name, if
necessary.

Examples

  - Pan troglodytes troglodytes
  - Zea mays subsp. mays
  - Brassica oleracea var. alboglabra
  - Oryza sativa Japonica Group
  - Escherichia coli K-12

Also, the qualifier corresponding to
[sub\_species](/ddbj/qualifiers-e.html#sub_species),
[variety](/ddbj/qualifiers-e.html#variety),
[serotype](/ddbj/qualifiers-e.html#serotype) or
[strain](/ddbj/qualifiers-e.html#strain) is required for the source
feature.

``` code flat-file
                     /organism="Pan troglodytes troglodytes"
                     /sub_species="troglodytes"
  
```

#### microbial genome

Previously, a strain name or some other lower taxon was required for the
organism name of whole genomic scale sequences from microorganisms.  
However, currently, the value of organism qualifier should be just a
scientific name, in principle, even though for microbial genomes.  
Describe a strain name in the [strain](/ddbj/qualifiers-e.html#strain)
qualifier.

``` code flat-file
                   /organism="Candida albicans"
                   /strain="WO-1"
```

#### virus

For viruses, basically, we also accept scientific names, following [the
International Committee on Taxonomy of
Viruses](https://talk.ictvonline.org/).  
<span style="color: #ff0000; font-weight:bold">By 2017, we used informal
names for frequently submitted pathogenic viruses including their strain
names and serotypes in the description of organism name. However, the
rule has been discontinued for the future submissions. </span>

Example

  - Influenza A virus

The qualifiers corresponding to
[collection\_date](/ddbj/qualifiers-e.html#collection_date),
[country](/ddbj/qualifiers-e.html#country),
[host](/ddbj/qualifiers-e.html#host),
[strain](/ddbj/qualifiers-e.html#strain) and
[serotype](/ddbj/qualifiers-e.html#serotype) are required for the source
feature.

Please describe them with appropriate qualifiers as below example.

``` code flat-file
                   /country="Japan:Tokyo"
                   /collection_date="2007-11-01"
                   /host="Gallus gallus"
                   /mol_type="viral cRNA"
                   /organism="Influenza A virus"
                   /serotype="H7N7"
                   /strain="A/chicken/Tokyo/2007"
```

#### hybrid

In cases of hybrids, the scientific names would be like as follows;

Examples

  - Rosa alba x Rosa corymbifera
  - Malus x domestica
  - Lilium hybrid division I

#### not found in taxonomy database

If the name is not available on taxonomy database
([TXSearch](http://ddbj.nig.ac.jp/tx_search/?lang=en)), please tell us
any of following items during your sequence submission.

Useful items for application of organism names to taxonomy database

  - [taxonomic lineage](#lineage)
  - [valid publication for species](#ref)
  - [already issued accession number](#acc)
  - [process of sampling and/or sequencing](#isolate)

### 2\. In case of unidentified species names

#### 2-1. In case of species not to be identified

If the scientific name is unclear and/or unidentified, we adopt a
tentative name for the organism. The tentative name is made up with the
lineage (in many cases, genus names) that as far as submitters could
specify.

Format

``` code
"<genus name> sp."
"<family (or upper) name> bacterium"
```

Example

  - Acetobacter sp.
  - Acetobacteraceae bacterium

The qualifiers corresponding to the lower rank such as
[strain](/ddbj/qualifiers-e.html#strain),
[isolate](/ddbj/qualifiers-e.html#isolate) etc. are required for the
source feature.

``` code flat-file
                   /organism="Acetobacter sp."
                   /strain="ITDI2.1"
```

In case of whole genomic scale sequences from microorganisms, or in case
of eukaryote sequences (except unicellular fungi), a strain name or some
other lower taxon is required for the organism name.  
The tentative name is made up with the lineage (in many cases, genus
names) that as far as submitters could specify and the lower taxon (in
many cases, strain names).  
Also, describe a strain name in the
[strain](/ddbj/qualifiers-e.html#strain) qualifier.

``` code flat-file
                    /organism="Acetobacter sp. ITDI2.1"
                    /strain="ITDI2.1"
```

#### 2-2. In case of proposing a novel species

If you are to propose a novel species, in the mean time, we adopt an
unique tentative name for the organism. The tentative name is made up
with the lineage (in many cases, genus names) that as far as submitters
could specify and the lower taxon (in many cases, strain names). Because
we have to avoid some confusions; for example, two different organisms
are mixed up.

Format

``` code
"<genus name> sp. <strain name>"
"<family (or upper) name> bacterium <strain name>"
```

Example

  - Acetobacter sp. ITDI2.1
  - Acetobacteraceae bacterium ITDI2.1

Also, the qualifiers corresponding to the lower rank such as
[strain](/ddbj/qualifiers-e.html#strain) etc. are required for the
source feature.

``` code flat-file
                   /organism="Acetobacter sp. ITDI2.1"
                   /strain="ITDI2.1"
```

If the name is not available on taxonomy database
([TXSearch](http://ddbj.nig.ac.jp/tx_search/?lang=en)), please tell us
any of following items during your sequence submission.

Useful items for application of organism names to taxonomy database

  - [taxonomic lineage](#lineage)
  - [proposing name for novel species](/ddbj/organism-e.html#propose)
  - [already issued accession number](#acc)
  - [process of sampling and/or sequencing](#isolate)

### 3\. Environmental samples

Environmental samples are sequences derived by direct molecular
isolation from a bulk environmental DNA sample (by PCR, DGGE, or other
anonymous methods) with no reliable identification of the source
organism.  
Though frequently confused, <span class="red">the term, 'environmental
samples', does not mean "wild type".</span>  
Please refer to [environmental samples](/ddbj/env-e.html) in detail.  
<span class="red">Mixed culture derived from an environmental sample is
also processed as a kind of environmental samples. </span>

For environmental sample, we assign the lineage that as far as
submitters could specify is used for the description of organism name
with the header "uncultured".

Format

``` code
"uncultured <genus name> sp."
  "uncultured <family (or upper) name>" or "uncultured <family (or upper) name> bacterium"
```

Examples

  - uncultured Acetobacter sp.
  - uncultured alpha proteobacterium
  - uncultured Bacillaceae bacterium

In cases of environmental samples, the qualifier,
[environmental\_sample](/ddbj/qualifiers-e.html#environmental_sample),
is required for source feature. Also,
[isolation\_source](/ddbj/qualifiers-e.html#isolation_source) and some
other qualifiers should be used to describe the process and conditions
of sample isolation.

``` code flat-file
                   /clone="4-11"
                   /environmental_sample
                   /isolation_source="PCR-derived sequence from sediment"
                   /mol_type="genomic DNA"
                   /organism="uncultured Acetobacter sp."
```

If the name is not available on taxonomy database
([TXSearch](http://ddbj.nig.ac.jp/tx_search/?lang=en)), please tell us
any of following items during your sequence submission.

Useful items for application of organism names to taxonomy database

  - [taxonomic lineage](#lineage)
  - [process of sampling and/or sequencing](#isolate)

### 4\. Artificially constructed sequences

In many cases, artificially constructed sequences are uniformly named
"synthetic construct", or, "eukaryotic synthetic construct". Sometimes,
vector names or something like that are described 'as is' in the
organism name.

Examples

  - Cloning vector pAP3neo
  - Expression vector pAMP

If the name is not available on taxonomy database
([TXSearch](http://ddbj.nig.ac.jp/tx_search/?lang=en)), please tell us
any of following items during your sequence submission.

Useful items for application of organism names to taxonomy database

  - [expected usage](/ddbj/organism-e.html#usage)

</div>

<div class="section chapter">

## Useful items for application of organism names to taxonomy database

If the organism name that you submit with your sequence data is not in
the database, it must be newly registered to the taxonomy database
through DDBJ.

  - taxonomic lineage  
    Please tell us the taxonomic lineage of the organism from which your
    sequence has been obtained, as far as inferable.
  - valid publication for species  
    If the scientific name of species has been publicized in papers,
    please tell us the references for the species.
  - proposing name for novel species  
    If it is really novel species and not yet published, please tell us
    proposing name for the novel species for the tracking purpose at
    taxonomy database. In addition, please send E-mail to DDBJ update,
    ![](/images/ddbj/ad_updt1.gif "ad_updt"), when it is published or
    particularly if the name changes.
  - already issued accession number  
    If you have already submitted other sequence data derived from the
    same organism of present submission and the previous data has not
    yet been published, please tell us the accession number(s) of your
    previous data.
  - process of sampling and/or sequencing  
    It would be helpful if you tell us some information such as process
    of sampling, sequencing and so on.
  - expected usage  
    In cases of artificial constructed sequences, please tell us how to
    use them.

</div>

</div>

</div>

</div>

</div>

<div id="side_navigation">

<div class="side_nav_menu handbook_contents handbook organism">

<div id="in_this_page">

## In this page

</div>

</div>

<div class="side_nav_menu handbook_related">

<div id="related_pages">

## Related pages

  - [DDBJ flat file format](/ddbj/flat-file-e.html)
  - [Feature Key](/ddbj/features-e.html)
  - [Qualifier key](/ddbj/qualifiers-e.html)
  - [Description of Location](/ddbj/location-e.html)
  - [Protein Coding Sequence; CDS feature](/ddbj/cds-e.html)
  - [The Genetic Codes](/ddbj/geneticcode-e.html)
  - [Codes Used in Sequence Description](/ddbj/code-e.html)
  - [Example of Submission](/ddbj/example-e.html)

</div>

</div>

</div>

</div>

<div id="footer_original" class="clearfix">

  - [Policies and
    Disclaimers](/policies-e.html "Policies and Disclaimers")
  - [News](/news/en/index-e.html "News archives")
  - [FAQs](/faq/en/index-e.html "Frequently asked questions")
  - [Sitemap](/sitemap-e.html "Sitemap")
  - [Calendar](/calendar-e.html "DDBJ Center Calendar")
  - [Address](/access-e.html "DDBJ Center address")
  - [Contact](/contact-e.html "Contact to DDBJ Center")
  - Last modified: 2019-02-19

</div>

</div>
