---
layout: indexed_content
title: Qualifier key
pathname: qualifiers-e
category: flatfile
---

<div id="page" class="top">

<div id="header" class="clearfix">

<div id="header-top">

[![](/images/parts/ddbj-logo-white.svg)](/index-e.html "DNA Data Bank of Japan")

<div id="header-top-left">

<span class="resources-btn">Services**</span>

</div>

<div id="header-top-right">

  - [Login & Submit](https://ddbj.nig.ac.jp/D-way)
  - [Policies and Disclaimers](/policies-e.html)
  - [Contact](/contact-e.html)
  - [Japanese](/ddbj/qualifiers.html)

</div>

</div>

<div class="resources-contents">

<div class="boxContainer">

<div class="resources-box">

  - ### ![](/images/logos/service-org.svg)Search

  - [getentry](http://getentry.ddbj.nig.ac.jp/top-e.html "Retrieve sequence records by accession numbers")

  - [ARSA](http://ddbj.nig.ac.jp/arsa/ "Retrieve sequence and annotation by keywords")

  - [DRA
    Search](http://ddbj.nig.ac.jp/DRASearch "Search and retrieve sequencing data.")

  - [TXSearch](http://ddbj.nig.ac.jp/tx_search/?lang=en "Search unified taxonomy database")

  - [BLAST](http://blast.ddbj.nig.ac.jp/blastn?lang=en "Search nucleotide sequences by sequence similarity")

<!-- end list -->

  - ### ![](/images/logos/wrench.svg)Analysis

  - [Vector Screening
    System](http://ddbj.nig.ac.jp/vecscreen/ "Screen vector sequences")

  - [ClustalW](http://clustalw.ddbj.nig.ac.jp/index.php?lang=en "Multiple alignment and tree-making")

  - [WABI (Web API for
    Biology)](/search/help/wabi/wabi-e.html "Web API for using the search services of DDB")

  - [DDBJ FTP
    Site](ftp://ftp.ddbj.nig.ac.jp/ "Download public data by ftp")

</div>

<div class="resources-box">

  - ### ![](/images/logos/database-org.svg)Databases

  - [Annotated/Assembled Sequences
    (DDBJ)](/ddbj/index-e.html "Annotated/Assembled Sequences (DDBJ)")

  - [Sequence Read Archive
    (DRA)](/dra/index-e.html "Sequence Read Archive")

  - [Genomic Expression Archive
    (GEA)](/gea/index-e.html "Genomic Expression Archive")

  - [BioProject](/bioproject/index-e.html "BioProject")

  - [BioSample](/biosample/index-e.html "BioSample")

  - [Japanese Genotype-phenotype Archive
    (JGA)](/jga/index-e.html "Japanese Genotype-phenotype Archive")

  - [Submission portal
    D-way](https://ddbj.nig.ac.jp/D-way "Submission portal D-way")

<!-- end list -->

  - ### ![](/images/logos/sc-org.svg)NIG SuperComputer

  - [NIG
    SuperComputer](https://sc.ddbj.nig.ac.jp/en "NIG SuperComputer")

</div>

<div class="resources-box dbcls">

  - ### ![](/images/logos/dbcls-s.svg)DBCLS Services

  - [AOE](http://aoe.dbcls.jp/en "Statistics and trends of gene expression data")

  - [CRISPRdirect](https://crispr.dbcls.jp/ "Designing CRISPR/Cas9 guide RNA with reduced off-target sites")

  - [DBCLS
    SRA](http://sra.dbcls.jp/ "Statistics and trends of SRA data")

  - [Gendoo](http://gendoo.dbcls.jp/ "Functional profiling of gene and disease features for omics analysis")

  - [GGGenome](https://gggenome.dbcls.jp/en/ "A ultrafast sequence search")

  - [GGRNA](https://ggrna.dbcls.jp/en/ "A Google-like, ultrafast search engine for genes and transcripts")

  - [RefEx](http://refex.dbcls.jp/index.php?lang=en "A web tool for a comfortable search of reference data for gene expression analysis")

</div>

</div>

</div>

<div id="header-middle" class="clearfix">

<div class="page-title">

[DDBJ Annotated/Assembled Sequences](/ddbj/index-e.html)

</div>

<div class="search-box">

<div id="search-box-flex">

<div class="search-options-wrapper">

BI-DDBJ Web Sites Annotated/Assembled Sequences Sequence Read Archive

</div>

</div>

</div>

</div>

<div id="header-bottom">

  - [Home](/ddbj/index-e.html)
  - [Submission![](/images/parts/arrow_down.svg)](/ddbj/submission-e.html)
      - [Before Submission](submission-e.html)
      - [Web submission](websub-e.html)
      - [Mass submission](mss-e.html)
      - [Data Update](updt-e.html)
  - [Search![](/images/parts/arrow_down.svg)](http://ddbj.nig.ac.jp/arsa/?lang=en)
      - [getentry](http://getentry.ddbj.nig.ac.jp/top-e.html)
      - [ARSA](http://ddbj.nig.ac.jp/arsa/?lang=en)
  - [Flat File![](/images/parts/arrow_down.svg)](/ddbj/flat-file-e.html)
      - [Feature key](/ddbj/features-e.html)
      - [Qualifier key](/ddbj/qualifiers-e.html)
      - [Organism qualifier](/ddbj/organism-e.html)
      - [Description of Location](/ddbj/location-e.html)
      - [Protein Coding Sequence](/ddbj/cds-e.html)
      - [The Genetic Codes](/ddbj/geneticcode-e.html)
      - [Codes Used in Sequence Description](/ddbj/code-e.html)
      - [Example of Submission](/ddbj/example-e.html)
  - [Data
    Categories![](/images/parts/arrow_down.svg)](/ddbj/data-categories-e.html)
      - [Data Submission from Genome Project](/ddbj/genome-e.html)
      - [WGS](/ddbj/wgs-e.html)
      - [CON](/ddbj/con-e.html)
      - [GSS](/ddbj/gss-e.html)
      - [HTG](/ddbj/htg-e.html)
      - [Submission of environmental
        sequences](/ddbj/environmental-e.html)
      - [ENV](/ddbj/env-e.html)
      - [TLS](/ddbj/tls-e.html)
      - [Data Submission from Transcriptome
        Project](/ddbj/transcriptome-e.html)
      - [TSA](/ddbj/tsa-e.html)
      - [EST](/ddbj/est-e.html)
      - [HTC](/ddbj/htc-e.html)
      - [Third Party Data (TPA)](/ddbj/tpa-e.html)
  - [FAQ](/ddbj/faq-e.html)
  - [Other![](/images/parts/arrow_down.svg)](/)
      - [Patent](/ddbj/pat-data-e.html)
      - [MGA](/ddbj/mga-e.html)

</div>

</div>

<div id="main" class="clearfix">

<div id="breadcrumb" class="clearfix">

  - [HOME](/ddbj/index-e.html "DDBJ Annotated/Assembled Sequences top page")
  - \>
  - [DDBJ flat file format](/ddbj/flat-file-e.html)
  - \>
  - Qualifier key

</div>

<div id="primary">

<div id="page_main">

# Qualifier key

<div id="description_container">

<div id="description">

The qualifier keys that are used and recommended for DDBJ new
submissions are as follows.  
For further information of INSDC qualifier keys, read [Feature Table
Definition](/ddbj/full_index-e.html): [7.3.1 Qualifier
List](/ddbj/full_index-e.html#7.3.1).

## Feature/Qualifier Usage Matrix

The chart below explains recommended combinations of feature and
qualifier keys for DDBJ submissions.

[Feature/Qualifier usage matrix ![](/images/parts/pdf.png
"pdficon")](https://drive.google.com/open?id=1b4TRlgUIyMi9f_CryLLTiBkgBn0lnNF7)
(December 20, 2018)

For more detail of available combinations of feature and qualifier keys
in INSDC entries, read: [Feature Table
Definition](/ddbj/full_index-e.html): [7.2 Appendix II: Feature keys
reference](/ddbj/full_index-e.html#7.2).

## Definision of Qualifier key

  - allele<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#allele)</span>
    
    |              |                                               |
    | ------------ | --------------------------------------------- |
    | Definition   | name of the allele for the given gene.        |
    | Value format | \<text\>, excluding double quotation mark (") |
    | Example      | adh1-1                                        |
    

  - altitude<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#altitude)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>geographical altitude of the location from which the sample was collected</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>&lt;text&gt;, excluding double quotation mark (")</td>
    </tr>
    <tr class="odd">
    <td>Example</td>
    <td>-256 m<br />
    330.12 m</td>
    </tr>
    <tr class="even">
    <td>Comment</td>
    <td>Values indicate altitudes above or below nominal sea level provided in metres</td>
    </tr>
    </tbody>
    </table>

  - anticodon<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#anticodon)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>location of the anticodon of <a href="/ddbj/features-e.html#tRNA">tRNA</a> and the amino acid for which it codes</td>
    </tr>
    <tr class="even">
    <td>Value format for input:</td>
    <td>(pos:&lt;location&gt;,aa:&lt;amino_acid&gt;)<br />
    where location is the position of the anticodon and amino_acid is the abbreviation either for the abbreviation for <a href="/ddbj/code-e.html#amino-1">Amino Acid Codes</a>, or for <a href="/ddbj/code-e.html#amino-2">Modified and unusual Amino Acids</a>.</td>
    </tr>
    <tr class="odd">
    <td>Example for input:</td>
    <td>(pos:34..36,aa:Phe)<br />
    (pos:join(5,495..496),aa:Leu)<br />
    (pos:complement(4156..4158),aa:Gln)</td>
    </tr>
    <tr class="even">
    <td>Value format for output:</td>
    <td>(pos:&lt;location&gt;,aa:&lt;amino_acid&gt;,seq:&lt;nucleotides&gt;)</td>
    </tr>
    <tr class="odd">
    <td>Example for output:</td>
    <td>(pos:34..36,aa:Phe,seq:aaa)<br />
    (pos:join(5,495..496),aa:Leu,seq:tag)<br />
    (pos:complement(4156..4158),aa:Gln,seq:ttg)</td>
    </tr>
    </tbody>
    </table>

  - artificial\_location<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#artificial_location)</span>
    
    |              |                                                                                                                                                                                                                                                                            |
    | ------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
    | Definition   | indicates that [location](/ddbj/location-e.html) of the [CDS](/ddbj/features-e.html#cds) or [mRNA](/ddbj/features-e.html#mRNA) is modified to adjust for the presence of a frameshift or internal stop codon and not because of biological processing between the regions. |
    | Value format | "heterogeneous population sequenced" or "low-quality sequence region"                                                                                                                                                                                                      |
    | Comment      | expected to be used only for genome-scale annotation, either because a heterogeneous population was sequenced or because the feature is in a region of low-quality sequence.                                                                                               |
    

  - bio\_material<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#bio_material)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>identifier for the biological material (living individual or strain) from which the nucleic acid sequence was obtained, with optional institution code and collection code for the place where it is currently stored.</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>[&lt;institution_code&gt;:[&lt;collection_code&gt;:]]&lt;material_id&gt;</td>
    </tr>
    <tr class="odd">
    <td>Example</td>
    <td>CGC:CB3912      &lt;- Caenorhabditis stock centre</td>
    </tr>
    <tr class="even">
    <td>Comment</td>
    <td>the bio_material qualifier should be used to annotate the identifiers of material in biological collections that are not appropriate to annotate as either /specimen_voucher or /culture_collection; these include zoos and aquaria, stock centres, seed banks, germplasm repositories and DNA banks; material_id is mandatory, institution_code and collection_code are optional; institution code is mandatory where collection code is present.<br />
    <br />
    You can find &lt;institution_code&gt; at<br />
    <a href="ftp://ftp.ncbi.nlm.nih.gov/pub/taxonomy/coll_dump.txt">institution_code list</a> (NCBI FTP site)<br />
    <a href="https://www.ncbi.nlm.nih.gov/biocollections">BioCollections</a></td>
    </tr>
    </tbody>
    </table>

  - bound\_moiety<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#bound_moiety)</span>
    
    |              |                                                                 |
    | ------------ | --------------------------------------------------------------- |
    | Definition   | name of the molecule/complex that may bind to the given feature |
    | Value format | \<text\>, excluding double quotation mark (")                   |
    | Example      | GAL4                                                            |
    

  - cell\_line<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#cell_line)</span>
    
    |              |                                                |
    | ------------ | ---------------------------------------------- |
    | Definition   | cell line from which the sequence was obtained |
    | Value format | \<text\>, excluding double quotation mark (")  |
    | Example      | MCF7                                           |
    

  - cell\_type<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#cell_type)</span>
    
    |              |                                                |
    | ------------ | ---------------------------------------------- |
    | Definition   | cell type from which the sequence was obtained |
    | Value format | \<text\>, excluding double quotation mark (")  |
    | Example      | leukocyte                                      |
    

  - chromosome<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#chromosome)</span>
    
    |              |                                                                          |
    | ------------ | ------------------------------------------------------------------------ |
    | Definition   | chromosome (e.g. Chromosome number) from which the sequence was obtained |
    | Value format | \<text\>, excluding double quotation mark (")                            |
    | Example      | 1                                                                        |
    

  - clone<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#clone)</span>
    
    |              |                                               |
    | ------------ | --------------------------------------------- |
    | Definition   | clone from which the sequence was obtained    |
    | Value format | \<text\>, excluding double quotation mark (") |
    | Example      | lambda-hIL7.3                                 |
    

  - clone\_lib<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#clone_lib)</span>
    
    |              |                                                    |
    | ------------ | -------------------------------------------------- |
    | Definition   | clone library from which the sequence was obtained |
    | Value format | \<text\>, excluding double quotation mark (")      |
    | Example      | lambda-hIL7                                        |
    

  - codon\_start<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#codon_start)</span>
    
    |              |                                                                                                                                                                |
    | ------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------- |
    | Definition   | indicates [the offset at which the first complete codon](/ddbj/cds-e.html#frame) of a coding feature can be found, relative to the first base of that feature. |
    | Value format | 1, 2 or 3                                                                                                                                                      |
    

  - collected\_by<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#collected_by)</span>
    
    |              |                                                         |
    | ------------ | ------------------------------------------------------- |
    | Definition   | name of persons or institute who collected the specimen |
    | Value format | \<text\>, excluding double quotation mark (")           |
    | Example      | Dan Janzen                                              |
    

  - collection\_date<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#collection_date)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>The date on which the specimen was collected.<br />
    Date/time ranges are supported by providing two collection dates from among the supported value formats, delimited by a forward-slash character.<br />
    Collection times are supported by adding "T", then the hour and minute, after the date.<br />
    Collection times must be in Coordinated Universal Time (UTC), otherwise known as "Zulu Time" (Z).</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>YYYY-MM-DDThh:mm:ssZ<br />
    YYYY-MM-DDThh:mmZ<br />
    YYYY-MM-DDThhZ<br />
    YYYY-MM-DD<br />
    YYYY-MM<br />
    YYYY<br />
    <br />
    YYYY/YYYY<br />
    YYYY-MM-DD/YYYY-MM-DD<br />
    YYYY-MM/YYYY-MM<br />
    YYYY-MM-DDThh:mmZ/YYYY-MM-DDThh:mmZ<br />
    <br />
    'YYYY' is a four-digit value representing the year.<br />
    'MM' is a two-digit value representing the month (01 to 12) .<br />
    'DD' is a two-digit value representing the day of the month (01 to 31) .<br />
    'hh' is a two-digit value representing the hour of the day (00 to 23).<br />
    'mm' is a two-digit value representing the minute of the hour (00 to 59).<br />
    'ss' is a two-digit value representing the second of the hour (00 to 59).</td>
    </tr>
    <tr class="odd">
    <td>Example</td>
    <td>2015-10-11T17:53:03Z<br />
    1952-10-21T11:43Z<br />
    1952-10-21T11Z<br />
    1952-10-21<br />
    1952-10<br />
    1952<br />
    <br />
    1952/1953<br />
    1952-10-21/1953-02-15<br />
    1952-10/1953-02<br />
    1952-10-21T11:43Z/1952-10-21T17:43Z</td>
    </tr>
    <tr class="even">
    <td>Comment</td>
    <td>Collection dates that are specified to at least the month, day, and year (YYYY-MM-DD) are strongly encouraged.<br />
    <br />
    Though INSDC still keep and accept old value formats that make use of 'Mmm' (month abbreviations), such as "21-Oct-1952", DDBJ no longer accepts new data submissions with old value formats of collection_date.</td>
    </tr>
    </tbody>
    </table>

  - compare<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#compare)</span>
    
    |              |                                                                                   |
    | ------------ | --------------------------------------------------------------------------------- |
    | Definition   | Reference details of an existing public INSD entry to which a comparison is made. |
    | Value format | \<accession number\>.\<version\>                                                  |
    | Example      | AB123456.1                                                                        |
    

  - country<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#country)</span>
    
    |              |                                                                                                                                                       |
    | ------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------- |
    | Definition   | locality of isolation of the sequenced organism indicated in terms of political names for nations, oceans or seas, followed by regions and localities |
    | Value format | \<country\>\[:\<free-text for geographical name\>\]                                                                                                   |
    | Example      | Japan:Kanagawa, Hakone, Lake Ashi                                                                                                                     |
    | Comment      | any \<country\> from [the country list](/ddbj/country-e.html).                                                                                        |
    

  - cultivar<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#cultivar)</span>
    
    |              |                                                                          |
    | ------------ | ------------------------------------------------------------------------ |
    | Definition   | cultivar (cultivated variety) of plant from which sequence was obtained. |
    | Value format | \<text\>, excluding double quotation mark (")                            |
    | Example      | Nipponbare                                                               |
    

  - culture\_collection<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#culture_collection)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>institution_code and identifier for the culture from which the nucleic acid sequence was obtained, with optional collection code.</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>&lt;institution_code&gt;:[&lt;collection_code&gt;:]&lt;culture_id&gt;</td>
    </tr>
    <tr class="odd">
    <td>Example</td>
    <td>ATCC:26370</td>
    </tr>
    <tr class="even">
    <td>Comment</td>
    <td>the culture_collection qualifier should be used to annotate live microbial and viral cultures, and cell lines that have been deposited in curated culture collections; microbial cultures in personal or laboratory collections should be annotated in strain qualifiers.<br />
    culture_id and institution_code are mandatory, collection_code is optional.<br />
    <br />
    You can find &lt;institution_code&gt; at<br />
    <a href="ftp://ftp.ncbi.nlm.nih.gov/pub/taxonomy/coll_dump.txt">institution_code list</a> (NCBI FTP site)<br />
    <a href="https://www.ncbi.nlm.nih.gov/biocollections">BioCollections</a></td>
    </tr>
    </tbody>
    </table>

  - db\_xref<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#db_xref)</span>
    
    <table>
    <colgroup>
    <col style="width: 33%" />
    <col style="width: 33%" />
    <col style="width: 33%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>database cross-reference: pointer to related information in another database.
    Value format
    Value format</td>
    <td>&lt;database&gt;:&lt;identifier&gt;, excluding double quotation mark (")</td>
    </tr>
    <tr class="even">
    <td>Example</td>
    <td>UniProtKB/Swiss-Prot:P28763</td>
    <td></td>
    </tr>
    <tr class="odd">
    <td>Comment</td>
    <td>In principle, the db_xref qualifier can not be entered in new submissions.<br />
    When you referred records in other database as evidence for annotation, use <a href="#inference">inference</a>, not db_xref.<br />
    The controlled values of &lt;database&gt; are in <a href="/ddbj/db_xref-e.html">the database list</a>.</td>
    <td></td>
    </tr>
    </tbody>
    </table>

  - dev\_stage<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#dev_stage)</span>
    
    |              |                                                                                                                      |
    | ------------ | -------------------------------------------------------------------------------------------------------------------- |
    | Definition   | if the sequence was obtained from an organism in a specific developmental stage, it is specified with this qualifier |
    | Value format | \<text\>, excluding double quotation mark (")                                                                        |
    | Example      | fourth instar larva                                                                                                  |
    

  - direction<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#direction)</span>
    
    |              |                                                                                                                    |
    | ------------ | ------------------------------------------------------------------------------------------------------------------ |
    | Definition   | direction of DNA replication                                                                                       |
    | Value format | left, right, or both                                                                                               |
    | Comment      | Where left indicates toward the 5' end of the entry sequence (as presented) and right indicates toward the 3' end. |
    

  - EC\_number<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#EC_number)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>Enzyme Commission number for enzyme product of sequence</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>&lt;identifier&gt;.&lt;identifier&gt;.&lt;identifier&gt;.&lt;identifier&gt;</td>
    </tr>
    <tr class="odd">
    <td>Example</td>
    <td>1.1.2.4<br />
    1.1.2.-<br />
    1.1.2.n</td>
    </tr>
    <tr class="even">
    <td>Comment</td>
    <td>The format represents a string of four numbers separated by full stops; up to three numbers starting from the end of the string can be replaced by dash "-" to indicate uncertain assignment. Symbol "n" can be used in the last position instead of a number where the EC number is awaiting assignment.</td>
    </tr>
    </tbody>
    </table>

  - ecotype<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#ecotype)</span>
    
    |              |                                                                                                                                 |
    | ------------ | ------------------------------------------------------------------------------------------------------------------------------- |
    | Definition   | a population within a given species displaying genetically based, phenotypic traits that reflect adaptation to a local habitat. |
    | Value format | \<text\>, excluding double quotation mark (")                                                                                   |
    | Example      | Columbia                                                                                                                        |
    

  - environmental\_sample<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#environmental_sample)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>identifies sequences derived by direct molecular isolation from a bulk environmental DNA sample (by PCR with or without subsequent cloning of the product, DGGE, or other anonymous methods) with no reliable identification of the source organism.<br />
    See also <a href="/ddbj/env-e.html">environmental samples</a> in detail.</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>no value.</td>
    </tr>
    <tr class="odd">
    <td>Comment</td>
    <td>source feature keys containing the /environmental_sample qualifier should also contain the <a href="#isolation_source">/isolation_source</a> qualifier.<br />
    entries including /environmental_sample must not include the <a href="#strain">/strain</a> qualifier</td>
    </tr>
    </tbody>
    </table>

  - estimated\_length<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#estimated_length)</span>  
    Definition

  - exception<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#exception)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>indicates that the amino acid or RNA sequence will not translate or agree with the DNA sequence according to standard biological rules.</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>one of followings;
    <ul>
    <li>RNA editing</li>
    <li>reasons given in citation</li>
    <li>rearrangement required for product</li>
    <li>annotated by transcript or proteomic data</li>
    </ul></td>
    </tr>
    <tr class="odd">
    <td>Comment</td>
    <td>An <a href="#inference">/inference qualifier</a> should accompany any use of /exception="annotated by transcript or proteomic data", to provide support for the existence of the transcript/protein.</td>
    </tr>
    </tbody>
    </table>

  - experiment<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#experiment)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>a brief description of the nature of the experimental evidence that supports the feature identification or assignment.</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>[CATEGORY:]&lt;text&gt;, excluding double quotation mark (")<br />
    CATEGORY is optional. if describing it, use either of followings;
    <ul>
    <li>COORDINATES</li>
    <li>DESCRIPTION</li>
    <li>EXISTENCE</li>
    </ul></td>
    </tr>
    <tr class="odd">
    <td>Example</td>
    <td>COORDINATES: 5' and 3' RACE<br />
    Northern blot<br />
    heterologous expression system of Xenopus laevis oocytes.</td>
    </tr>
    <tr class="even">
    <td>Comment</td>
    <td>detailed experimental details should not be included, and would normally be found in the cited publications; value<br />
    "experimental evidence, no additional details recorded"<br />
    was used to replace instances of /evidence=EXPERIMENTAL in December 2005</td>
    </tr>
    </tbody>
    </table>

  - focus<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#focus)</span>
    
    |              |                                                                                                                                                                               |
    | ------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
    | Definition   | identifies the source feature of primary biological interest for records that have multiple source features originating from different organisms and that are not transgenic. |
    | Value format | none                                                                                                                                                                          |
    

  - frequency<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#frequency)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>frequency of the occurrence of a feature</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>&lt;number of observed instances&gt; in &lt;total number of sequenced isolates&gt;,<br />
    &lt;number of observed instances&gt;/&lt;total number of sequenced isolates&gt;, or &lt;decimal fraction&gt;</td>
    </tr>
    <tr class="odd">
    <td>Example</td>
    <td>1 in 12<br />
    23/108<br />
    .85</td>
    </tr>
    </tbody>
    </table>

  - function<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#function)</span>
    
    |              |                                               |
    | ------------ | --------------------------------------------- |
    | Definition   | function attributed to a sequence             |
    | Value format | \<text\>, excluding double quotation mark (") |
    | Example      | essential for recognition of cofactor         |
    

  - gap\_type<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#gap_type)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>type of gap connecting components in records of a genome assembly, or the type of biological gap in a record that is part of a genome assembly</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>one of the followings
    <ul>
    <li>between scaffolds</li>
    <li>within scaffold</li>
    <li>telomere</li>
    <li>centromere</li>
    <li>short arm</li>
    <li>heterochromatin</li>
    <li>repeat within scaffold</li>
    <li>repeat between scaffolds</li>
    <li>unknown</li>
    </ul></td>
    </tr>
    <tr class="odd">
    <td>Comment</td>
    <td>This qualifier is used only for <a href="/ddbj/features-e.html#assembly_gap">assembly_gap</a> features and its values are controlled by <a href="https://www.ncbi.nlm.nih.gov/assembly/agp/AGP_Specification/">the AGP Specification</a></td>
    </tr>
    </tbody>
    </table>

  - gene<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#q_gene)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>symbol of the gene corresponding to a sequence region</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>&lt;text&gt;, excluding double quotation mark (")</td>
    </tr>
    <tr class="odd">
    <td>Example</td>
    <td>ilvE</td>
    </tr>
    <tr class="even">
    <td>Guidance for Submission:</td>
    <td>See also <a href="/ddbj/cds-e.html#product">Gene nomenclature at DDBJ</a>.
    <ul>
    <li>Please enter the abbreviation as gene symbol.</li>
    <li>Even if there are multiple general abbreviations for the same gene, do not enter multiple abbreviations in 'gene'. Do not use needless symbolic letters as delimiter for multiple names. If you would like to describe more than two, please enter one of the most representative abbreviation in 'gene', and other(s) in <a href="#gene_synonym">gene_synonym</a> qualifier.</li>
    </ul></td>
    </tr>
    </tbody>
    </table>

  - gene\_synonym<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#gene_synonym)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>symbol of the gene corresponding to a sequence region, synonym for the value used for gene or locus_tag qualifier</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>&lt;text&gt;, excluding double quotation mark (")</td>
    </tr>
    <tr class="odd">
    <td>Example</td>
    <td>ilvE</td>
    </tr>
    <tr class="even">
    <td>Guidance for Submission:</td>
    <td>See also <a href="/ddbj/cds-e.html#product">Gene nomenclature at DDBJ</a>.
    <ul>
    <li>Please enter the abbreviation as gene symbol.</li>
    <li>Even if there are multiple general abbreviations for the same gene, do not enter multiple abbreviations in 'gene'. Do not use needless symbolic letters as delimiter for multiple names. If you would like to describe more than two, please enter one of the most representative abbreviation in '<a href="#gene">gene</a>', and other(s) in gene_synonym qualifier.</li>
    </ul></td>
    </tr>
    </tbody>
    </table>

  - germline<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#germline)</span>
    
    |              |                                                                                                                                                                                                    |
    | ------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
    | Definition   | the sequence presented in the entry has not undergone somatic rearrangement as part of an adaptive immune response; it is the unrearranged sequence that was inherited from the parental germline. |
    | Value format | none                                                                                                                                                                                               |
    | Comment      | Do not use with [rearranged](#rearranged) qualifier.                                                                                                                                               |
    

  - haplogroup<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#haplogroup)</span>
    
    |              |                                                                                                                                                |
    | ------------ | ---------------------------------------------------------------------------------------------------------------------------------------------- |
    | Definition   | name for a group of similar haplotypes that share some sequence variation. Haplogroups are often used to track migration of population groups. |
    | Value format | \<text\>, excluding double quotation mark (")                                                                                                  |
    | Example      | H\*                                                                                                                                            |
    

  - haplotype<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#haplotype)</span>
    
    |              |                                                                                                                                                                                                                        |
    | ------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
    | Definition   | name for a combination of alleles that are linked together on the same physical chromosome. In the absence of recombination, each haplotype is inherited as a unit, and may be used to track gene flow in populations. |
    | Value format | \<text\>, excluding double quotation mark (")                                                                                                                                                                          |
    | Example      | Dw3 B5 Cw1 A1                                                                                                                                                                                                          |
    

  - host<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#host)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>Natural (as opposed to laboratory) host to the organism from which sequenced molecule was obtained</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>&lt;text&gt;, excluding double quotation mark (")</td>
    </tr>
    <tr class="odd">
    <td>Example</td>
    <td>Homo sapiens<br />
    Homo sapiens 12 years old girl</td>
    </tr>
    </tbody>
    </table>

  - identified\_by<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#identified_by)</span>
    
    |              |                                                              |
    | ------------ | ------------------------------------------------------------ |
    | Definition   | name of the expert who identified the specimen taxonomically |
    | Value format | \<text\>, excluding double quotation mark (")                |
    | Example      | John Burns                                                   |
    

  - inference<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#inference)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>a structured description of non-experimental evidence that supports the feature identification or assignment.</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>[CATEGORY:]TYPE[ (same species)][:EVIDENCE_BASIS]<br />
    CATEGORY is optional. if describing it, use either of followings;
    <ul>
    <li>COORDINATES</li>
    <li>DESCRIPTION</li>
    <li>EXISTENCE</li>
    </ul>
    <br />
     <br />
    where TYPE is one of the following
    <ul>
    <li>similar to sequence</li>
    <li>similar to AA sequence</li>
    <li>similar to DNA sequence</li>
    <li>similar to RNA sequence</li>
    <li>similar to RNA sequence, mRNA</li>
    <li>similar to RNA sequence, EST</li>
    <li>similar to RNA sequence, other RNA</li>
    <li>profile</li>
    <li>nucleotide motif</li>
    <li>protein motif</li>
    <li>ab initio prediction</li>
    <li>alignment</li>
    </ul></td>
    </tr>
    <tr class="odd">
    <td>Example</td>
    <td>similar to DNA sequence:INSD:AY411252.1<br />
    similar to RNA sequence, mRNA:RefSeq:NM_000041.2<br />
    similar to DNA sequence (same species):INSD:AACN010222672.1<br />
    profile:tRNAscan:2.1<br />
    protein motif:InterPro:IPR001900<br />
    ab initio prediction:Genscan:2.0<br />
    alignment:Splign:1.26p:RefSeq:NM_000041.2,INSD:BC003557.1</td>
    </tr>
    <tr class="even">
    <td>Comment</td>
    <td>where the optional text "(same species)" is included when the inference comes from the same species as the entry.<br />
    where the optional "EVIDENCE_BASIS" is either a reference to a database entry (including accession and version) or an algorithm (including version)<br />
    <a href="http://www.insdc.org/documents/inference-qualifiers">Recommendations for vocabulary in INSDC /inference qualifiers.</a><br />
    * /inference="non-experimental evidence, no additional details recorded" was used to replace instances of /evidence=NOT_EXPERIMENTAL in December 2005</td>
    </tr>
    </tbody>
    </table>

  - isolate<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#isolate)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>individual isolate from which the sequence was obtained</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>&lt;text&gt;, excluding double quotation mark (")</td>
    </tr>
    <tr class="odd">
    <td>Example</td>
    <td>SI-152<br />
    DGGE: C12</td>
    </tr>
    </tbody>
    </table>

  - isolation\_source<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#isolation_source)</span>
    
    |              |                                                                                                                                     |
    | ------------ | ----------------------------------------------------------------------------------------------------------------------------------- |
    | Definition   | describes the physical, environmental and/or local geographical source of the biological sample from which the sequence was derived |
    | Value format | \<text\>, excluding double quotation mark (")                                                                                       |
    | Example      | rumen isolates from standard Pelleted ration-fed steer \#67                                                                         |
    

  - lab\_host<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#lab_host)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>scientific name of the laboratory host used to propagate the source organism from which the sequenced molecule was obtained</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>&lt;text&gt;, excluding double quotation mark (")</td>
    </tr>
    <tr class="odd">
    <td>Example</td>
    <td>Gallus gallus<br />
    Gallus gallus embryo<br />
    Escherichia coli strain DH5 alpha<br />
    Homo sapiens HeLa cells<br />
    </td>
    </tr>
    </tbody>
    </table>

  - lat\_lon<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#lat_lon)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>geographical coordinates of the location where the sequenced sample was collected</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>d[d.dddd] &lt;N or S&gt; d[dd.dddd] &lt;W or E&gt;</td>
    </tr>
    <tr class="odd">
    <td>Example</td>
    <td>47.94 N 28.12 W<br />
    45.0123 S 4.1234 E<br />
    </td>
    </tr>
    <tr class="even">
    <td>Comment</td>
    <td>Please describe the figure below the decimal point by not minute and second but the decimal.</td>
    </tr>
    </tbody>
    </table>

  - linkage\_evidence<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#linkage_evidence)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>type of evidence establishing linkage across an <a href="/ddbj/features-e.html#assembly_gap">assembly_gap</a>. Only allowed to be used with <a href="/ddbj/features-e.html#assembly_gap">assembly_gap</a> features that have a /<a href="#gap_type">gap_type</a> value of "within scaffold" or "repeat within scaffold"</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>one of the followings
    <ul>
    <li>paired-ends</li>
    <li>pcr</li>
    <li>align genus</li>
    <li>align xgenus</li>
    <li>align trnscpt</li>
    <li>within clone</li>
    <li>clone contig</li>
    <li>map</li>
    <li>strobe</li>
    <li>unspecified</li>
    </ul></td>
    </tr>
    <tr class="odd">
    <td>Comment</td>
    <td>This qualifier is used only for <a href="/ddbj/features-e.html#assembly_gap">assembly_gap</a> features and its values are controlled by <a href="https://www.ncbi.nlm.nih.gov/assembly/agp/AGP_Specification/">the AGP Specification</a></td>
    </tr>
    </tbody>
    </table>

  - locus\_tag<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#locus_tag)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>a submitter-supplied (mainly genome project), systematic, stable identifier for a gene and its associated features, used for tracking purpose</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>&lt;text&gt;, excluding double quotation mark (")</td>
    </tr>
    <tr class="odd">
    <td>Example</td>
    <td>ABC_0022<br />
    A1C_00001</td>
    </tr>
    <tr class="even">
    <td>Comment</td>
    <td>identical /locus_tag values may be used within an entry/record, but only if the identical /locus_tag values are associated with the same gene; in all other circumstances the /locus_tag value must be unique within that entry/record.<br />
    INSDC requires <a href="/ddbj/locus_tag-e.html">prior registrations of the prefix for values of /locus_tag</a> to keep uniqueness of the /locus_tag value through the database</td>
    </tr>
    </tbody>
    </table>

  - macronuclear<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#macronuclear)</span>
    
    |              |                                                                                                                                                                                                                          |
    | ------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
    | Definition   | if the sequence shown is DNA and from an organism which undergoes chromosomal differentiation between macronuclear and micronuclear stages, this qualifier is used to denote that the sequence is from macronuclear DNA. |
    | Value format | none                                                                                                                                                                                                                     |
    

  - map<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#map)</span>
    
    |              |                                               |
    | ------------ | --------------------------------------------- |
    | Definition   | genomic map position of feature               |
    | Value format | \<text\>, excluding double quotation mark (") |
    | Example      | 8q12-13                                       |
    

  - mating\_type<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#mating_type)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>mating type of the organism from which the sequence was obtained. mating type is used for prokaryotes, and for eukaryotes that undergo meiosis without sexually dimorphic gametes (cf. <a href="#sex">sex</a>).</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>&lt;text&gt;, excluding double quotation mark (")</td>
    </tr>
    <tr class="odd">
    <td>Example</td>
    <td>MAT-1<br />
    plus<br />
    -<br />
    odd<br />
    even</td>
    </tr>
    </tbody>
    </table>

  - metagenome\_source<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#metagenome_source)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>sequences from a Metagenome Assembled Genome (MAG), i.e a single-taxon<br />
    assembly drawn from a binned metagenome, are specified with this qualifier to<br />
    indicate that the assembly is derived from a metagenomic source, rather than from<br />
    an isolated organism.</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>&lt;text&gt;, excluding double quotation mark (")<br />
    It must contain <a href="https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Undef&amp;id=408169&amp;lvl=3&amp;keep=1&amp;srchmode=1&amp;unlock">the word "metagenome" and must exist in the taxonomy database</a>.</td>
    </tr>
    <tr class="odd">
    <td>Example</td>
    <td>human gut metagenome<br />
    soil metagenome</td>
    </tr>
    <tr class="even">
    <td>Comment</td>
    <td>To use metagenome_source, <a href="environmental_sample">environmental_sample</a> is required.</td>
    </tr>
    </tbody>
    </table>

  - mobile\_element\_type<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#mobile_element_type)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>type and name or identifier of the mobile element which is described by the parent feature</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>&lt;mobile_element_type&gt; [:&lt;mobile_element_name&gt;] where mobile_element_type is one of the following;
    <ul>
    <li>transposon</li>
    <li>retrotransposon</li>
    <li>integron</li>
    <li>insertion sequence</li>
    <li>non-LTR retrotransposon</li>
    <li>SINE</li>
    <li>MITE</li>
    <li>LINE</li>
    <li>other</li>
    </ul></td>
    </tr>
    <tr class="odd">
    <td>Example</td>
    <td>transposon:Tnp9</td>
    </tr>
    </tbody>
    </table>

  - mod\_base<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#mod_base)</span>
    
    |              |                                                                                                                           |
    | ------------ | ------------------------------------------------------------------------------------------------------------------------- |
    | Definition   | abbreviation for a modified nucleotide base                                                                               |
    | Value format | modified\_base where modified\_base is the abbreviation for [Modified Base Abbreviation](/ddbj/code-e.html#nucleotide-2). |
    | Example      | m2g                                                                                                                       |
    

  - mol\_type<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#mol_type)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>describes the in vivo, synthetic or hypothetical molecule represented in sequence corresponding to the parent feature</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>limited to followings;<br />
    genomic DNA, genomic RNA, mRNA, tRNA, rRNA, transcribed RNA,<br />
    other RNA, other DNA ,viral cRNA, unassigned DNA, unassigned RNA</td>
    </tr>
    <tr class="odd">
    <td>Comment</td>
    <td>all values refer to the in vivo or synthetic molecule for primary entries and the hypothetical molecule in Third Party Annotation entries;
    <ul>
    <li>The value "genomic DNA" does not imply that the molecule is nuclear (e.g. organelle and plasmid DNAs should be described by using "genomic DNA").</li>
    <li>For ribosomal RNA genes (rDNA), select "genomic DNA".</li>
    <li>For cDNA sequence, template of mRNA, select "mRNA".</li>
    <li>For cDNA sequence, template of premature RNA, select "transcribed RNA".</li>
    <li>"other RNA" and "other DNA" should be applied to synthetic molecules.</li>
    <li>In general, select "genomic RNA" for RNA viruses.</li>
    <li>For ssRNA negative-strand virus, select "viral cRNA", in principal.<br />
    "viral cRNA" is a plus-strand copy of a minus strand RNA genome which serves as a template to make viral progeny genomes.<br />
    For genomic sequence data derived from ssRNA negative-strand viruses, in principle, DDBJ uniformly uses following values for mol_type qualifier;</li>
    </ul>
    Protein-coding sequences exist in positive orientation: viral cRNA Protein-coding sequences exist in complementary orientation: genomic RNA</td>
    </tr>
    </tbody>
    </table>

  - ncRNA\_class<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#ncRNA_class)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>the classification of the non-protein-coding RNA (<a href="/ddbj/features-e.html#ncRNA">ncRNA</a>)</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>&lt;TYPE&gt;</td>
    </tr>
    <tr class="odd">
    <td>Example</td>
    <td>miRNA<br />
    siRNA</td>
    </tr>
    <tr class="even">
    <td>Comment</td>
    <td><a href="http://www.insdc.org/documents/ncrna-vocabulary">Controlled vocabulary for ncRNA classes</a> is valid for &lt;TYPE&gt;. /ncRNA_class="other" with /product="&lt;name of novel ncRNA_class&gt;" or /note="&lt;brief explanation of novel ncRNA_class&gt;"</td>
    </tr>
    </tbody>
    </table>

  - note<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#note)</span>
    
    |              |                                               |
    | ------------ | --------------------------------------------- |
    | Definition   | any comment or additional information         |
    | Value format | \<text\>, excluding double quotation mark (") |
    

  - number<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#number)</span>
    
    |              |                                                                                                                                                                     |
    | ------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
    | Definition   | a number to indicate the order of genetic elements (e.g., [exon](/ddbj/features-e.html#exon)s or [intron](/ddbj/features-e.html#intron)s) in the 5' to 3' direction |
    | Value format | unquoted text (single token)                                                                                                                                        |
    | Example      | 5a                                                                                                                                                                  |
    

  - old\_locus\_tag<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#old_locus_tag)</span>
    
    |              |                                               |
    | ------------ | --------------------------------------------- |
    | Definition   | feature tag assigned for tracking purposes    |
    | Value format | \<text\>, excluding double quotation mark (") |
    | Example      | RSc0382                                       |
    

  - operon<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#q_operon)</span>
    
    |              |                                                                                                           |
    | ------------ | --------------------------------------------------------------------------------------------------------- |
    | Definition   | name of the group of contiguous genes transcribed into a single transcript to which that feature belongs. |
    | Value format | \<text\>, excluding double quotation mark (")                                                             |
    | Example      | lac                                                                                                       |
    

  - organelle<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#organelle)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>type of membrane-bound intracellular structure from which the sequence was obtained</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>limited to followings
    <ul>
    <li>mitochondrion</li>
    <li>mitochondrion:kinetoplast</li>
    <li>hydrogenosome</li>
    <li>plastid:chloroplast</li>
    <li>plastid:apicoplast</li>
    <li>plastid:chromoplast</li>
    <li>plastid:cyanelle</li>
    <li>plastid:leucoplast</li>
    <li>plastid:proplastid</li>
    <li>plastid</li>
    <li>chromatophore</li>
    <li>nucleomorph</li>
    </ul></td>
    </tr>
    </tbody>
    </table>

  - organism<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#organism)</span>
    
    |              |                                                                                                |
    | ------------ | ---------------------------------------------------------------------------------------------- |
    | Definition   | The scientific name of the organism that provided the sequenced genetic material.              |
    | Value format | \<text\>, excluding double quotation mark (")                                                  |
    | Example      | Homo sapiens                                                                                   |
    | Comment      | For further information of this Qualifier key read [Organism Qualifier](/ddbj/organism-e.html) |
    

  - PCR\_conditions<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#PCR_conditions)</span>
    
    |              |                                                           |
    | ------------ | --------------------------------------------------------- |
    | Definition   | description of reaction conditions and components for PCR |
    | Value format | \<text\>, excluding double quotation mark (")             |
    | Example      | Initial denaturation:94degC,1.5min                        |
    

  - PCR\_primers<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#PCR_primers)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>A single /PCR_primers qualifier should contain all the primers used for a single PCR reaction. If multiple forward or reverse primers are present in a single PCR reaction, multiple sets of fwd_name/fwd_seq or rev_name/rev_seq values will be present.</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>[fwd_name: XXX1, ]fwd_seq: xxxxx1,[fwd_name: XXX2, ]fwd_seq: xxxxx2, [rev_name: YYY1, ]rev_seq: yyyyy1, [rev_name: YYY2, ]rev_seq: yyyyy2
    Example 1<br />
    fwd_name: CO1P1, fwd_seq: ttgattttttggtcayccwgaagt, rev_name: CO1R4, rev_seq: ccwvytardcctarraartgttg
    Example 2<br />
    fwd_seq: tgtgtgtgtgactgaca, rev_seq: tagcgatacggtcaatgc
    Example 3<br />
    fwd_name: hoge1, fwd_seq: cgkgtgtatcttact, rev_name: hoge2, rev_seq: cggtgtatcttact
    Example 4<br />
    fwd_name: CO1P1, fwd_seq: ttgattttttggtcayccwgaagt, fwd_name: CO1P2, fwd_seq: gatacacaggtcayccwgaagt, rev_name: CO1R4, rev_seq: ccwvytardcctarraartgttg"</td>
    </tr>
    <tr class="odd">
    <td>Comment</td>
    <td>fwd_seq and rev_seq are both mandatory; fwd_name and rev_name are both optional.<br />
    Both sequences should be presented in 5' to 3' order.<br />
    The sequences should be given in <a href="/ddbj/code-e.html#nucleotide-1">the IUPAC degenerate-base alphabet</a>, except for <a href="/ddbj/code-e.html#nucleotide-2">the modified bases</a>; those must be enclosed within angle brackets &lt; &gt;</td>
    </tr>
    </tbody>
    </table>

  - plasmid<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#plasmid)</span>
    
    |              |                                                                                                                               |
    | ------------ | ----------------------------------------------------------------------------------------------------------------------------- |
    | Definition   | name of naturally occurring plasmid from which the sequence was obtained. Must not be used for description of cloning vector. |
    | Value format | \<text\>, excluding double quotation mark (")                                                                                 |
    | Example      | C-589                                                                                                                         |
    

  - product<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#product)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>name of the product associated with the feature, e.g. the <a href="/ddbj/features-e.html#mRNA">mRNA</a> of an mRNA feature, the polypeptide of a <a href="/ddbj/features-e.html#cds">CDS</a>, the mature peptide of a <a href="/ddbj/features-e.html#mat_peptide">mat_peptide</a>, etc.</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>&lt;text&gt;, excluding double quotation mark (")</td>
    </tr>
    <tr class="odd">
    <td>Example</td>
    <td>trypsinogen (when qualifier appears in CDS feature)<br />
    trypsin (when qualifier appears in mat_peptide feature)<br />
    XYZ neural-specific transcript (when qualifier appears in mRNA feature)</td>
    </tr>
    <tr class="even">
    <td>Guidance for Submission:</td>
    <td>See also <a href="/ddbj/cds-e.html#product">Gene nomenclature at DDBJ</a>.<br />
    
    <ul>
    <li>In principle, please enter a general name, not abbreviation.</li>
    <li>Do not include the organism name.</li>
    <li>Even if there are multiple general names for the same product, do not enter multiple names in 'product'. Do not use needless symbolic letters as delimiter for multiple names. If you would like to describe more than two names, please enter one of the most representative name in 'product', and other(s) in '<a href="#note">note</a>' qualifier.</li>
    <li>If the name and function are not known, we recommend to describe as "hypothetical protein".</li>
    </ul></td>
    </tr>
    </tbody>
    </table>

  - protein\_id<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#protein_id)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>Protein Identifier for <a href="/ddbj/features-e.html#cds">CDS feature</a>, issued by INSDC.</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>&lt;identifier&gt;.&lt;version&gt;</td>
    </tr>
    <tr class="odd">
    <td>Example</td>
    <td>BAA12345.1<br />
    AAA1234567.1</td>
    </tr>
    <tr class="even">
    <td>Comment</td>
    <td>This qualifier consists of a stable ID portion (accepted data before the end of<br />
    2018 uses a 3+5 format with 3 position letters and 5 numbers; from the end of<br />
    2018 new data may be extended to a 3+7 accession format with 3 position letters<br />
    and 7 numbers) plus a version number after the decimal point.<br />
    When the protein sequence encoded by the CDS changes, only the version number of<br />
    the /protein_id value is incremented.<br />
    The stable part of the /protein_id remains unchanged and as a result will permanently<br />
    be associated with a given protein.<br />
    This qualifier is valid only on CDS features which translate into a valid protein.</td>
    </tr>
    </tbody>
    </table>

  - proviral<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#proviral)</span>
    
    |              |                                                                                                                               |
    | ------------ | ----------------------------------------------------------------------------------------------------------------------------- |
    | Definition   | this qualifier is used to flag sequence obtained from a virus or phage that is integrated into the genome of another organism |
    | Value format | none                                                                                                                          |
    

  - pseudo<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#pseudo)</span>
    
    |              |                                                                                                                                                                                                                      |
    | ------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
    | Definition   | indicates that this feature is a non-functional version of the element named by the feature key. When pseudo qualifier is shown, [CDS feature](/ddbj/features-e.html#cds) does not have [translation](#translation). |
    | Value format | none                                                                                                                                                                                                                 |
    | Comment      | Do not use for new submission. If necessary, describe [pseudogene](#pseudogene).                                                                                                                                     |
    

  - pseudogene<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#pseudogene)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>indicates that this feature is considered a pseudogene of the element named by the feature key. When pseudogene qualifier is shown, <a href="/ddbj/features-e.html#cds">CDS feature</a> does not have <a href="#translation">translation</a>.</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>"TYPE"<br />
    where TYPE is one of the following:
    <ul>
    <li>processed</li>
    <li>unprocessed</li>
    <li>unitary</li>
    <li>allelic</li>
    <li>unknown</li>
    </ul></td>
    </tr>
    <tr class="odd">
    <td>Comment</td>
    <td>See <a href="http://www.insdc.org/documents/pseudogene-qualifier-vocabulary">Controlled vocabulary for /pseudogene qualifier</a> for TYPE, in detail.</td>
    </tr>
    </tbody>
    </table>

  - rearranged<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#rearranged)</span>
    
    |              |                                                                                                                                                                                                    |
    | ------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
    | Definition   | the sequence presented in the entry has undergone somatic rearrangement as part of an adaptive immune response; it is not the unrearranged sequence that was inherited from the parental germline. |
    | Value format | none                                                                                                                                                                                               |
    | Comment      | Do not use with [germline](#germline) qualifier.                                                                                                                                                   |
    

  - regulatory\_class<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#regulatory_class)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>a structured description of the classification of transcriptional, translational, replicational and chromatin structure related regulatory elements in a sequence</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>TYPE<br />
    where TYPE is one of the following:
    <ul>
    <li>attenuator</li>
    <li>CAAT_signal</li>
    <li>DNase_I_hypersensitive_site</li>
    <li>enhancer</li>
    <li>enhancer_blocking_element</li>
    <li>GC_signal</li>
    <li>imprinting_control_region</li>
    <li>insulator</li>
    <li>locus_control_region</li>
    <li>matrix_attachment_region</li>
    <li>minus_35_signal</li>
    <li>minus_10_signal</li>
    <li>polyA_signal_sequence</li>
    <li>promoter</li>
    <li>recoding_stimulatory_region</li>
    <li>replication_regulatory_region</li>
    <li>response_element</li>
    <li>ribosome_binding_site</li>
    <li>riboswitch</li>
    <li>silencer</li>
    <li>TATA_box</li>
    <li>terminator</li>
    <li>transcriptional_cis_regulatory_region</li>
    <li>other</li>
    </ul></td>
    </tr>
    <tr class="odd">
    <td>Comment</td>
    <td>See <a href="http://www.insdc.org/controlled-vocabulary-regulatoryclass">Controlled vocabulary for /regulatory_class</a> for TYPE, in detail.</td>
    </tr>
    </tbody>
    </table>

  - replace<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#replace)</span>
    
    |              |                                                                                                          |
    | ------------ | -------------------------------------------------------------------------------------------------------- |
    | Definition   | indicates that the sequence identified a feature's intervals is replaced by the sequence shown in "text" |
    | Value format | \<text\>, excluding double quotation mark (")                                                            |
    | Example      | a                                                                                                        |
    

  - ribosomal\_slippage<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#ribosomal_slippage)</span>
    
    |              |                                                                                                                                                                                             |
    | ------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
    | Definition   | during protein translation, certain sequences can program ribosomes to change to an alternative reading frame by a mechanism known as ribosomal slippage                                    |
    | Value format | none                                                                                                                                                                                        |
    | Comment      | a join operator, e.g.: \[join(486..1784,1784..4810)\] should be used in the [CDS](/ddbj/features-e.html#cds) spans to indicate the [location](/ddbj/location-e.html) of ribosomal\_slippage |
    

  - rpt\_family<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#rpt_family)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>type of repeated sequence</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>&lt;text&gt;, excluding double quotation mark (")</td>
    </tr>
    <tr class="odd">
    <td>Example</td>
    <td>Alu<br />
    Kpn</td>
    </tr>
    </tbody>
    </table>

  - rpt\_type<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#rpt_type)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>organization of repeated sequence</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>limited to followings;<br />
    
    <ul>
    <li>tandem</li>
    <li>inverted</li>
    <li>flanking</li>
    <li>terminal</li>
    <li>direct</li>
    <li>dispersed</li>
    <li>nested</li>
    <li>long_terminal_repeat</li>
    <li>non_ltr_retrotransposon_polymeric_tract</li>
    <li>x_element_combinatorial_repeat</li>
    <li>y_prime_element</li>
    <li>telomeric_repeat</li>
    <li>centromeric_repeat</li>
    <li>other</li>
    </ul></td>
    </tr>
    <tr class="odd">
    <td>Comment</td>
    <td>See <a href="http://www.insdc.org/controlled-vocabulary-rpttype-qualifier">Controlled vocabulary for /rpt_type qualifier</a>, in detail.</td>
    </tr>
    </tbody>
    </table>

  - rpt\_unit\_seq<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#rpt_unit_seq)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>identity of a repeat sequence</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>text; limited to following letters; acgtmrwsykvhdbn0123456798()</td>
    </tr>
    <tr class="odd">
    <td>Example</td>
    <td>aagggc<br />
    ag(5)tg(8)<br />
    (aaaga)6(aaaa)1(aaaga)12</td>
    </tr>
    </tbody>
    </table>

  - satellite<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#satellite)</span>
    
    <table>
    <colgroup>
    <col style="width: 33%" />
    <col style="width: 33%" />
    <col style="width: 33%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>identifier for satellite DNA marker; many tandem repeats (identical or related) of a short basic repeating unit</td>
    <td></td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>&lt;satellite_type&gt;[:&lt;class&gt;][ &lt;identifier&gt;]</td>
    <td></td>
    </tr>
    <tr class="odd">
    <td>Example</td>
    <td>satellite: S1a<br />
    satellite: alpha<br />
    satellite: gamma III<br />
    microsatellite: DC130<br />
    
    Comment</td>
    <td>&lt;satellite_type&gt; is mandatory. Please select from either of followings; 
    <ul>
    <li>satellite</li>
    <li>microsatellite</li>
    <li>minisatellite</li>
    </ul></td>
    </tr>
    </tbody>
    </table>

  - segment<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#segment)</span>
    
    |              |                                                                  |
    | ------------ | ---------------------------------------------------------------- |
    | Definition   | name of viral or phage segment from which sequence was obtained. |
    | Value format | \<text\>, excluding double quotation mark (")                    |
    | Example      | 6                                                                |
    

  - serotype<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#serotype)</span>
    
    |              |                                                                                            |
    | ------------ | ------------------------------------------------------------------------------------------ |
    | Definition   | variety of a species (usually bacteria or virus) characterized by its antigenic properties |
    | Value format | \<text\>, excluding double quotation mark (")                                              |
    | Example      | B1                                                                                         |
    

  - serovar<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#serovar)</span>
    
    |              |                                                                                                   |
    | ------------ | ------------------------------------------------------------------------------------------------- |
    | Definition   | serological variety of a species (usually a prokaryote) characterized by its antigenic properties |
    | Value format | \<text\>, excluding double quotation mark (")                                                     |
    | Example      | O157:H7                                                                                           |
    

  - sex<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#sex)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>sex of the organism from which the sequence was obtained. sex is used for eukaryotic organisms that undergo meiosis and have sexually dimorphic gametes (cf. <a href="#mating_type">mating_type</a>).</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>&lt;text&gt;, excluding double quotation mark (")</td>
    </tr>
    <tr class="odd">
    <td>Example</td>
    <td>female<br />
    male<br />
    hermaphrodite<br />
    monoecious<br />
    dioecious</td>
    </tr>
    </tbody>
    </table>

  - specimen\_voucher<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#specimen_voucher)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>identifier for the specimen (a part or an individual of a typical animal or plant) from which the sequence was obtained</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>[&lt;institution_code&gt;:[&lt;collection_code&gt;:]]&lt;specimen_id&gt;</td>
    </tr>
    <tr class="odd">
    <td>Example</td>
    <td>UAM:Mamm:52179<br />
    AMCC:101706<br />
    USNM:field series 8798<br />
    personal:Dan Janzen:99-SRNP-2003</td>
    </tr>
    <tr class="even">
    <td>Comment</td>
    <td>the /specimen_voucher qualifier is intended to annotate a reference to the physical specimen that remains after the sequence has been obtained; &lt;collection_code&gt;is optional.<br />
    <br />
    You can find &lt;institution_code&gt; at<br />
    <a href="ftp://ftp.ncbi.nlm.nih.gov/pub/taxonomy/coll_dump.txt">institution_code list</a> (NCBI FTP site)<br />
    <a href="https://www.ncbi.nlm.nih.gov/biocollections">BioCollections</a></td>
    </tr>
    </tbody>
    </table>

  - strain<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#strain)</span>
    
    |              |                                               |
    | ------------ | --------------------------------------------- |
    | Definition   | strain from which the sequence was obtained   |
    | Value format | \<text\>, excluding double quotation mark (") |
    | Example      | BALB/c                                        |
    

  - sub\_clone<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#sub_clone)</span>
    
    |              |                                                |
    | ------------ | ---------------------------------------------- |
    | Definition   | sub-clone from which the sequence was obtained |
    | Value format | \<text\>, excluding double quotation mark (")  |
    | Example      | lambda-hIL7.20g                                |
    

  - submitter\_seqid<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#submitter_seqid)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>unique identifier within whole of the set version for <a href="/ddbj/wgs-e.html">WGS</a>, <a href="/ddbj/tsa-e.html">TSA</a>, <a href="/ddbj/tls-e.html">TLS</a> and <a href="/ddbj/con-e.html">CON</a></td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>&lt;text&gt;, excluding double quotation ("), vertical bar (|), equal (=), greater than (&gt;),<br />
    left/right square brackets ([ ]) and space</td>
    </tr>
    <tr class="odd">
    <td>Example</td>
    <td>contig53<br />
    scaffold25</td>
    </tr>
    </tbody>
    </table>

  - sub\_species<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#sub_species)</span>
    
    |              |                                                                  |
    | ------------ | ---------------------------------------------------------------- |
    | Definition   | subspecies name of organism from which the sequence was obtained |
    | Value format | \<text\>, excluding double quotation mark (")                    |
    | Example      | troglodytes                                                      |
    

  - sub\_strain<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#sub_strain)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>sub_strain from which sequence was obtained. name or identifier of a genetically or otherwise modified strain from which sequence was obtained, derived from a parental strain (which should be annotated in the /<a href="#strain">strain</a> qualifier). </td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>&lt;text&gt;, excluding double quotation mark (")</td>
    </tr>
    <tr class="odd">
    <td>Example</td>
    <td>abis</td>
    </tr>
    <tr class="even">
    <td>Comment</td>
    <td>If the parental strain is not given, this should be annotated in the strain qualifier instead of sub_strain. 
    <ul>
    <li>In general: /strain="K-12", /sub_strain="MG1655"</li>
    <li>not given parental: /strain="MG1655"</li>
    </ul></td>
    </tr>
    </tbody>
    </table>

  - tag\_peptide<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#tag_peptide)</span>
    
    |              |                                                                                               |
    | ------------ | --------------------------------------------------------------------------------------------- |
    | Definition   | base location encoding the polypeptide for proteolysis tag of tmRNA and its termination codon |
    | Value format | \<base\_range\>                                                                               |
    | Example      | 90..122                                                                                       |
    

  - tissue\_type<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#tissue_type)</span>
    
    |              |                                                  |
    | ------------ | ------------------------------------------------ |
    | Definition   | tissue type from which the sequence was obtained |
    | Value format | \<text\>, excluding double quotation mark (")    |
    | Example      | liver                                            |
    

  - trans\_splicing<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#trans_splicing)</span>
    
    |              |                                                                                                                                                                                                                                                                                                                                 |
    | ------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
    | Definition   | indicates that exons from two RNA molecules are ligated in intermolecular reaction to form mature RNA                                                                                                                                                                                                                           |
    | Value format | none                                                                                                                                                                                                                                                                                                                            |
    | Comment      | should be used on features such as [CDS](/ddbj/features-e.html#cds), [mRNA](/ddbj/features-e.html#mRNA) and other features that are produced as a result of a trans-splicing event. This qualifier should be used only when the splice event is indicated in the join operator eg join(complement(69611..69724),139856..140087) |
    

  - transgenic<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#transgenic)</span>
    
    |              |                                                                                         |
    | ------------ | --------------------------------------------------------------------------------------- |
    | Definition   | identifies the source feature of the organism which was the recipient of transgenic DNA |
    | Value format | no value                                                                                |
    

  - transl\_except<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#transl_except)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>translational exception: single codon the translation of which does not conform to genetic code indicated by <a href="#transl_table">transl_table</a></td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>(pos:location,aa:&lt;amino_acid&gt;)<br />
    where amino_acid is the amino acid coded by the codon at the base_range position. Amino acids are limited to the abbreviation either for <a href="/ddbj/code-e.html#amino-1">Amino Acid Codes</a>, or for <a href="/ddbj/code-e.html#amino-2">Modified and Unusual Amino Acids.</a></td>
    </tr>
    <tr class="odd">
    <td>Example 1</td>
    <td>For exceptional translation at the specific position;<br />
    /transl_except=(pos:213..215,aa:Sec)<br />
    <br />
    The codon at base 213 to 215 is exceptionally translated to selenocysteine(one letter code 'U' in amino-acid sequence)</td>
    </tr>
    <tr class="even">
    <td>Example 2</td>
    <td>For partial termination codons;<br />
    /transl_except=(pos:1017,aa:TERM)<br />
    /transl_except=(pos:2000..2001,aa:TERM)<br />
    TAA stop codon, either a single base T at base 1017, or two bases TA at base 2000 to 2001, are completed by the addition of 3' A residues to the mRNA.</td>
    </tr>
    <tr class="odd">
    <td>Example 3</td>
    <td>If the amino acid is not on the restricted vocabulary list use;<br />
    /transl_except=(pos:213..215,aa:OTHER)<br />
    <br />
    /note="name of unusual amino acid"<br />
    The codon at the position at base 213 to 215 is exceptionally translated to the amino acid defined in the /note qualifier (one letter code 'X' in amino-acid sequence).</td>
    </tr>
    </tbody>
    </table>

  - transl\_table<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#transl_table)</span>
    
    <table>
    <colgroup>
    <col style="width: 50%" />
    <col style="width: 50%" />
    </colgroup>
    <tbody>
    <tr class="odd">
    <td>Definition</td>
    <td>definition of genetic code table used if other than universal genetic code table.</td>
    </tr>
    <tr class="even">
    <td>Value format</td>
    <td>&lt;integer&gt; (1 - 6, 9 - 14, 16, 21 - 31)</td>
    </tr>
    <tr class="odd">
    <td>Example</td>
    <td>11</td>
    </tr>
    <tr class="even">
    <td>Comment</td>
    <td>Nucleotide sequence of <a href="/ddbj/features-e.html#cds">CDS</a> is automatically translated to one-letter abbreviated<br />
    amino acid sequence.<br />
    Genetic code exceptions should be reported in <a href="#transl_except">/transl_except</a> or <a href="#exception">/exception</a>.<br />
    <br />
    See <a href="/ddbj/geneticcode-e.html">the genetic code list</a>.<br />
    When /transl_table is not specified, standard code (/transl_table=1) is used for<br />
    translation automatically.</td>
    </tr>
    <tr class="odd">
    <td>Input method</td>
    <td>for Nucleotide Sequence Submission System<br />
    If the organism name is not found in the taxonomy database, please enter 'genetic<br />
    code' for source feature. Then the value is reflected to transl_table qualifier of<br />
    each <a href="/ddbj/features-e.html#cds">CDS</a> feature.
    for MSS<br />
    Please specify the appropriate genetic code corresponds to the organism and<br />
    organelle.</td>
    </tr>
    </tbody>
    </table>

  - translation<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#translation)</span>
    
    |              |                                                                                                                                                                                                                                                                                                                                                                                                                |
    | ------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
    | Definition   | In usual, [automatically generated one-letter abbreviated amino acid sequence](/ddbj/cds-e.html#translation) derived from either the universal genetic code or the table as specified in [/transl\_table](#transl_table) and as determined by exceptions in the [/transl\_except](#transl_except) qualifiers. So, it is not required for submitter to describe except using [exception](#exception) qualifier. |
    | Value format | IUPAC one-letter amino acid abbreviation as shown in [Amino Acid Codes](/ddbj/code-e.html#amino-1),"X" is to be used for AA exceptions.                                                                                                                                                                                                                                                                        |
    | Example      | MERRYCHRISTMASANDHAPPYNEWYEAR                                                                                                                                                                                                                                                                                                                                                                                  |
    | Comment      | When [pseudo](#pseudo) or [pseudogene](#pseudogene) qualifier is shown, [CDS](/ddbj/features-e.html#cds) does not have /translation.                                                                                                                                                                                                                                                                           |
    

  - variety<span class="FT">[Feature Table
    Definition](/ddbj/full_index-e.html#variety)</span>
    
    |              |                                                                                           |
    | ------------ | ----------------------------------------------------------------------------------------- |
    | Definition   | variety (= varietas, a formal Linnaean rank) of organism from which sequence was derived. |
    | Value format | \<text\>, excluding double quotation mark (")                                             |
    | Example      | insularis                                                                                 |
    

</div>

</div>

</div>

</div>

<div id="side_navigation">

<div class="side_nav_menu handbook_contents page-noside">

<div id="in_this_page">

## In this page

</div>

</div>

</div>

</div>

<div id="footer_original" class="clearfix">

  - [Policies and
    Disclaimers](/policies-e.html "Policies and Disclaimers")
  - [News](/news/en/index-e.html "News archives")
  - [FAQs](/faq/en/index-e.html "Frequently asked questions")
  - [Sitemap](/sitemap-e.html "Sitemap")
  - [Calendar](/calendar-e.html "DDBJ Center Calendar")
  - [Address](/access-e.html "DDBJ Center address")
  - [Contact](/contact-e.html "Contact to DDBJ Center")
  - Last modified: 2019-03-26

</div>

</div>
