---
layout: indexed_content
title: TPA Submission Guidelines
pathname: tpa-table-e
category: tentative
---

<div id="page" class="top">

<div id="header" class="clearfix">

<div id="header-top">

[![](/images/parts/ddbj-logo-white.svg)](/index-e.html "DNA Data Bank of Japan")

<div id="header-top-left">

<span class="resources-btn">Services**</span>

</div>

<div id="header-top-right">

  - [Login & Submit](https://ddbj.nig.ac.jp/D-way)
  - [Policies and Disclaimers](/policies-e.html)
  - [Contact](/contact-e.html)
  - [Japanese](/ddbj/tpa-table.html)

</div>

</div>

<div class="resources-contents">

<div class="boxContainer">

<div class="resources-box">

  - ### ![](/images/logos/service-org.svg)Search

  - [getentry](http://getentry.ddbj.nig.ac.jp/top-e.html "Retrieve sequence records by accession numbers")

  - [ARSA](http://ddbj.nig.ac.jp/arsa/ "Retrieve sequence and annotation by keywords")

  - [DRA
    Search](http://ddbj.nig.ac.jp/DRASearch "Search and retrieve sequencing data.")

  - [TXSearch](http://ddbj.nig.ac.jp/tx_search/?lang=en "Search unified taxonomy database")

  - [BLAST](http://blast.ddbj.nig.ac.jp/blastn?lang=en "Search nucleotide sequences by sequence similarity")

<!-- end list -->

  - ### ![](/images/logos/wrench.svg)Analysis

  - [Vector Screening
    System](http://ddbj.nig.ac.jp/vecscreen/ "Screen vector sequences")

  - [ClustalW](http://clustalw.ddbj.nig.ac.jp/index.php?lang=en "Multiple alignment and tree-making")

  - [WABI (Web API for
    Biology)](/search/help/wabi/wabi-e.html "Web API for using the search services of DDB")

  - [DDBJ FTP
    Site](ftp://ftp.ddbj.nig.ac.jp/ "Download public data by ftp")

</div>

<div class="resources-box">

  - ### ![](/images/logos/database-org.svg)Databases

  - [Annotated/Assembled Sequences
    (DDBJ)](/ddbj/index-e.html "Annotated/Assembled Sequences (DDBJ)")

  - [Sequence Read Archive
    (DRA)](/dra/index-e.html "Sequence Read Archive")

  - [Genomic Expression Archive
    (GEA)](/gea/index-e.html "Genomic Expression Archive")

  - [BioProject](/bioproject/index-e.html "BioProject")

  - [BioSample](/biosample/index-e.html "BioSample")

  - [Japanese Genotype-phenotype Archive
    (JGA)](/jga/index-e.html "Japanese Genotype-phenotype Archive")

  - [Submission portal
    D-way](https://ddbj.nig.ac.jp/D-way "Submission portal D-way")

<!-- end list -->

  - ### ![](/images/logos/sc-org.svg)NIG SuperComputer

  - [NIG
    SuperComputer](https://sc.ddbj.nig.ac.jp/en "NIG SuperComputer")

</div>

<div class="resources-box dbcls">

  - ### ![](/images/logos/dbcls-s.svg)DBCLS Services

  - [AOE](http://aoe.dbcls.jp/en "Statistics and trends of gene expression data")

  - [CRISPRdirect](https://crispr.dbcls.jp/ "Designing CRISPR/Cas9 guide RNA with reduced off-target sites")

  - [DBCLS
    SRA](http://sra.dbcls.jp/ "Statistics and trends of SRA data")

  - [Gendoo](http://gendoo.dbcls.jp/ "Functional profiling of gene and disease features for omics analysis")

  - [GGGenome](https://gggenome.dbcls.jp/en/ "A ultrafast sequence search")

  - [GGRNA](https://ggrna.dbcls.jp/en/ "A Google-like, ultrafast search engine for genes and transcripts")

  - [RefEx](http://refex.dbcls.jp/index.php?lang=en "A web tool for a comfortable search of reference data for gene expression analysis")

</div>

</div>

</div>

<div id="header-middle" class="clearfix">

<div class="page-title">

[DDBJ Annotated/Assembled Sequences](/ddbj/index-e.html)

</div>

<div class="search-box">

<div id="search-box-flex">

<div class="search-options-wrapper">

BI-DDBJ Web Sites Annotated/Assembled Sequences Sequence Read Archive

</div>

</div>

</div>

</div>

<div id="header-bottom">

  - [Home](/ddbj/index-e.html)
  - [Submission![](/images/parts/arrow_down.svg)](/ddbj/submission-e.html)
      - [Before Submission](submission-e.html)
      - [Web submission](websub-e.html)
      - [Mass submission](mss-e.html)
      - [Data Update](updt-e.html)
  - [Search![](/images/parts/arrow_down.svg)](http://ddbj.nig.ac.jp/arsa/?lang=en)
      - [getentry](http://getentry.ddbj.nig.ac.jp/top-e.html)
      - [ARSA](http://ddbj.nig.ac.jp/arsa/?lang=en)
  - [Flat File![](/images/parts/arrow_down.svg)](/ddbj/flat-file-e.html)
      - [Feature key](/ddbj/features-e.html)
      - [Qualifier key](/ddbj/qualifiers-e.html)
      - [Organism qualifier](/ddbj/organism-e.html)
      - [Description of Location](/ddbj/location-e.html)
      - [Protein Coding Sequence](/ddbj/cds-e.html)
      - [The Genetic Codes](/ddbj/geneticcode-e.html)
      - [Codes Used in Sequence Description](/ddbj/code-e.html)
      - [Example of Submission](/ddbj/example-e.html)
  - [Data
    Categories![](/images/parts/arrow_down.svg)](/ddbj/data-categories-e.html)
      - [Data Submission from Genome Project](/ddbj/genome-e.html)
      - [WGS](/ddbj/wgs-e.html)
      - [CON](/ddbj/con-e.html)
      - [GSS](/ddbj/gss-e.html)
      - [HTG](/ddbj/htg-e.html)
      - [Submission of environmental
        sequences](/ddbj/environmental-e.html)
      - [ENV](/ddbj/env-e.html)
      - [TLS](/ddbj/tls-e.html)
      - [Data Submission from Transcriptome
        Project](/ddbj/transcriptome-e.html)
      - [TSA](/ddbj/tsa-e.html)
      - [EST](/ddbj/est-e.html)
      - [HTC](/ddbj/htc-e.html)
      - [Third Party Data (TPA)](/ddbj/tpa-e.html)
  - [FAQ](/ddbj/faq-e.html)
  - [Other![](/images/parts/arrow_down.svg)](/)
      - [Patent](/ddbj/pat-data-e.html)
      - [MGA](/ddbj/mga-e.html)

</div>

</div>

<div id="main" class="clearfix">

<div id="breadcrumb" class="clearfix">

  - [HOME](/ddbj/index-e.html "DDBJ Annotated/Assembled Sequences top page")
  - \>
  - TPA Submission Guidelines

</div>

<div id="primary">

<div id="page_main">

# TPA Submission Guidelines

The original site; [TPA Submission
Guidelines](http://www.insdc.org/documents/tpa-submission-guidelines)
2013.10.15 version

Third PArty data (TPA) are submitted to the International Nucleotide
Sequence Databases as part of the process of publishing biological
studies that include the assembly and/or annotation of existing INSDC
reads and primary sequences. Publicly accessible TPA data are therefore
linked to a publication or publications that document the derivation of
the data supported by peer-reviewed scientific evidence.

## TPA subcategories

All TPA records belong to one of these classes:
<span class="bold">TPA:experimental</span>,
<span class="bold">TPA:inferential, TPA:specialist\_db,</span> or
<span class="bold">TPA:assembly</span>.

  - TPA:experimental  
    describes records that include functional annotation derived at
    least in part from peer-reviewed wet-lab experimental investigation.
  - TPA:inferential  
    describes records that include functional annotation derived from
    peer-reviewed bioinformatic investigation.
  - TPA:specialist\_db  
    describes records whose sequences are submitted from an existing
    authoritative public database that is built using INSDC sequence
    data and is described in an accepted peer-reviewed publication. The
    existing database is therefore recognized to be comprehensive, to
    have added value, and to be maintained long term.
  - TPA:assembly  
    describes records reporting assembly or reassembly, for which the
    generation, whether it is purely informatic or informed by
    experimentation, has been subject to peer review. Annotation may or
    may not be available and does not require to be part of the peer
    review for this TPA class. A further requirement for TPA:assembly is
    for submitters to provide an alignment file containing the
    reads/sequences mapped to the assembly.

TPA records are clearly labeled with keywords indicating their TPA
status and their class. Constructed genomes where no experimental
evidence is presented (in
<span style="font-weight:bold">TPA:assembly</span>) are permitted to
include only annotation relating to genes of known function (as opposed
to hypothetical proteins, for example). Submissions containing neither
assembly information nor annotation that has resulted from peer-reviewed
*in vivo*, *in vitro* or *in silico* experimentation are not accepted in
TPA. The outputs of computational tools, feature identification
algorithms, and homology search tools alone are not sufficient for TPA.

## Acceptable records for TPA

Below is a list of typical TPA entry types and the class to which they
belong. Please note that this list is neither exhaustive (there may be
further entry types that would be acceptable) nor defining of the
complete set of requirements (other constraints will be applied that are
documented outside this list)

<div class="main_table format">

<table>
<colgroup>
<col style="width: 33%" />
<col style="width: 33%" />
<col style="width: 33%" />
</colgroup>
<thead>
<tr class="header">
<th>Record<br />
Type</th>
<th>TPA Tier</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>1</td>
<td><strong>Experimental</strong></td>
<td>CDS and related annotation applied to a sequence derived from existing genomic, EST and/or mRNA primary records with wet laboratory experimental evidence for existence of at least part of the transcript (eg. RT-PCR, Northern)</td>
</tr>
<tr class="even">
<td>2</td>
<td><strong>Experimental</strong></td>
<td>CDS and related annotation applied to a sequence derived from existing genomic, EST and/or mRNA primary records, in addition to novel sequencing, with wet laboratory experimental evidence for existence of at least part of the transcript (eg. RT-PCR, Northern)</td>
</tr>
<tr class="odd">
<td>3</td>
<td><strong>Experimental</strong></td>
<td>CDS and related annotation applied to a sequence derived from existing genomic, EST and/or mRNA primary records with experimental evidence for the presence of the product (eg. antibody staining, biochemical assay)</td>
</tr>
<tr class="even">
<td>4</td>
<td><strong>Experimental</strong></td>
<td>CDS and related annotation applied to a sequence derived from existing genomic, EST and/or mRNA primary records, in addition to novel sequencing, with experimental evidence for the presence of the product (eg. antibody staining, biochemical assay)</td>
</tr>
<tr class="odd">
<td>5</td>
<td><strong>Experimental</strong></td>
<td>Re-assignment of the product name and/or function of a coding gene where there is no change to existing annotated exon, mRNA and CDS locations and wet laboratory experimental evidence is presented</td>
</tr>
<tr class="even">
<td>6</td>
<td><strong>Experimental</strong></td>
<td>Annotation of non-coding transcripts, such as antisense regulators, with wet laboratory experimental evidence for their existence and/or function</td>
</tr>
<tr class="odd">
<td>7</td>
<td><strong>Experimental</strong></td>
<td>Annotation of repeat features in association with transposon, retrotransposon, integron, iteron and insertion sequences with wet laboratory experimental evidence.</td>
</tr>
<tr class="even">
<td>8</td>
<td><strong>Experimental</strong></td>
<td>Annotation of functional RNA genes, such as tRNAs, scRNAs, etc. with wet laboratory experimental evidence</td>
</tr>
<tr class="odd">
<td>9</td>
<td><strong>Experimental</strong></td>
<td>A record submitted as part of a collection of annotated members of a gene family, where wet laboratory experimental evidence exists for the annotation.</td>
</tr>
<tr class="even">
<td>10</td>
<td><strong>Experimental</strong></td>
<td>A record or set of records representing a novel assembly or reassembly of primary read and sequence data that exist in INSDC, for which wet laboratory experimental functional annotation data are presented and have been subject to peer review associated with the linked publication.</td>
</tr>
<tr class="odd">
<td>11</td>
<td><strong>Inferential</strong></td>
<td>CDS and related annotation applied to a sequence derived from existing genomic, EST and/or mRNA primary records with reported wet laboratory experimental evidence for a homologous molecule, but no direct wet laboratory experimental evidence. The reported experimental evidence must have been generated by the submission group and must comply with TPA requirements for peer review.</td>
</tr>
<tr class="even">
<td>12</td>
<td><strong>Inferential</strong></td>
<td>CDS and related annotation applied to a sequence derived from existing genomic, EST and/or mRNA primary records, in addition to novel sequencing, with no wet laboratory experimental evidence. If novel sequence is used to bridge two pieces of sequence, experimental evidence for a homologous molecule should exist.</td>
</tr>
<tr class="odd">
<td>13</td>
<td><strong>Inferential</strong></td>
<td>Record of sequence and annotation concepts covered in a review paper or discussion section, where wet laboratory experimental evidence is reported, but not generated by the TPA submitter</td>
</tr>
<tr class="even">
<td>14</td>
<td><strong>Inferential</strong></td>
<td>Annotation of non-coding genes and transcripts with no wet laboratory experimental evidence for their existence and/or function, when submitted as part of a collection of sequences with experimental evidence for at least one member of the collection</td>
</tr>
<tr class="odd">
<td>15</td>
<td><strong>Inferential</strong></td>
<td>Annotation of pseudogenes with no wet laboratory experimental evidence, when submitted as part of a study that includes TPA records of functional homologues of the pseudogene</td>
</tr>
<tr class="even">
<td>16</td>
<td><strong>Inferential</strong></td>
<td>Annotation of pseudogenes that are not part of a study for which there exists experimental evidence</td>
</tr>
<tr class="odd">
<td>17</td>
<td><strong>Inferential</strong></td>
<td>A record submitted as part of a collection of annotated members of a gene family, where wet laboratory experimental evidence does not exist for the annotation. One or more other members of the set should have experimental evidence and should have been submitted to TPA:experimental or to the INSDC primary database.</td>
</tr>
<tr class="even">
<td>18</td>
<td><strong>Inferential</strong></td>
<td>A record representing a completely sequenced genome, or completely sequenced naturally occurring extrachromosomal element, comprising features, most of which have assigned gene symbols or product identifiers, where the annotated features may be a mix of experimentally and inferentially determined data.</td>
</tr>
<tr class="odd">
<td>19</td>
<td><strong>Inferential</strong></td>
<td>A record or set of records representing a novel assembly or reassembly of primary read and sequence data that exist in INSDC, for which inferred functional annotation data are presented and have been subject to peer review associated with the linked publication.</td>
</tr>
<tr class="even">
<td>20</td>
<td><strong>Specialist DB</strong></td>
<td>A record submitted as part of a comprehensive collection of annotations from a given class or set of classes of gene that are derived using a published standard operating procedure that has undergone peer review. These submissions will come only from authoritative public databases of standing (such as those meeting the criteria for repeated publications in the NAR Annual Database Issue*) for which a genuine case can be made for the use of TPA.</td>
</tr>
<tr class="odd">
<td>21</td>
<td><strong>Assembly</strong></td>
<td>A record or set of records representing a novel assembly or reassembly of primary read and sequence data that exist in INSDC, for which no functional annotation data are presented, but for which the assembly has been assessed in the peer review process associated with the linked publication.</td>
</tr>
<tr class="even">
<td>22</td>
<td><strong>Assembly</strong></td>
<td>A record or set of records representing a novel assembly or reassembly, with functional annotation, of primary read and sequence data that exist in INSDC, for which the assembly but not the functional annotation has been assessed in the peer review process associated with the linked publication.</td>
</tr>
</tbody>
</table>

</div>

## Unacceptable records for TPA

Please note that this list is not exhaustive.

<div class="main_table format">

<table>
<colgroup>
<col style="width: 33%" />
<col style="width: 33%" />
<col style="width: 33%" />
</colgroup>
<thead>
<tr class="header">
<th>Record<br />
Type</th>
<th>TPA Tier</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>A</td>
<td><strong>Not accepted</strong></td>
<td>Annotation of repeat (and no other) features</td>
</tr>
<tr class="even">
<td>B</td>
<td><strong>Not accepted</strong></td>
<td>Annotation that has arisen from an automated tool, such as GeneMark, tRNA scan or ORF finder, where no further evidence, experimental or otherwise, is presented for the annotation. The annotation in these cases has not been the subject of the peer review of the publication.</td>
</tr>
<tr class="odd">
<td>C</td>
<td><strong>Not accepted</strong></td>
<td>A record representing a completely sequenced genome including only features that have not been assigned gene symbols or product identifiers, for which none has wet laboratory experimental evidence.</td>
</tr>
</tbody>
</table>

</div>

</div>

</div>

<div id="side_navigation">

<div class="side_nav_menu handbook_contents page-noside">

<div id="in_this_page">

## In this page

</div>

</div>

</div>

</div>

<div id="footer_original" class="clearfix">

  - [Policies and
    Disclaimers](/policies-e.html "Policies and Disclaimers")
  - [News](/news/en/index-e.html "News archives")
  - [FAQs](/faq/en/index-e.html "Frequently asked questions")
  - [Sitemap](/sitemap-e.html "Sitemap")
  - [Calendar](/calendar-e.html "DDBJ Center Calendar")
  - [Address](/access-e.html "DDBJ Center address")
  - [Contact](/contact-e.html "Contact to DDBJ Center")
  - Last modified: 2018-01-25

</div>

</div>
