---
layout: indexed_content
title: Submission to JGA
pathname: submission-e
category: aboutjga
---

<div id="page" class="top">

<div id="header" class="clearfix">

<div id="header-top">

[![](/images/parts/ddbj-logo-white.svg)](/index-e.html "DNA Data Bank of Japan")

<div id="header-top-left">

<span class="resources-btn">Services**</span>

</div>

<div id="header-top-right">

  - [Login & Submit](https://ddbj.nig.ac.jp/D-way)
  - [Policies and Disclaimers](/policies-e.html)
  - [Contact](/contact-e.html)
  - [Japanese](/jga/submission.html)

</div>

</div>

<div class="resources-contents">

<div class="boxContainer">

<div class="resources-box">

  - ### ![](/images/logos/service-org.svg)Search

  - [getentry](http://getentry.ddbj.nig.ac.jp/top-e.html "Retrieve sequence records by accession numbers")

  - [ARSA](http://ddbj.nig.ac.jp/arsa/ "Retrieve sequence and annotation by keywords")

  - [DRA
    Search](http://ddbj.nig.ac.jp/DRASearch "Search and retrieve sequencing data.")

  - [TXSearch](http://ddbj.nig.ac.jp/tx_search/?lang=en "Search unified taxonomy database")

  - [BLAST](http://blast.ddbj.nig.ac.jp/blastn?lang=en "Search nucleotide sequences by sequence similarity")

<!-- end list -->

  - ### ![](/images/logos/wrench.svg)Analysis

  - [Vector Screening
    System](http://ddbj.nig.ac.jp/vecscreen/ "Screen vector sequences")

  - [ClustalW](http://clustalw.ddbj.nig.ac.jp/index.php?lang=en "Multiple alignment and tree-making")

  - [WABI (Web API for
    Biology)](/search/help/wabi/wabi-e.html "Web API for using the search services of DDB")

  - [DDBJ FTP
    Site](ftp://ftp.ddbj.nig.ac.jp/ "Download public data by ftp")

</div>

<div class="resources-box">

  - ### ![](/images/logos/database-org.svg)Databases

  - [Annotated/Assembled Sequences
    (DDBJ)](/ddbj/index-e.html "Annotated/Assembled Sequences (DDBJ)")

  - [Sequence Read Archive
    (DRA)](/dra/index-e.html "Sequence Read Archive")

  - [Genomic Expression Archive
    (GEA)](/gea/index-e.html "Genomic Expression Archive")

  - [BioProject](/bioproject/index-e.html "BioProject")

  - [BioSample](/biosample/index-e.html "BioSample")

  - [Japanese Genotype-phenotype Archive
    (JGA)](/jga/index-e.html "Japanese Genotype-phenotype Archive")

  - [Submission portal
    D-way](https://ddbj.nig.ac.jp/D-way "Submission portal D-way")

<!-- end list -->

  - ### ![](/images/logos/sc-org.svg)NIG SuperComputer

  - [NIG
    SuperComputer](https://sc.ddbj.nig.ac.jp/en "NIG SuperComputer")

</div>

<div class="resources-box dbcls">

  - ### ![](/images/logos/dbcls-s.svg)DBCLS Services

  - [AOE](http://aoe.dbcls.jp/en "Statistics and trends of gene expression data")

  - [CRISPRdirect](https://crispr.dbcls.jp/ "Designing CRISPR/Cas9 guide RNA with reduced off-target sites")

  - [DBCLS
    SRA](http://sra.dbcls.jp/ "Statistics and trends of SRA data")

  - [Gendoo](http://gendoo.dbcls.jp/ "Functional profiling of gene and disease features for omics analysis")

  - [GGGenome](https://gggenome.dbcls.jp/en/ "A ultrafast sequence search")

  - [GGRNA](https://ggrna.dbcls.jp/en/ "A Google-like, ultrafast search engine for genes and transcripts")

  - [RefEx](http://refex.dbcls.jp/index.php?lang=en "A web tool for a comfortable search of reference data for gene expression analysis")

</div>

</div>

</div>

<div id="header-middle" class="clearfix">

<div class="page-title">

[Japanese Genotype-phenotype Archive](/jga/index-e.html)

</div>

<div class="search-box">

<div id="search-box-flex">

<div class="search-options-wrapper">

BI-DDBJ Web Sites Annotated/Assembled Sequences Sequence Read Archive

</div>

</div>

</div>

</div>

<div id="header-bottom">

  - [Home](/jga/index-e.html)
  - [Studies](https://ddbj.nig.ac.jp/jga/viewer/view/studies)
  - [Submission![](/images/parts/arrow_down.svg)](/jga/submission-e.html)
      - [How to submit](/jga/substep-e.html)
      - [Example
        submission](https://docs.google.com/spreadsheets/d/1HHlxItj89fQv2oWUNBIHZ4VVGwbcC09WGD5tEiXAQZ4/edit#gid=744299318)
      - [How to download](/jga/download-e.html)
      - [XML
        Schema](https://github.com/ddbj/pub/tree/master/docs/jga/xsd/)
  - [FAQ](/jga/faq-e.html)

</div>

</div>

<div id="main" class="clearfix">

<div id="breadcrumb" class="clearfix">

  - [HOME](/jga/index-e.html "Japanese Genotype-phenotype Archive top page")
  - \>
  - Submission

</div>

<div id="primary">

<div id="page_main">

<div id="book">

# Submission to JGA

<div class="section book">

# Submission to JGA

<div class="section chapter">

## Submission Guide

The JGA accepts only de-identified data approved by NBDC. The users
apply for data submission directly to the
[NBDC](http://humandbs.biosciencedbc.jp/en/) and the JGA will only
accept submissions once the information of a successful application
process has been passed from the NBDC to the JGA.

It should be clear from the original consent agreements affiliated to
your samples if your data should be subject to controlled access. All
data submitted to the JGA has controlled access, which means that access
to the data submitted is subject to approval by
[NBDC](http://humandbs.biosciencedbc.jp/en/) as a part of a formal
application procedure.

Controlled access does not correspond to holding a release prior to
publication. All the DDBJ archival resources including the JGA enable
you to hold a submission before publication.

Accession numbers are issued to metadata objects with following
prefixes.

<div id="accession" class="main_table">

| Prefix | Metadata object | Number digit | Example         |
| ------ | --------------- | ------------ | --------------- |
| JGA    | Submission      | 11           | JGA00000000001  |
| JGAS   | Study           | 11           | JGAS00000000001 |
| JGAN   | Sample          | 11           | JGAN00000000001 |
| JGAX   | Experiment      | 11           | JGAX00000000001 |
| JGAR   | Data            | 11           | JGAR00000000001 |
| JGAZ   | Analysis        | 11           | JGAZ00000000001 |
| JGAD   | Data set        | 11           | JGAD00000000001 |
| JGAP   | Policy          | 11           | JGAP00000000001 |

</div>

Currently, there is no requirement for authors to use a specific JGA
accession in their publications. We would recommend that authors use the
JGAS (study) accession, which would provide the reader/user with a
complete overview of the study and a set of links to all the data from
that study.

An example of accession number citation: "Genotype data has been
deposited at the Japanese Genotype-phenotype Archive (JGA,
http://trace.ddbj.nig.ac.jp/jga), which is hosted by the DDBJ, under
accession number JGASXXXXXXXXX."

</div>

<div class="section chapter">

## Sequence data file (raw/unaligned and aligned)

Supported JGA data submission formats are described below. If you have
data in any other format or have any questions please [contact the
JGA](/contact-e.html) .

Data files should be de-multiplexed prior to submission so that each
data object is submitted with files containing data for a single sample
only.

<div class="section section">

### BAM format

Binary Alignment/Map files (BAM) represent one of the preferred JGA
submission formats. BAM is a compressed version of the Sequence
Alignment/Map (SAM) format (see
[SAMv1.pdf](https://samtools.github.io/hts-specs/SAMv1.pdf)). BAM files
can be decompressed to a human-readable text format (SAM) using
SAM/BAM-specific utilities (e.g. [samtools](http://www.htslib.org/)) and
can contain unaligned sequences as well.  
JGA strongly recommends to submit BAM including unaligned reads as
primary data into Data.

The BAM files are nearly optimal in terms of compression and should be
submitted uncompressed.

</div>

<div class="section section">

### Fastq format

Primary sequence data submissions of single and paired reads are
accepted as [Fastq](http://en.wikipedia.org/wiki/FASTQ_format#Quality)
files that meet the following the requirements:

<div class="sub_index">

  - Quality scores must be in
    [Phred](http://en.wikipedia.org/wiki/Phred_quality_score) scale. For
    example, quality scores from early Solexa pipelines must be
    converted to use this scale. Both ASCII and space delimitered
    decimal encoding of quality scores are supported. We will
    automatically detect the Phred quality offset of either 33 or 64.
  - No technical reads (adapters, linkers, barcodes) are allowed.
  - Single reads must be submitted using a single Fastq file.
  - Paired reads must split and submitted using two Fastq files.Â The
    read names must have a suffix identifying the first and second read
    from the pair, for example '/1' and '/2'; (regular expression for
    the reads "^(.\*)(\[\\\\.|:|/|\_\])(\[12\])$").
  - The first line for each read must start with '@'.
  - The base calls and quality scores must be separated by a line
    starting with '+'.
  - The Fastq files must be compressed using gzip or bzip2.

</div>

Example of Fastq file containing single reads:

``` code
@read_name
GATTTGGGGTTCAAAGCAGTATCGATCAAATAGTAAATCCATTTGTTCAACTCACAGTTT
+
!''*((((***+))%%%++)(%%%%).1***-+*''))**55CCF>>>>>>CCCCCCC65
...
```

Example of Fastq file containing paired reads:

``` code
@read_name/1
GATTTGGGGTTCAAAGCAGTATCGATCAAATAGTAAATCCATTTGTTCAACTCACAGTTT
+
!''*((((***+))%%%++)(%%%%).1***-+*''))**55CCF>>>>>>CCCCCCC65
@read_name/2
GATTTGGGGTTCAAAGCAGTATCGATCAAATAGTAAATCCATTTGTTCAACTCACAGTTT
+
!''*((((***+))%%%++)(%%%%).1***-+*''))**55CCF>>>>>>CCCCCCC65
...
```

</div>

<div class="section section">

### SFF format

The recommended submission format for the 454 platform is SFF.

The SFF files are nearly optimal in terms of compression and should be
submitted uncompressed.

</div>

</div>

<div class="section chapter">

## Array based (Genotypes, SNP, Expression)

The JGA accepts processed data from all types of array based
technologies, such as genotypes, gene expression, methylations, etc.

We are very flexible regarding these submissions and would accept all
types of files affiliated to the array experiment into the JGA Analysis.

We recommend that both raw data (CELs) and analysis files be submitted,
which will enable potential users of the data to recreate your results
cited in the paper as well as view the described results.

</div>

<div class="section chapter">

## Variation data file

The JGA supports submissions of sequence variations in [VCF
format](http://en.wikipedia.org/wiki/Variant_Call_Format) . The vcf
files are accepted as secondary analysis object.

</div>

<div class="section chapter">

## Other data files

Analysis can refer to multiple Data and Sample objects and summary data
summarizing data files of these objects can be registered to the
Analysis. Example analysis files are listed below.

<div class="sub_index">

  - A reference sequence GFF3 file assembled by using multiple sequence
    files in a objects.
  - A table file summarizes normalized array data files in Analysis.

</div>

</div>

<div class="section chapter">

## Metadata

The JGA data model is created by extending the [Sequence Read Archive's
model](/dra/submission-e.html). The JGA metadata are composed of the XML
objects.

JGA XML schema: [JGA
xsd](https://github.com/ddbj/pub/tree/master/docs/jga)

![Example: single policy.](/images/submission/jga_metadata_1.jpg)

![Example: separate policies for control and
case.](/images/submission/jga_metadata_2.jpg)

<div class="section">

### Summary information for public display

When JGA submissions are released, following metadata objects will be
public on the JGA and NBDC websites to display summary of submissions.

  - Submission:  
    All fields excluding submitters' contact information.
  - Study:  
    All fields.
  - Data set:  
    All fields, number of samples, and filesize.
  - Policy:  
    All fields.

</div>

<div class="section">

### Submission

The submission XML is used to describe the submission transaction. It
contains contact details of the submitter.

XML schema:
[JGA.submission.xsd](https://github.com/ddbj/pub/tree/master/docs/jga)

</div>

<div class="section">

### Study

The study XML describes your study in detail. It contains a title, a
study type and an abstract, as it would appear in the publication. After
publication, the study can be updated to include the PubMed ID of the
paper containing the submitted data.

XML schema:
[JGA.study.xsd](https://github.com/ddbj/pub/tree/master/docs/jga)

</div>

<div class="section">

### Sample

The sample XML describes each of the samples used in your submission. A
sample generally corresponds to a participant. Whilst the mandatory
fields are minimal we highly recommend to describe the sample in as much
detail as possible including subject phenotypes.

XML schema:
[JGA.sample.xsd](https://github.com/ddbj/pub/tree/master/docs/jga)

</div>

<div class="section">

### Experiment

The experiment XML is used to describe each unique set of experimental
setup details. This includes the library preparation details, next
generation sequencing platforms. The experiment XML can represent next
generation sequencing experiments.

XML schema:
[JGA.experiment.xsd](https://github.com/ddbj/pub/tree/master/docs/jga)

</div>

<div class="section">

### Data

The data XML describes the next generation sequencing data files and
their relationship with the experiments. The data XML serves as a file
container.

XML schema:
[JGA.data.xsd](https://github.com/ddbj/pub/tree/master/docs/jga)

</div>

<div class="section">

### Analysis

The Analysis XML can be used to submit array data, analyzed and
summarized data to JGA. Variation data can be submitted as VCF files.
Only one VCF file can be submitted in each analysis and the samples used
within the VCF files must be associated with JGA Samples. Optimally the
VCF file would be associated with an INSDC reference assembly and
sequences either by using accessions or by using commonly used labels.

XML schema:
[JGA.analysis.xsd](https://github.com/ddbj/pub/tree/master/docs/jga)

</div>

<div class="section">

### Data set

The dataset XML describes the data files, defined by the Data XML and
Analysis XML, that make up the dataset and links the collection of data
files to a specified Policy (Data Access Agreement). When a Study
contains datasets with different policies, separate Datasets need to be
created.

In general, policy is determined by informed consent with participants.

XML schema:
[JGA.dataset.xsd](https://github.com/ddbj/pub/tree/master/docs/jga)

</div>

<div class="section">

### Policy

The Policy XML describes the Data Access Agreement (DAA). If only [NBDC
policy](http://humandbs.biosciencedbc.jp/en/nbdc-policy) is associated
with a submission, submitters do not need to create a new policy.

If additional policies are needed, you need to register a policy to NBDC
and reference issued JGAP number from submitting data set.

XML schema:
[JGA.policy.xsd](https://github.com/ddbj/pub/tree/master/docs/jga)

</div>

</div>

<div class="section chapter">

## Analysis only submission

To submitting only aggregated data or data which are not suitable for
Sample-Experiment-Data, these data can be submitted as Sample-Analysis
instead of Sample-Experiment-Data.

![One Analysis for each Sample](/images/submission/sample-analysis.jpg)

![Analysis data aggregating
Samples](/images/submission/samples-analysis.jpg)

To submitting Analysis only data, please contact [JGA
team](/contact-e.html) .

</div>

<div class="section chapter">

## Update submissions

To update your submissions, please [contact to the JGA
team](/contact-e.html) .

</div>

</div>

</div>

</div>

</div>

<div id="side_navigation">

<div class="side_nav_menu handbook_contents handbook submission">

<div id="in_this_page">

## In this page

</div>

</div>

<div class="side_nav_menu">

## Apply data access/submission

<div id="links">

  - <span id="nbdc">[![](/images/parts/nbdc_logo.png)National Bioscience
    Database
    Center](https://humandbs.biosciencedbc.jp/en/ "National Bioscience Database Center")</span>

</div>

</div>

</div>

</div>

<div id="footer_original" class="clearfix">

  - [Policies and
    Disclaimers](/policies-e.html "Policies and Disclaimers")
  - [News](/news/en/index-e.html "News archives")
  - [FAQs](/faq/en/index-e.html "Frequently asked questions")
  - [Sitemap](/sitemap-e.html "Sitemap")
  - [Calendar](/calendar-e.html "DDBJ Center Calendar")
  - [Address](/access-e.html "DDBJ Center address")
  - [Contact](/contact-e.html "Contact to DDBJ Center")
  - Last modified: 2018-12-13

</div>

</div>
