---
layout: indexed_content
title: Categories for Sequence Data
category: documents
lang: en
related_pages:
  - title: Prefix Letter List
    url: /documents/prefix-e.html
  - title: Accession Number Assigned by INSD
    url: /documents/accessions.html
  - title: Principle of “Hold-Until-Published” data release
    url: /documents/data-release-policy-e.html
---


# Categories for Sequence Data

## Acceptable data for DDBJ  {#accept}

For the request of [Primary entry](/ddbj/submission.html#primary_entry) submission, in principle, 
DDBJ accepts any nucleotide sequences that are experimentally determined by submitters, 
but can not accept computational predicted and/or cited sequences. 

Even if your sequence is identical to previously reported sequence(s), on the condition that the sequence is independently determined, you can submit it as a "new" entry.

DDBJ also acccepts an entry that is obtained by assembling primary entries publicized from DDBJ/ENA/GenBank of INSDC and/or is added
annotation(s) by experimental or inferential method by submitter as [TPA (third party data)](/ddbj/tpa-e.html).    

However, some types of sequence data are [not acceptable for DDBJ](/ddbj/sequence-e.html#not_acceptable).    

When you are to publicize raw output data for your studies related to SNPs, [WGS](/ddbj/wgs-e.html#acceptance), [transcriptome](/ddbj/tsa-e.html) and so on, 
we recommend you to contact with [DDBJ Trace Archive](/dta/index-e.html), or [DDBJ Sequence Read Archive](/dra/index-e.html), instead of DDBJ / ENA / GenBank.

See [Overview of International Nucleotide Sequence Databases Policies](/about/insdc-e.html#policy)

### Submisson of the data including identical sequences or partially duplicated sequences

Basically, DDBJ accepts all sequence data that are independently determined, even though seqences are identical each other. 
For variation studies, DDBJ also accepts [submissions of representative data](/ddbj/representative-sequence-e.html).

If you determine many sequences derived from the same indivisual, 
we strongly recommend to update sequence data submitted previously, rather than to submit new sequence data many times.  
However, since multicycle submissions for a single resource are required by any reasons; right for sequence data, phases of sequencing etc., 
DDBJ does not restrict them.

## Sequencing Data  {#real}

### Annotated/assembled sequences  {#ann}

[DDBJ](/ddbj/index.html)  
: Narrowly-defined DDBJ. DDBJ is a counterpart of GenBank and ENA (EMBL-bank) to accept sequences with feature annotation and to
provide them in [flat file](/ddbj/flat-file-e.html).
: About the data in traditional DDBJ is classified, see [Categories of
Annotated/Assembled Data](#detail).


<div class="attention" markdown="1">
If you are not sure to which database you should submit your data, see
following sites;

  - [Steps of genome sequencing, categories of sequence data and their
    correspondences](/ddbj/genome-e.html)
  - [Steps of transcriptome project, categories of sequence data and
    their correspondences](/ddbj/transcriptome-e.html)
  - [Division](/ddbj/flat-file-e.html#Division)
  - [Categories of Annotated/Assembled Data](#detail)

Using [Mass Submission System (MSS)](/ddbj/mss-e.html), the submitted
nucleotide sequences are classified into one of the categories according
to the descriptions of the [DATATYPE, DIVISION, KEYWORD](/ddbj/file-format-e.html#annotation).
</div>

### Sequencing and alignment data from next-generation sequencing platforms  {#ngs}

[DRA: DDBJ Sequence Read Archive](/dra/index-e.html)  
: Archival database for output data generated by next-generation
sequencing machines including Roche 454 GS System，Illumina Genome
Analyzer，Applied Biosystems SOLiD System and others.

[DTA: DDBJ Trace Archive](/dta/index-e.html)  
: Archival database of DNA sequence chromatograms (traces), base
calls, and quality estimates for single-pass reads from various
large-scale sequencing projects.

### Functional genomics data  {#fg}

[Genomic Expression Archive (GEA)](/gea/index-e.html)  
: A public database of functional genomics data such as gene
expression, epigenetics and genotyping SNP array. Both microarray-
and sequence-based data are accepted.

## Research project  {#project}

[BioProject](/bioproject/index-e.html)  
: Database to organize research projects and the corresponding data.  
It is required to submit to BioProject before sequence data
submissions for [TSA](/ddbj/tsa-e.html), [TLS](/ddbj/tls-e.html),
[WGS](/ddbj/wgs-e.html) or [complete-genome
scale](/ddbj/genome-e.html) except viruses, plasmids and organelles.

## Biological sample  {#project}

[BioSample](/biosample/index-e.html)  
: Database to capture and store descriptive information about the
biological source materials, or samples, used to generate
experimental data.

## Human data requiring controlled-access  {#control}

[JGA: Japanese Genotype-phenotype Archive](/jga/index-e.html)  
: Database for permanent archiving and sharing of all types of
individual-level genetic and de-identified phenotypic data resulting
from biomedical research projects.




## Annotated/Assembled Data Categories  {#detail} 

### [Division](/ddbj/flat-file-e.html#Division) conventional sequence data  {#division} 

#### **General data: classified by source species**  {#general}

The data that are not classified into any categories described in the sections are called general data and belong here.<br>In principle, it is required for general data to have at least one source feature and at least one other [Biological feature](/ddbj/file-format-e.html#biological_feature).<br>Submitted sequences are automatically classified into one of the following divisions on the basis of the taxonomy of the source organisms.

| *Division* | *Description* |
|---|
| HUM | Human |
| PRI | Primates (other than human) |
| ROD | Rodents |
| MAM | Mammals (other than primates or rodents) |
| VRT | Vertebrates (other than mammals) |
| INV | Invertebrates |
| PLN | Plants or fungi |
| BCT | Bacteria |
| VRL | Viruses |
| PHG | Phages |


#### **ENV/SYN: impossible to identify souce species, Environmental Samples and Synthetic Constructs**  {#env}

Environmental samples and artificially constructed sequences are classified into [ENV](/ddbj/env-e.html) and SYN division,respectively. <br>In principle, it is required for ENV and SYN data to have at least one source feature and at least one other [Biological feature](/ddbj/file-format-e.html#biological_feature).

| *Division* | *Description* |
|---|
| [ENV](/ddbj/env-e.html) | Sequences obtained via environmental sampling methods, direct PCR, DGGE, etc.<br>For ENV submissions, it is necessary to describe an [environmental_sample qualifier](/ddbj/qualifiers-e.html#environmental_sample) on the source feature. |
| SYN | Synthetic constructs, sequences constructed by artificial manipulations<br>For SYN submissions, in general, the entry often has plural source features, so it should be cared.<br>See also [Description Examples of Sequence Data: E05) synthetic construct.](/ddbj/example-e.html#E05). |


#### **EST/GSS/HTC/HTG/STS: Divisions for Feasibility of Sequencing**  {#est}

Sequences derived from high throughput projects, such as large scale analyses like EST dataset, ongoing whole genome scale sequencing, and so on, are classified into the following divisions, respectively.<br>
Basically only one source feature should be described for an entry in those divisions. <br>
In this regard, however, the entries including HTC or HTG division can have some [Biological features](/ddbj/file-format-e.html#biological_feature) like as general data, if necessary.


| *Division* | *Description* |
|---|
| [EST](/ddbj/est-e.html) | Expressed sequence tags, cDNA sequences read short single pass. |
| [GSS](/ddbj/gss-e.html) | Genome survey sequences, genome sequences read short single pass. |
| [HTC](/ddbj/htc-e.html) | High throughput cDNA sequences from cDNA sequencing projects, not EST.<br>This division is to include unfinished high throughput cDNA sequences. |
| [HTG](/ddbj/htg-e.html) | High throughput genomic sequences mainly from genome sequencing projects.<br>Unfinished HTG entries are classified into different levels, as follow;<br>{::nomarkdown}<ul><li>phase0；Survey sequence generated for the purpose of library quality assessment and detection of overlaps with other clones before construction of piece contig(s)</li><li>phase1；Unfinished sequence having contigs that have NOT been ordered and oriented</li><li>phase2；Unfinished sequence having contigs that have been ordered and oriented</li></ul>{:/} |

### Data type, bulk sequence data  {#data_type} 

#### **WGS: Fragment Sequences during WGS Assembling Process**  {#wgs}

The large set of contigs from the proceeding genome project can be submitted as one of bulk sequence data, [Whole Genome Shotgun (WGS)](/ddbj/wgs-e.html).<br>
Please note that WGS data is different from others in its [format of accession number](/ddbj/flat-file-e.html#Accession). <br>
See also [Steps of genome sequencing, categories of sequence data and their correspondences](/ddbj/genome-e.html).

#### **TSA: Transcriptome Shotgun Assembly**  {#tsa}

Since 2008, we have accepted one of bulk sequence data, [Transcriptome Shotgun Assembly (TSA)](/ddbj/tsa-e.html) categorized for assembled RNA transcript sequences.<br>
Basically only one source feature should be described for a TSA entry. <br>
TSA entries can have some [Biological features](/ddbj/file-format-e.html#biological_feature) like as general data, if necessary. <br>
Please note that TSA data may be different from others in its [format of accession number](/ddbj/flat-file-e.html#Accession).<br>
See also [steps of transcriptome project, categories of sequence data and their correspondences](/ddbj/transcriptome-e.html)

#### **TLS: Targeted Locus Study**  {#tls}

Since 2016, we have accepted one of bulk sequence data, [Targeted Locus Study (TLS)](/ddbj/tls.html), including 16S rRNA or some other targeted loci mainly to be clustered into operational taxonomic unit.<br>
TLS entries can have some [Biological features](/ddbj/file-format-e.html#biological_feature) like as general data. <br>Please note that TLS data is different from others in its [format of accession number](/ddbj/flat-file-e.html#Accession).

### Sequenced by whom  {#whom} 

#### **TPA; Third Party Data and primary sequence data**  {#tpa}

[TPA (Third Party Data)](/ddbj/tpa-e.html) is a nucleotide sequence data collection in which each entry is obtained by assembling primary entries publicized from DDBJ/ENA/GenBank, [Trace Archive](//www.ncbi.nlm.nih.gov/Traces/trace.cgi), and/or [Sequence Read Archive](/dra/index-e.html) with additional feature annotation(s) determined by experimental or inferential methods by TPA submitter. Those assemblies include two cases; one or more primary entries are used and newly determined sequence is contained. TPA sequence data should be submitted to DDBJ/EMBL-Bank/GenBank as a part of the process to publish biological research for primary nucleotide sequences.<br>See also [TPA Submission Guidelines](/ddbj/tpa-table-e.html).
