---
layout: tabbed_indexed_content
service_name: Sequence Read Archive
title: DDBJ Sequence Read Archive Handbook
category: dra
current_tab: handbook
panels: 
- title: Search 
  url: https://ddbj.nig.ac.jp/search  
  icon: ../assets/images/icon-search_analysis.png 
- title: How_to_submit  
  url: /dra/submission-e.html 
  icon: ../assets/images/icon-search_analysis.png 
- title: Login_and_submit 
  url: https://ddbj.nig.ac.jp/D-way/  
  icon: ../assets/images/icon-search_analysis.png
lang: en
---

[DDBJ Sequence Read Archive (DRA)](/index-e.html) is an archive database
for output data generated by next-generation sequencing machines
including Roche 454 GS System®, Illumina Genome Analyzer®, Applied
Biosystems SOLiD® System, and others. DRA is a member of the
[International Nucleotide Sequence Database Collaboration
(INSDC)](/about/insdc-e.html)and archiving the data in a close collaboration
with [NCBI Sequence Read Archive
(SRA)](https://www.ncbi.nlm.nih.gov/Traces/sra/sra.cgi) and [EBI Sequence
Read Archive (ERA)](https://www.ebi.ac.uk/ena/).

Three INSDC partners regularly exchange data other than Analysis.

DRA accepts sequencing data from capillary sequencing platforms in the fastq
format. Because [DDBJ Trace Archive](/dta/index-e.html) will be integrated to SRA, we basically do not accept sequencing chromatograms.

## Metadata  {#metadata}

### Metadata objects  {#metadata-objects}

The metadata describes how the associated data have been obtained. The
metadata are composed of 6 objects, [Submission](#Submission),
[BioProject](#BioProject), [BioSample](#BioSample),
[Experiment](#Experiment), [Run](#Run) and [Analysis](#Analysis). Each of
these objects is defined by its XML schema and is related each other.
Multiple Experiments can "point" to a single Sample, but not vice-versa.

Accession numbers with distinct prefixes are assigned to each object.
Metadata and accession number system are common in DRA/ERA/SRA. The
Experiment, Run and Analysis are the SRA objects, and the BioProject and
BioSample are external database objects.

For details, please see the [DRA XML
schema](https://github.com/ddbj/pub/tree/master/docs/dra)


[Submission](#Submission){: #Submission-obj}    
: A container object only for grouping objects to be submitted.

[BioProject](#BioProject){: #BioProject-obj}  
: An overall description of a single research initiative; a project
will typically relate to multiple samples and datasets.

[BioSample](#BioSample){: #BioSample-obj}  
: Description of biological source material; each physically unique
specimen should be registered as a single BioSample with a unique
set of attributes.

[Experiment](#Experiment){: #Experiment-obj}   
: A description of sample-specific sequencing library and sequencing
methods. An Experiment references 1 BioProject and 1 BioSample.
Multiple Experiments can "point" to a single Sample, but not
vice-versa.

[Run](#Run){: #Run-obj}   
: Runs describe the files that belong to the previously created
Experiments. They specify the data files for a specific sample to be
processed by DRA. Note that all data files listed in a Run will be
merged into a single SRA archive file, so files from different
samples or replicates should not be grouped in the same Run.
Paired-end data files, conversely, MUST be listed in a single run in
order for the two files to be correctly processed as paired-end.

[Analysis](#Analysis){: #Analysis-obj}  
: Packages data associated with sequence read objects that are
intended for downstream usage or that otherwise needs an archival
home. Submit alignment data in bam file to Run. Please [contact to
DRA team to ask mirroring of analysis
data.](/contact-ddbj-e.html)Analysis files are provided on the DDBJ ftp
site and indexed by the
[DDBJ Search](https://ddbj.nig.ac.jp/search).

{% include image.html url="books/sra_object.png" caption="Data model" class="w450" %}

### Organization of metadata objects  {#Organization_of_metadata_objects} 

Followings are examples of metadata. Submitters can organize metadata
objects flexibly.

  - [Most simple case](#ex_simple)
  - [Comparative genome sequencing of three strains
    (paired-end)](#ex_samples)
  - [Biological replicates (paired-end)](#ex_replicates)
  - [Related sequencing data are reported in two
    publications](#ex_submissions)

#### Most simple case  {#ex_simple}

{% include image.html url="submission/obj_ex1.jpg" caption="Most simple case" class="w450" %}

#### Comparative genome sequencing of three strains (paired-end)  {#ex_samples}

Include paired-end read files in a Run.

{% include image.html url="books/datamodel1.jpg" caption="Comparative genome sequencing of three strains (paired-end)" class="w450" %}

#### Biological replicates (paired-end)  {#ex_replicates}

{% include image.html url="submission/obj_exreplicate.jpg" caption="Biological replicates (paired-end)" class="w450" %}

#### Related sequencing data are reported in two publications.  {#ex_submissions}

{% include image.html url="submission/obj_ex3.jpg" caption="Related sequencing data are reported in two publications." class="w450" %}

Items in each metadata object.

Required<span class="red">*</span>  
Conditionally required<span class="conditionally_required">*</span>

### Submission  {#Submission} 

#### Center Name  {#Area_center-name}

Enter submitter's organization.

[Center Name](#Submission_Center_Name)<a name="Submission_Center_Name"><span class="red">*</span>  
: A submitter's center name. [Center Name List.](https://ddbj.nig.ac.jp/public/ddbj_database/dra/meta/list/centerlist)
A center name abbreviation is required to submit data to DRA.

In the metadata creation tool, the center name is automatically
filled with the account information.

<span class="red">The Center Name is an abbreviation
operationally used by SRA and is not for indicating ownership of
submission. Submitters listed in
[Submitter](#Area_submitter-submission) hold ownership of
submission.</span>

<!-- end list -->

[Lab Name](#Laboratory_Name)<a name="Laboratory_Name"></a><span class="red">*</span>  
: Laboratory name within submitting institution. The Lab name is
pre-entered with "Lab/Group", "Department (2)", "Department (1)","Organization" of D-way account. Text can be editted.

#### Hold Until  {#Area_hold-until}

Specify how to release the data.

[Hold Until](#Hold_Until)<a name="Hold_Until"></a><span class="conditionally_required">*</span>  
: Direct the DRA to release the record on or after the specified
date.Submitter can set the hold date for a maximum of 2 years and can change the date before the record is released.

<!-- end list -->

[Immediate Release](#Immediate_Release)<a name="Immediate_Release"></a><span class="conditionally_required">*</span>  
: Direct the DRA to release the record immediately after submission is processed.

#### Submitter  {#Area_submitter-submission}

The DRA contacts the listed address(es) regarding the submission by
e-mail.<span class="red">Include contact information of PI
and non-PI member(s) who actually submits data.</span>The contact
information is not made public. If you want to display the contact
information, enter the information in the BioProject.

[Name](#Contact_Name)<a name="Contact_Name"></a><span class="red">*</span>  
: Name of submitter.

<!-- end list -->

[E-mail](#Contact_E-mail)<a name="Contact_E-mail"></a><span class="red">*</span>  
: E-mail of submitter.

### BioProject  {#BioProject} 

[BioProject ID](#DRA_BioProjectID)<a name="DRA_BioProjectID"></a><span class="red">*</span>  
: Select a project registered to
[BioProject](/bioproject/index-e.html) or submit a new project. For submission to BioProject, please refer to the [BioProject Handbook](/bioproject/submission-e.html).

### BioSample  {#BioSample} 

[BioSample ID](#DRA_BioSampleID)<a name="DRA_BioSampleID"></a><span class="red">*</span>  
: Select samples registered to [BioSample](/biosample/index-e.html) or
create and submit new samples. For submission to BioSample, please
refer to [BioSample Handbook](/biosample/submission-e.html).

### Experiment  {#Experiment} 

[Alias](#Experiment_Alias)<a name="Experiment_Alias"></a>  
: Name of the experiment designated by the archive. This alias is used to reference metadata objects without accession numbers.

<!-- end list -->

[BioSample Used](#BioSample_Used)<a name="BioSample_Used"></a><span class="red">*</span>  
: Select the BioSample this experiment uses.

<!-- end list -->

[Title](#Experiment_Title)<a name="Experiment_Title"></a><span class="red">*</span>  
: Short text that can be used to call out experiment records in
searches or in displays. A title like "[Sequencing Instrument
Model] [paired end] sequencing of [BioSample ID]" (for example,
"Illumina HiSeq 2000 paired end sequencing of SAMD00025741") is
automatically constructed. To enter user-defined titles, download Experiment metadata into a tab-delimited text file, edit title values and upload it.

<!-- end list -->

[Library Name](#Library_Name)<a name="Library_Name"></a>  
: The submitter's name for this library.

<!-- end list -->

[Library Source](#Library_Source)<a name="Library_Source"></a><span class="red">*</span>  
: The Library Source specifies the type of source material that is being sequenced.  

    | Library Source     | Description                                                                        |
    |---|
    | GENOMIC            | Genomic DNA (includes PCR products from genomic DNA).                              |
    | TRANSCRIPTOMIC     | Transcription products or non genomic DNA (EST, cDNA, RT-PCR, screened libraries). |
    | METAGENOMIC        | Mixed material from metagenome.                                                    |
    | METATRANSCRIPTOMIC | Transcription products from community targets.                                     |
    | SYNTHETIC          | Synthetic DNA.                                                                     |
    | VIRAL RNA          | Viral RNA.                                                                         |
    | OTHER              | Other, unspecified, or unknown library source material.                            |

<!-- end list -->

[Library Selection](#Library_Selection)<a name="Library_Selection"></a><span class="red">*</span>  
: Whether any method was used to select and/or enrich the material
being sequenced.

    | Library Selection     | Description                                                                        |
    |---|
    | RANDOM                                 | Random shearing only.                                                                                                                                                                              |
    | PCR                                    | Source material was selected by designed primers.                                                                                                                                                  |
    | RANDOM PCR                             | Source material was selected by randomly generated primers.                                                                                                                                        |
    | RT-PCR                                 | Source material was selected by reverse transcription PCR.                                                                                                                                         |
    | HMPR                                   | Hypo-methylated partial restriction digest.                                                                                                                                                        |
    | MF                                     | Methyl Filtrated.                                                                                                                                                                                  |
    | repeat fractionation                   | Selection for less repetitive (and more gene rich) sequence through Cot filtration (CF) or other fractionation techniques based on DNA kinetics.                                                   |
    | size fractionation                     | Physical selection of size appropriate targets.                                                                                                                                                    |
    | MSLL                                   | Methylation Spanning Linking Library.                                                                                                                                                              |
    | cDNA                                   | complementary DNA.                                                                                                                                                                                 |
    | cDNA_randomPriming                    |                                                                                                                                                                                                    |
    | cDNA_oligo_dT                        |                                                                                                                                                                                                    |
    | PolyA                                  | PolyA selection or enrichment for messenger RNA (mRNA); should replace cDNA enumeration.                                                                                                           |
    | Oligo-dT                               | enrichment of messenger RNA (mRNA) by hybridization to Oligo-dT.                                                                                                                                   |
    | Inverse rRNA                           | depletion of ribosomal RNA by oligo hybridization.                                                                                                                                                 |
    | ChIP                                   | Chromatin immunoprecipitation.                                                                                                                                                                     |
    | MNase                                  | Micrococcal Nuclease (MNase) digestion.                                                                                                                                                            |
    | DNAse                                  | Deoxyribonuclease (DNase) digestion.                                                                                                                                                               |
    | Hybrid Selection                       | Selection by hybridization in array or solution.                                                                                                                                                   |
    | Reduced Representation                 | Reproducible genomic subsets, often generated by restriction fragment size selection, containing a manageable number of loci to facilitate re-sampling.                                            |
    | Restriction Digest                     | DNA fractionation using restriction enzymes.                                                                                                                                                       |
    | 5-methylcytidine antibody              | Selection of methylated DNA fragments using an antibody raised against 5-methylcytosine or 5-methylcytidine (m5C)MBD2 protein methyl-CpG binding domain : Enrichment by methyl-CpG binding domain. |
    | MBD2 protein methyl-CpG binding domain | MBD2 protein methyl-CpG binding domain.                                                                                                                                                            |
    | CAGE                                   | Cap-analysis gene expression.                                                                                                                                                                      |
    | RACE                                   | Rapid Amplification of cDNA Ends.                                                                                                                                                                  |
    | MDA                                    | multiple displacement amplification.                                                                                                                                                               |
    | padlock probes capture method          | Padlock Probes capture strategy to be used in conjuction with Bisulfite-Seq.                                                                                                                       |
    | other                                  | Other library enrichment, screening, or selection process.                                                                                                                                         |
    | unspecified                            | Library enrichment, screening, or selection is not specified.                                                                                                                                      |

<!-- end list -->

[Library Strategy](#Library_Strategy)<a name="Library_Strategy"></a><span class="red">*</span>  
: Sequencing technique intended for this library. 
For the "Sequel IIe", select the "Sequel II" and indicate "Sequel IIe" usage in the [Library Construction Protocol](#Library_Construction_Protocol).

    | Library Strategy                        | Description                                                                                                                                                                                                                                                               |
    |---|
    | WGS                                     | Whole genome shotgun.                                                                                                                                                                                                                                                     |
    | WGA                                     | Whole genome amplification.                                                                                                                                                                                                                                               |
    | WXS                                     | Random sequencing of exonic regions selected from the genome.                                                                                                                                                                                                             |
    | RNA-Seq                                 | Random sequencing of whole transcriptome.                                                                                                                                                                                                                                 |
    | miRNA-Seq                               | Micro RNA and other small non-coding RNA sequencing.                                                                                                                                                                                                                      |
    | ncRNA-Seq                               | Capture of other non-coding RNA types, including post-translation modification types such as snRNA (small nuclear RNA) or snoRNA (small nucleolar RNA), or expression regulation types such as siRNA (small interfering RNA) or piRNA/piwi/RNA (piwi-interacting RNA).    |
    | ssRNA-seq                               | strand-specific RNA sequencing                                                                                                                                                                                                                                            |
    | WCS                                     | Whole chromosome (or other replicon) shotgun.                                                                                                                                                                                                                             |
    | CLONE                                   | Genomic clone based (hierarchical) sequencing.                                                                                                                                                                                                                            |
    | POOLCLONE                               | Shotgun of pooled clones (usually BACs and Fosmids).                                                                                                                                                                                                                      |
    | AMPLICON                                | Sequencing of overlapping or distinct PCR or RT-PCR products.                                                                                                                                                                                                             |
    | CLONEEND                                | Clone end (5', 3', or both) sequencing.                                                                                                                                                                                                                                   |
    | FINISHING                               | Sequencing intended to finish (close) gaps in existing coverage.                                                                                                                                                                                                          |
    | RAD-Seq                                 | Restriction Site Associated DNA Sequence                                                                                                                                                                                                                                  |
    | ChIP-Seq                                | Direct sequencing of chromatin immunoprecipitates.                                                                                                                                                                                                                        |
    | MNase-Seq                               | Direct sequencing following MNase digestion.                                                                                                                                                                                                                              |
    | DNase-Hypersensitivity                  | Sequencing of hypersensitive sites, or segments of open chromatin that are more readily cleaved by DNaseI.                                                                                                                                                                |
    | Bisulfite-Seq                           | Sequencing following treatment of DNA with bisulfite to convert cytosine residues to uracil depending on methylation status.                                                                                                                                              |
    | EST                                     | Single pass sequencing of cDNA templates.                                                                                                                                                                                                                                 |
    | FL-cDNA                                 | Full-length sequencing of cDNA templates.                                                                                                                                                                                                                                 |
    | CTS                                     | Concatenated Tag Sequencing.                                                                                                                                                                                                                                              |
    | MRE-Seq                                 | Methylation-Sensitive Restriction Enzyme Sequencing strategy.                                                                                                                                                                                                             |
    | MeDIP-Seq                               | Methylated DNA Immunoprecipitation Sequencing strategy.                                                                                                                                                                                                                   |
    | MBD-Seq                                 | Direct sequencing of methylated fractions sequencing strategy.                                                                                                                                                                                                            |
    | Tn-Seq                                  | Gene fitness determination through transposon seeding.                                                                                                                                                                                                                    |
    | FAIRE-seq                               | Formaldehyde Assisted Isolation of Regulatory Elements                                                                                                                                                                                                                    |
    | SELEX                                   | Systematic Evolution of Ligands by EXponential enrichment                                                                                                                                                                                                                 |
    | NOMe-Seq                                | Nucleosome Occupancy and Methylome sequencing.                                                                                                                                                                                                                            |
    | RIP-Seq                                 | Direct sequencing of RNA immunoprecipitates (includes CLIP-Seq, HITS-CLIP and PAR-CLIP).                                                                                                                                                                                  |
    | ChIA-PET                                | Direct sequencing of proximity-ligated chromatin immunoprecipitates.                                                                                                                                                                                                      |
    | Hi-C                                    | Chromosome Conformation Capture technique where a biotin-labeled nucleotide is incorporated at the ligation junction, enabling selective purification of chimeric DNA ligation junctions followed by deep sequencing                                                      |
    | ATAC-seq                                | Assay for Transposase-Accessible Chromatin (ATAC) strategy is used to study genome-wide chromatin accessibility. alternative method to DNase-seq that uses an engineered Tn5 transposase to cleave DNA and to integrate primer DNA sequences into the cleaved genomic DNA |
    | Targeted-Capture                        |                                                                                                                                                                                                                                                                           |
    | Tethered Chromatin Conformation Capture |                                                                                                                                                                                                                                                                           |
    | Synthetic-Long-Read                     | binning and barcoding of large DNA fragments to facilitate assembly of the fragment                                                                                                                                                                                       |
    | Other                                   | Library strategy not listed.                                                                                                                                                                                                                                              |

<!-- end list -->

[Library Construction Protocol](#Library_Construction_Protocol)<a name="Library_Construction_Protocol"></a>  
: Free form text describing the protocol by which the sequencing
library was constructed. Please include protocols of DNA
fragmentation, ligation and enrichment. If a library preparation kit
is used, include the name and version (if any) of the kit (for
example, Illumina Nextera DNA Library Preparation Kit).

Reference: Alnasir J, Shanahan HP. Investigation into the annotation
of protocol sequencing steps in the sequence read archive.
Gigascience. 2015 May 9;4:23. doi: 10.1186/s13742-015-0064-7.
eCollection 2015. PMID:
[25960871](https://www.ncbi.nlm.nih.gov/pubmed/25960871/) (Open
Access)

<!-- end list -->

[Instrument](#Instrument)<a name="Instrument"></a><span class="red">*</span>  
: Select a sequencing instrument model.

| Instrument Model                    |
|---|
| 454 GS                              |
| 454 GS 20                           |
| 454 GS FLX                          |
| 454 GS FLX+                         |
| 454 GS FLX Titanium                 |
| 454 GS Junior                       |
| Illumina Genome Analyzer            |
| Illumina Genome Analyzer II         |
| Illumina Genome Analyzer IIx        |
| Illumina HiSeq 1000                 |
| Illumina HiSeq 1500                 |
| Illumina HiSeq 2000                 |
| Illumina HiSeq 2500                 |
| Illumina HiSeq 3000                 |
| Illumina HiSeq 4000                 |
| Illumina NovaSeq 6000               |
| Illumina MiSeq                      |
| Illumina MiniSeq                    |
| Illumina iSeq 100                   |
| Illumina HiScanSQ                   |
| HiSeq X Five                        |
| HiSeq X Ten                         |
| NextSeq 500                         |
| NextSeq 550                         |
| NextSeq 1000                        |
| NextSeq 2000                        |
| Helicos HeliScope                   |
| AB SOLiD System                     |
| AB SOLiD System 2.0                 |
| AB SOLiD System 3.0                 |
| AB SOLiD 3 Plus System              |
| AB SOLiD 4 System                   |
| AB SOLiD 4hq System                 |
| AB SOLiD PI System                  |
| AB 5500 Genetic Analyzer            |
| AB 5500xl Genetic Analyzer          |
| AB 5500xl-W Genetic Analysis System |
| Complete Genomics                   |
| BGISEQ-500                          |
| DNBSEQ-G400                         |
| DNBSEQ-T7                           |
| DNBSEQ-G50                          |
| MGISEQ-2000RS                       |
| MinION                              |
| GridION                             |
| PromethION                          |
| PacBio RS                           |
| PacBio RS II                        |
| Sequel                              |
| Sequel II                           |
| Ion Torrent PGM                     |
| Ion Torrent Proton                  |
| Ion Torrent S5                      |
| Ion Torrent S5 XL                   |
| Ion GeneStudio S5                   |
| Ion GeneStudio S5 plus              |
| Ion GeneStudio S5 prime             |
| AB 310 Genetic Analyzer             |
| AB 3130 Genetic Analyzer            |
| AB 3130xL Genetic Analyzer          |
| AB 3500 Genetic Analyzer            |
| AB 3500xL Genetic Analyzer          |
| AB 3730 Genetic Analyzer            |
| AB 3730xL Genetic Analyzer          |

<!-- end list -->

[Spot Type](#Spot_Type)<a name="Spot_Type"></a><span class="red">*</span>  
: Select a layout of reads in sequencing data files.

| Spot Type   | Description                           |
|---|
| single      | Single read                           |
| paired (FF) | Paired reads with same direction.     |
| paired (FR) | Paired reads with opposite direction. |

<!-- end list -->

[Nominal Length](#Nominal_Length)<a name="Nominal_Length"></a><span class="conditionally_required">*</span>  
: Size of the insert for Paired reads.

<!-- end list -->

[Nominal Sdev](#Nominal_Sdev)<a name="Nominal_Sdev"></a>
: Standard deviation of insert size.

<!-- end list -->

[Spot Length](#Spot_Length)<a name="Spot_Length"></a><span class="red">*</span>  
: The read length in submitted sequencing files. For mate pairs, this
number includes mate pairs, but does not include gap lengths.

  - When the spot length is constant, enter a constant value.
  - For 454 platforms producing reads with variable length, enter a
    constant flow count.
  - For fastq files with variable length, enter an average length.

### Run  {#Run} 

[Alias](#Run_Alias)<a name="Run_Alias"></a>  
: Name of the run designated by the archive. This alias is used to
reference metadata objects without accession numbers.

<!-- end list -->

[Title](#Run_Title)<a name="Run_Title"></a><span class="red">*</span>  
: Short text that can be used to call out run records in searches or
in displays. A title like "[Sequencing Instrument Model] [paired
end] sequencing of [BioSample ID]" (for example, "Illumina HiSeq
2000 paired end sequencing of SAMD00025741") is automatically
constructed. To enter user-defined titles, download Run metadata
into a tab-delimited text file, edit title values and upload it.

<!-- end list -->

[Experiment Referenced](#Experiment_Referenced)<a name="Experiment_Referenced"></a><span class="red">*</span>  
: Select the experiment this run belongs to.

#### Data files for Run  {#Area_data-files-for-run}

Select data files for a Run.

[Run/Analysis](#Run_Analysis)<a name="Run_Analysis"></a>  
: Specify whether a data file belongs to the Run or Analysis. In the
web submission form, this field is un-editable and is automatically
filled according to the selected Run or Analysis. To upload metadata
in tsv file, this field needs to be specified manually.

<!-- end list -->

[File Name](#File_Name)<a name="File_Name"></a><span class="red">*</span>  
: The name of a sequence data file. Uploaded filenames are automatically filled in.

<!-- end list -->

[Run/Analysis contains files](#Run_Analysis_contains_files)<a name="Run_Analysis_contains_files"></a><span class="red">*</span>  
: Select a Run to which the data file belongs.

<!-- end list -->

[File Type](#File_Type)<a name="File_Type"></a><span class="red">*</span>  
: The sequence data file format. For the fastq files, select 'fastq' irrespective of read length.
      
    | File Type        | Description                                                                                                            |
    |---|
    | fastq            | fastq file                                                                       |
    | sff              | 454 Standard Flowgram Format file                                                                                      |
    | hdf5             | PacBio hdf5 Format file                                                                                                |
    | bam              | Binary SAM format for use by loaders that combine alignment and sequencing data                                        |
    | tab              | A tab-delimited table maps "SN in SQ line of BAM header" and "reference fasta file"                                    |
    | reference_fasta | Reference sequence file in single fasta format used to construct SRA archive file format. Filename must end with ".fa" |

<!-- end list -->

[MD5 Checksum](#MD5_Checksum)<a name="MD5_Checksum"></a><span class="red">*</span>  
: MD5 checksum of a sequence data file. [How to obtain the MD5
checksum values.](#supplement-md5)

### Analysis  {#Analysis} 

[Alias](#Analysis_Alias)<a name="Analysis_Alias"></a>
: Name of the analysis designated by the archive.This alias is used to reference metadata objects without accession numbers.

<!-- end list -->

[Title](#Analysis_Title)<a name="Analysis_Title"></a><span class="red">*</span>  
: Title of the analyis object.

<!-- end list -->

[Description](#Analysis_Description)<a name="Analysis_Description"></a><span class="red">*</span>  
: Describes the contents of the analysis.

<!-- end list -->

[Analysis Type](#Analysis_Type)<a name="Analysis_Type"></a><span class="red">*</span>  
: Select an Analysis type. Submit alignment data to Run in bam format.

  |  Analysis Type  |  Description  |
  |---|
  |  De Novo Assembly  |  A placement of sequences including trace, SRA, GI records into a multiple alignment from which a consensus is computed..  |
  |  Sequence Annotation  |  Per sequence annotation of named attributes and values.<br> Example: Processed sequencing data for submission to dbEST without assembly.<br> Reads have already been submitted to one of the sequence read archives in raw form.<br> The fasta data submitted under this analysis object result from the following treatments, which may serve to filter reads from the raw dataset:<br>     - sequencing adapter removal<br>     - low quality trimming<br>- poly-A tail removal<br>- strand orientation <br>- contaminant removal.  |
  |  Abundance Measurement  |  Identify the tools and processing steps used to produce the abundance measurements (coverage tracks).  |

#### Data files for Analysis  {#Area_data-files-for-analysis}

Select data files for an Analysis.

[Run/Analysis](#Run_Analysis_for_Analysis)<a name="Run_Analysis_for_Analysis"></a>  
: Specify whether a data file belongs to the Run or Analysis. In the
web submission form, this field is un-editable and is automatically
filled according to the selected Run or Analysis. To upload metadata
in tsv file, this field needs to be specified manually.

<!-- end list -->

[File Name](#Analysis_File_Name)<a name="Analysis_File_Name"></a><span class="red">*</span>  
: The name of an analysis file.

<!-- end list -->

[Run/Analysis contains files](#Analysis_Run_Analysis_contains_files)<a name="Analysis_Run_Analysis_contains_files"></a><span class="red">*</span>  
: Select an Analysis to which the data file belongs.

<!-- end list -->

[File Type](#Analysis_File_Type)<a name="Analysis_File_Type"></a><span class="red">*</span>  
: The analysis data file format.

    |  File Type  |  Description  |
    |---|
    |  bam  |  Binary form of the Sequence alignment/map format for read placements, from the SAM tools project.<br/>See <a href="http://sourceforge.net/projects/samtools/">http://sourceforge.net/projects/samtools/</a>.  |
    |  tab  |  A tab delimited text file that can be viewed as a spreadsheet. The first line should contain column headers..  |
    |  ace  |  Multiple alignment file output from the phred assembler and similar programs.<br/>See <a href="http://www.phrap.org/consed/distributions/README.16.0.txt">http://www.phrap.org/consed/distributions/README.16.0.txt</a> for a description of the ACE file format..  |
    |  fasta  |  Sequence data format indicating sequence base calls.The format is simple: a header line initiated with the &gt; character, data lines following with base calls..  |
    |  wig  |  The wiggle (WIG) format allows display of continuous-valued data in track format.This display type is useful for GC percent, probability scores, and transcriptome data.<br/> See <a href="http://genome.ucsc.edu/goldenPath/help/wiggle.html">http://genome.ucsc.edu/goldenPath/help/wiggle.html</a> for a description of the Wiggle Track format..  |
    |  bed  |  BED format provides a flexible way to define the data lines that are displayed in an annotation track.<br/> See <a href="http://genome.ucsc.edu/FAQ/FAQformat#format1">http://genome.ucsc.edu/FAQ/FAQformat#format1</a> for a description of the BED format..  |
    |  vcf  |  Variant Call Format.<br/>See <a href="http://www.1000genomes.org/wiki/analysis/variant%20call%20format/vcf-variant-call-format-version-41">http://www.1000genomes.org/wiki/analysis/variant%20call%20format/vcf-variant-call-format-version-41</a> for a description of the VCF format.  |
    |  maf  |  Mutation Annotation Format  |
    |  gff  |  General Feature Format  |
    |  csv  |    |
    |  tsv  |    |

<!-- end list -->

[MD5 Checksum](#Analysis_MD5_Checksum)<a name="Analysis_MD5_Checksum"></a><span class="red">*</span>  
: MD5 checksum of a run data file. [How to obtain the MD5 checksum
values.](#supplement-md5)

## Run data files  {#run-data-files}


<div class="attention" markdown="1">
  - The DRA does NOT accept fasta only datasets. The minimum submission
  level for SRA is base/color calls with quality scores.
  - Make sure the file names are constructed only from alphanumerals
  [A-Z,a-z,0-9], underscores [_], hyphens [-] and dots [.],
  with no whitespaces, brackets, other punctuations or symbols.
  - Barcoded data files should be demultiplexed prior to submission and
  a unique BioSample should be created for each barcoded sample; in
  other words, each BioSample must be linked to one or more unique
  data files.
  - In case of fastq files, submit paired reads in separate files. For
  bam files, paired reads need to be described in single file.
  - Upload data files directly under a submission directory. Submitted
  archive files should NOT contain any directory structure.
  - Binary data formats, including BAM and HDF5 should be submitted without applying any additional compression.
</div>

### Formats of sequencing data files  {#formats-sequencing-data-files} 

The DRA metadata submission tool cannot describe technical reads
(adapter, primer and barcode sequences). "To submit raw data contain
technical reads" and "To use metadata elements in [DRA XML schema](https://github.com/ddbj/pub/tree/master/docs/dra) the but not in
the submission tool", [submitters need to create metadata in XML files](#create-metadata-in-xml-files)([XML examples](/ddbj/example.html)).

For 10x Genomics data files, please see the GEA [Single-cell submission guide](/gea/single-cell-e.html).

Generic formats

| Format           | Platform      | Recommended |
|---|
| [BAM](#BAM_file) | all platforms | Yes         |
| [fastq](#fastq)  | all platforms | Yes         |

Platform specific formats

| Format                                           | Platform            | Recommended                      |
|---|
| [SFF](#454)                                      | 454 and Ion Torrent | No (please convert to fastq/bam) |
| [PacBio HDF](#Pacific_Biosciences)               | PacBio              | Yes                              |
| [SOLiD csfasta/qual](#SOLiD)                     | SOLiD               | No (please convert to fastq/bam) |
| [Illumina qseq and scarf](#Illumina_Native_Data) | Illumina            | No (please convert to fastq/bam) |

### BAM file  {#BAM_file} 

BAM is a compressed version of the Sequence
Alignment/Map (SAM) format (see
[SAMv1.pdf](https://samtools.github.io/hts-specs/SAMv1.pdf)). BAM files
can be decompressed to a human-readable text format (SAM) using
SAM/BAM-specific utilities (e.g. [samtools](http://www.htslib.org/)) and can contain unaligned sequences as well. DRA recommends to submit BAM including unaligned reads as primary data into Run.

SAM is a tab-delimited format including both the raw read data and
information about the alignment of that read to a known reference
sequence(s). There are two main sections in a SAM file, the header and
the alignment (sequence read) sections, each of which are described
below. Note that this documentation will focus on a description of the
SAM format with respect to submission of BAM files to the DRA (i.e. DRA
doe not accept SAM files for submission). A more comprehensive
discussion of the format specifications can be found at the [samtools website](http://www.htslib.org/).

SAM Header Example:

```
@HD    VN:1.4    SO:coordinate
@SQ    SN:CHROMOSOME_I    LN:15072423
UR:ftp://ftp.ncbi.nlm.nih.gov/genbank/genomes/Eukaryotes/invertebrates/Caenorhabditis_elegans/
WBcel215/Primary_Assembly/assembled_chromosomes/FASTA/chrI.fa.gz    AS:ce10    
SP:Caenorhabditis elegans
 
@SQ    SN:CHROMOSOME_II    LN:15279345    
UR:ftp://ftp.ncbi.nlm.nih.gov/genbank/genomes/Eukaryotes/invertebrates/Caenorhabditis_elegans/
WBcel215/Primary_Assembly/assembled_chromosomes/FASTA/chrII.fa.gz     AS:ce10    
SP:Caenorhabditis elegans  
 
@RG    ID:1    PL:ILLUMINA    LB:C_ele_05    DS:WGS of C elegans    PG:BamIndexDecoder
@PG    ID:bwa    PN:bwa    VN:0.5.10-tpx
```

SAM Alignment Example:

```
3658435    145    CHROMOSOME_I    1    0    100M    CHROMOSOME_II    2716898    0    
GCCTAAGCCTAAGCCTAAGCCTAAGCCTAAGCCTAAGCCTAAGCCTAAGCCTAAGCCTAAGCCTAAGCCTAAGCCTAAGCCTAAGCCTAAGCCT
AAGCCT    
@CCC?:CCCCC@CCCEC>AFDFDBEGHEAHCIGIHHGIGEGJGGIIIHFHIHGF@HGGIGJJJJJIJJJJJJJJJJJJJJJJJJJJJHHHHHFF
FFFCCC    RG:Z:1    NH:i:1    NM:i:0
    
5482659    65    CHROMOSOME_I    1    0    100M    CHROMOSOME_II    11954696    0    
GCCTAAGCCTAAGCCTAAGCCTAAGCCTAAGCCTAAGCCTAAGCCTAAGCCTAAGCCTAAGCCTAAGCCTAAGCCTAAGCCTAAGCCTAAGCCT
AAGCCT    
CCCFFFFFHHGHGJJGIJHIJIJJJJJIJJJJJIJJGIJJJJJIIJIIJFJJJJJFIJJJJIIIIGIIJHHHHDEEFFFEEEEEDDDDCDCCCA
AA?CC:    RG:Z:1    NH:i:1    NM:i:0
```

#### BAM file processing  {#BAM_file_processing}

The header and alignment section are internally consistent: each aligned read has an RNAME (reference sequence name, 3rd field) that matches an SN tag value from the header (e.g., CHROMOSOME_I), and, if provided,the alignment read group optional field (RG:Z:) is consistent with the
read group ID in the header (1). It is also important to ensure that the
FLAG fields (2nd field in each line) are correctly set for the data. The
SRA pipeline will attempt to resolve incorrect FLAG values, but
sufficiently incorrect values can lead to processing errors. The SRA
does not archive optional and non-standard tags/field values contained
in the alignment section. However, the entire header section of the bam
file is retained. Additionally, although the SAM format allows for an
equal sign (=) in the sequence field to represent a match to the
reference sequence or only an asterisk (*) in both the sequence and
quality fields, the DRA processing software does not recognize either of these formats.
     
For 10x Genomics data files, please see the GEA [Single-cell submission guide](/gea/single-cell-e.html).

Please note that unexpected notations used to indicated paired reads can lead to failure to recognize the pairs and an improper SRA archive (i.e.
paired reads are treated like fragments). For example, using :0 and :1
at the end of the read names is atypical and is currently not recognized
as an indication of read 1 and 2 in a pair. It would be better to
exclude these notations and provide the two reads with the same names.
Expected notations for particular platforms will work. For example,
Illumina reads with /1 or /2 appended is an expected notation. Further,
neglecting to set the proper bits for paired reads in the SAM/BAM flags
(e.g. multi-segment template 1-bit, first segment 64-bit, and last
segment 128-bit) or splitting paired reads into separate bam files can
result in an improper SRA archive or failure to generate the SRA
archive.

In the case of submitting alignment data, you need to submit "BAM",
"INSDC, refseq accession number OR reference multi-fasta" and
"SN-reference mapping table". Submit one bam file per Run.

When submitting bam file into Analysis instead of Run, the mapping table
is unnecessary. However, please consider to submit bam including
unaligned reads as primary data into Run.

<div class="attention">
When submitting unmapped bam (without SQ header line) from PacBio and
IonTorrent, the mapping table and reference sequences are not necessary.
</div>

<div class="attention">
If only BAM alignment files are submitted, then please make sure that
the BAM files also contain the unaligned reads. This is critical to
enable primary re-analysis and re-alignment of the dataset using new
tools or future genome assembilies.
</div>

{% include image.html url="books/bam-mapping.jpg" caption="mapping between bam and reference sequences" class="w500" %}

1. #### BAM file submission {#BAM_file_submission}
  The alignment data can be submitted in the BAM format. The bam files should be readable by [SAMtools](http://samtools.sourceforge.net/) and [picard](https://broadinstitute.github.io/picard/). The BAM files are nearly optimal in terms of compression and should be submitted uncompressed.

2. #### Specify reference by INSDC/RefSeq accession number  {#Specify_reference_by_INSDC/RefSeq_accession_number}
  If references are found in [list](/assets/files/ref/ref-file.txt), references can be specified by their accession.version number (for example, NC_000001.11). [Version number](/ddbj/flat-file-e.html#Version)is necessary. Accession numbers for references can be searched in [NCBI Assembly](https://www.ncbi.nlm.nih.gov/assembly/).

3. #### Specify reference by supplying multi-fasta  {#Specify_reference_by_supplying_multi-fasta}
  If references are not found in the [list](/assets/files/ref/ref-file.txt), submit a reference file in multi-fasta format. Select "reference_fasta" in the [Run file type](/dra/submission-e.html#File_Type). The reference name in the bam header and reference sequence are linked by the name in bam header and fasta defline via the mapping table. If sequence length is different between @SQ-LN and multi-fasta, a warning is raised.

4. #### Specify reference by both INSDC/RefSeq accession number and multi-fasta  {#Specify_reference_by_both_INSDC/RefSeq_accession_number_and_multi-fasta}
  If a part of references are found in [list](/assets/files/ref/ref-file.txt), these references can be specified by their accession.version number (for example, NC_000001.11). The rest of references needs to be supplied by uploading a multi-fasta file. In the SN-reference mapping table, list accession.version numbers and sequence names of multi-fasta deflines.

5. #### SN-reference mapping table  {#SN-reference_mapping_table}
  A tab delimited text file describing mapping between "SN in SQ line in BAM header" and "accession OR sequence name in fasta file". Select "tab" in the [Run file type](/dra/submission-e.html#File_Type)

  BAM header

```
@HD VN:1.0 GO:none SO:coordinate
@SQ SN:chr1 LN:249698942
@SQ SN:chr2 LN:242508799
@SQ SN:chr3 LN:198450956
...
```

SN-fasta mapping table. In the example, the reference named ref1 in
multi-fasta file corresponds to the SN:chr1.

```
chr1 ref1
chr2 ref2
chr3 ref3
...
```

Reference multi-fasta.

```
>ref1
CGGTGGGGGTGGTGTTAGTACCCCATCTTGTAGGTCTGAAACACAAAGTGTGGGGTGTCT
...
>ref2
TCCACCAACGTTAGAAGGCCTTGGCCCCCAGAGAGCCAATTTCACAATCCAGAAGTCCCC
...
>ref3
GTGTGTGACCAGGGAGGTCCCCGGCCCAGCTCCCATCCCAGAACCCAGCTCACCTACCTT
...
```

SN-fasta mapping table. In the example, the reference "NC_000001.11"
corresponds to the SN:chr1.

```
chr1 NC_000001.11
chr2 NC_000002.12
chr3 NC_000003.12
...
```

### fastq  {#fastq} 

Run filetype needs to be specified depending on whether read length is
constant or not.

  - For the fastq files with constact read length, select 'fastq' in the
    [file type](#File_Type)of Run. Paired reads should appear in the
    same order in the paired files.
  - For the fastq files, select 'fastq' in the [file type](#File_Type) of Run irrespective of read length.

Format of fastq, for details, please see [NCBI
website](https://www.ncbi.nlm.nih.gov/books/NBK242622/#File_Format_Guide_BK.FASTQ).


  - Quality values must be in Phred scale. By default, 33 (!) is used
    for Phred quality offset. In the case of 64 (@), [update the
    ascii_offset of Run XML](#create-metadata-in-xml-files)to
    'ascii_offset="@"'.
  - In the DRA metadata submission web interface, technical reads (adapters, linkers, barcodes) cannot be described. When submitting fastq including technical reads, please describe technical reads in the Experiment XML according to [Formats of sequencing data files](#formats-sequencing-data-files) [(XML examples)](/ddbj/example-e.html). The Experiment XML submission is not necessary for fastq without technical reads.
  - Paired reads must split and submitted using two Fastq files. The
    read names must have a suffix identifying the first and second read
    from the pair, for example '/1' and '/2'.
  - The first line for each read must start with '@'.
  - The base calls and quality scores must be separated by a line
    starting with '+'.
  - The Fastq files must be compressed using gzip or bzip2.


### 454  <a name="454"></a> 

The DRA accepts sequencing run data from the 454 platform in the fastq/bam formats. Please convert output sff files to fastq/bam files.

### Illumina Genome Analyzer  {#Illumina-Genome-Analyzer}

#### Illumina pipeline v1.4 and later  {#Illumina_pipeline_v1.4_and_later}

DRA does not accept qseq files. Please convert qseq to fastq/bam.

### BGI-seq  {#BGI}

Submit fastq files.       

### SOLiD  {#SOLiD} 

#### SOLiD Native Format  {#SOLiD_Native_Format}

DRA does not accept SOLiD native files. Please convert the native files
to fastq/bam.

### Ion Torrent  {#Ion-Torrent}

Submit Ion Torrent data in the fastq format. Bam files from Ion Torrent instruments can be converted to fastq by using samtools. [Converting BAM to fastq](https://www.metagenomics.wiki/tools/samtools/converting-bam-to-fastq)

### Helicos Heliscope  {#Helicos-Heliscope}

Submit Helicos data in the sms（helicos_native) or fastq/bam format
created with the fixed-quality value, "14".

### Complete Genomics  {#Complete-Genomics}

Submit Complete Genomics data in the fastq/bam format.

### Pacific Biosciences  {#Pacific-Biosciences}

#### HDF5  {#hdf5}

Pacific Biosciences uses HDF5, a container file with a directory-like
structure, to store raw data. The DRA accepts both bas.h5 and bax.h5
file submissions. Note that submission of data from the RS II instrument
requires one Run consists of one *.bas.h5 file and three *.bax.h5
files. Do not rename files.

<div class="attention">
Do NOT include files other than HDF5 in a Run.
</div>
  
<div class="attention">
The current DRA system (9th July 2021) is not able to load the HDF5 files. Please submit fastq instead of HDF5.
</div>

#### bam  {#pacbio_bam}

We support the submission of the following types of PacBio bam files. Include 1 bam file per Run. For an unaligned bam file, reference and mapping table are not necessary.

- *.subreads.bam
- *.ccs.bam
- *.reads.bam

#### fastq  {#pacbio_fastq}

The DRA also accepts Pacific Biosciences data in the
[fastq](#fastq) format. Select the ["fastq"](#fastq) for the Run filetype.

### Oxford Nanopore  {#Oxford-Nanopore}

Submit Oxford Nanopore data in the fastq/bam format.

### Capillary sequencing platform  {#Capillary-sequencing-platform}

Submit capillary sequencing data in the fastq/bam format.

## Analysis data files  {#analysis-data-files}

### PacBio Base Modification Files  {#Pacbio-Base-Modification-Files}

[PacBio](http://www.pacb.com)sequence data also permits the analysis of
methylated bases within the sequence, which can be extremely helpful to
the scientific community. For example, the precise positions of those
modified bases can be used to determine the specificity of the DNA
methyltransferases that produced them. The PacBio analysis suite
contains an analysis workflow (RS_Modification_and_Motif_Analysis)
to extract these sequences and produce several files:

- motif_summary.csv
- modifications.csv
- modifications.gff
- motifs.gff

It would be beneficial to the scientific community if you were able to
perform this analysis and submit at least the motif_summary.csv file
for prokaryotes via as a DRA Analysis object. Please submit these files
as data files of the Analysis with [Sequence Annotation
type](#Analysis_Type)in addition to sequencing reads in Run. For
assistance, [contact us](/contact-ddbj-e.html).

[NCBI guidelines of PacBio Base Modification Files](https://www.ncbi.nlm.nih.gov/genbank/basemodificationfiles)

### BioNano Whole Genome Map Files {#bionano}

BioNano mapping technology produces whole genome maps. These maps can be used in a variety of genomic analyses, including de novo assembly, structural variant detection and assembly curation. For example, BioNano physical maps can be integrated with de novo genome assemblies produced from next-generation technologies to produce high quality hybrid assemblies with increased continuity and completeness, especially in regions of genomic complexity. Files produced as part of the BioNano mapping and or hybrid assembly process include:

- CMAP
    - The BioNano Genomics Irys .cmap file is a raw data view of a molecule set or assembly reporting a label site position within a genome map identified during a run. The Irys .cmap file reports the start and end coordinates and the locations of the labels on a map using a tab-delimited text based file.
- COORD
    - The purpose of the .coord file is to relate the coordinates of scaffolds in a hybrid assembly to the corresponding AGP submission. The .coord file maps positions from the hybrid cmap, which may not begin or end with sequence gaps. The scaffolds are trimmed up to the leftmost label of leftmost sequence and the rightmost label of the rightmost sequence.
- XMAP
    - The BioNano Genomics Irys .xmap file is a cross-comparison between two maps. The Irys .xmap file reports the comparison derived from the alignment between an anchor .cmap file and a query .cmap file. The data line displays the map start and end coordinates and the locations of the labels on the map using a tab-delimited text based file.
- SMAP
    - The BioNano Genomics Irys .smap file is a description of structural variations (SV) detected between two genome maps. The Irys .smap file reports the structural variants discovered during an alignment between an anchor .cmap file and a query .cmap file. The data line displays the start and end coordinates and the locations of the SV on the map using a tab-delimited, text-based file.
- BNX
    - The BioNano Genomics Irys .bnx file is a raw data view of molecule and label information and quality scores per channel identified during a run.

For the latest file specifications, please see the [BioNano GitHub site](https://github.com/bionanogenomics/File-Format-Specification-Sheets).

If you are using BioNano data as part of your assembly generation pipeline, it would be extremely useful to the scientific community if you could submit a package comprised minimally of the molecule .bnx file and the resulting de novo assembly file EXP_REFINEFINAL1.cmap and COORD files as a DRA Analysis. We will add an analysis type and filetypes for the BioNano Genome Map files. In the meantime, please submit the BioNano files as the analysis type "[De Novo Assembly](#Analysis_Type)" and the filetype "[tsv](#Analysis_File_Type)" (Example, [DRZ011299, DRZ011300](https://ddbj.nig.ac.jp/public/ddbj_database/dra/fastq/DRA005/DRA005897/DRA005897.analysis.xml)). 

## Submission to the DRA  {#dra-submission}

<div class="attention">
Never submit data without the permission of the principal investigator.
</div>

<div class="attention" markdown="1">
Submission of research data from human subjects  
: <span class="red">For submitting data from human subjects (human data) to the
databases of DDBJ center, it is submitter's responsibility to ensure
that the dignity and right of human subject are protected in
accordance with all applicable laws, ordinances, guidelines and
policies of submitter's institution. In principle, make sure to
remove any direct personal identifiers of human subjects from your
data to be submitted. Before submitting human data, read the
"[Submission of research data from human
subjects](/policies-e.html#submission-of-human-data)".</span>
</div>

<div class="attention" markdown="1">
Submission of Patent Related Sequences  
: <span class="red">Please read ["Submission of Patent Related Sequences" and "Patent
Priority and Other Priority"](/policies-e.html#disclaimer)before
submitting patent related sequences.</span>
</div>

Metadata and sequence data are required for submission to the DRA.

Please submit the assembled sequence data to the [DDBJ](../). The [DDBJ
Mass Submission System (MSS)](/ddbj/mss-e.html)accepts the genomic or
abundant sequence data generated by massively parallel sequencing
platforms.

## Data submission to DRA  {#dra-data-submission}

### 1. Obtain a submission account  {#obtain-account}

  - Create a [D-way submission account](https://ddbj.nig.ac.jp/D-way/)
  - To enable DRA submission, [register a public key and a center name
    to the account](/account-e.html#enable-dra-submission-in-account)

### 2. Create a DRA submission and upload data files  {#upload-data-files} 

  - Create a new DRA submission ( [Add DRA submission functionality to
    your account](/account-e.html#enable-dra-submission-in-account))  
    All sequencing data in single submission will be released at the
    same time.
  - Upload data files by scp before submitting BioProject, BioSample,
    Experiment and Run

### 3. Submit project and sample information  {#project-sample}

#### [BioProject](/bioproject/submission-e.html)  {#BioProject_Study}

  - A description of the reseach effort
  - "Why" you sequenced your samples

#### [BioSample](/biosample/submission-e.html) <img src="/assets/images/parts/tsv.png" alt="" title="" class="tsv"> {#BioSample_Sampple}

  - A description of biologically or physically unique specimens
  - "What" you sequenced

<img src="/assets/images/parts/tsv.png" alt="" title="" class="tsv">metadata can be submitted as a tab-delimited text file

### 4. Submit Experiment and Run {#submit-object} 

#### DRA Experiment <img src="/assets/images/parts/tsv.png" alt="" title="" class="tsv"> {#DRA_Experiment}
  - A description of a sample-specific sequencing library
  - "How" you performed the sequencing
  - Multiple Experiments “point” to a single Sample, but not vice-versa.

#### DRA Run <img src="/assets/images/parts/tsv.png" alt="" title="" class="tsv"> {#DRA_Run}

  - Validate data files after submitting Experiment and Run
  - All files linked to a Run are “merged” into a single SRA file format

### 5. Validate sequencing data files {#validate}

  - Start to convert sequencing data files into a SRA file for
    archiving.
  - Submission passed validation step will be reviewed and accessioned.

## How to submit data to the DRA  {#how-to-submit-dra-data}

Submission to BioProject/BioSample/DRA (6 min 50 sec、created:2015)

<iframe width="560" height="315" src="https://www.youtube.com/embed/Hg4h8ytbIV4" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>

### Submission Account  {#submission-account}

At the [DNA Data Bank of Japan (DDBJ) center](../),
[BioProject](/bioproject/index-e.html),
[BioSample](/biosample/index-e.html), and
[DRA](/dra/index-e.html)submissions are managed in user's account.

According to the [Submission Account Handbook](/account-e.html), obtain
a submission account and enable DRA submission in the account.

### Organize data  {#organize-data}

Examples of metadata object organization is
[here](#Organization_of_metadata_objects). In single submission, only
one BioProject can be registered. Multiple BioSample, Experiment, Run
objects can be registered. To easily organize your data into a
submission, please first consider number of BioSamples.

In this chapter, submission steps are explained by submitting a example
submission "paired-end genome sequencing of three bacterial strains".

{% include image.html url="books/datamodel1.jpg" caption="Genome sequencing data of three bacterial strains" class="w450" %}

### Create a new submission  {#create-new-submission}

Login [the D-way (https://ddbj.nig.ac.jp/D-way/)](https://ddbj.nig.ac.jp/D-way/) and the top page is displayed. Move to the DRA submission site from the “DRA” menu at the top.

Create a new submission by clicking the [New submission]. At this
time, in the DRA file server (ftp-private.ddbj.nig.ac.jp), the
corresponding subdirectory is created under the submitter’s home
directory. [Upload sequence data files to this subdirectory](#upload-sequence-data).

<div class="attention">
If there is no reply from submitters after three months of initial
contact, submissions will be cancelled.
</div>

<div class="attention">
All data in a submission are released at the same time. If you want to release data at different time, please divide a submission.
</div>

<div class="attention">
Maximum numbers of objects per submission are, BioSample:1,000, DRA:2,000 (Runs) and GEA:1,000 (Assays). 
If you have objects more than these limits, please create multiple submissions with the same BioProject reference.
</div>

{% include image.html url="books/hdra01.jpg" caption="Create a new submission" class="w450" %}

List of submission status is as follows. The DRA team reviews submission
whose status is in "submission_validated" or "data_error".

List of submission status
{: .tablecaption}

| Status                | Explanation                                                   |
|---|
| New                   | Metadata has not been submitted.                              |
| metadata_submitted   | Metadata has been submitted.                                  |
| data_validating      | Validating data files.                                        |
| data_error           | Error occurred in data validation process.                    |
| submission_validated | Metadata and data have been validated.                        |
| completed             | Accession numbers have been issued.                           |
| confidential          | Archive files has been created and submission is kept private |
| Public                | Released to public.                                           |

## Upload sequence data  {#upload-sequence-data} 

Upload sequencing data files to the corresponding DRA submission directory on the file server.   
Regarding how to upload your data files, please see "[Data upload](/upload.html)".

## Submit metadata {#submit-metadata}

You may submit the metadata in two ways, one is ["Submit metadata by the web tool"](#metadata-web) and second is ["Submit metadata by the excel"](#metadata-excel).    
When it is difficult to submit large-scale metadata (exceeds 100 Runs) by using the web tool whose resnposen is slow, it is recommended to submit the metadata by the metadata excel.    

### Submit metadata by the web tool  {#metadata-web} 

Move to the submission detail page by clicking the submission ID.

{% include image.html url="books/hdra02.jpg" caption="Move to the submission page" class="w450" %}

Click the [Enter / Update metadata] button to run the DRA metadata
creation tool.

{% include image.html url="books/hdra03.jpg" caption="run the DRA metadata creation tool" class="w450" %}

When no file is uploaded to the submission directory, following message
is displayed. To submit metadata, please [upload data
files](#upload-sequence-data).

To submit metadata first, upload some files (for example, empty text
file).

{% include image.html url="books/hdra-not-uploaded.jpg" caption="when no data file is uploaded" class="w200" %}

<div id="submission-excel2xml" class="attention" markdown="1">
When there are many Experiment and Run objects, create metadata XMLs by using [the excel for the DRA metadata](/assets/files/submission/dra_metadata.xlsx) and the XML generator. The metadata can be registered by uploading the Submission/Experiment/Run XMLs in D-way. Please see the [GitHub page](https://github.com/ddbj/submission-excel2xml) for details.
</div>

The metadata are composed of the Submission, BioProject, BioSample,
Experiment, Run, Analysis (optional) objects. In the metadata creation
tool, enter content from left to right tabs.

Required items are marked with <span class="red">*</span>.

The entered content is checked when submitters click the [Save] button
or before moving to the other tab. When error messages are displayed,
please revise the content.

#### Submission  {#Submission}

Set the hold date within two years. Include principal investigator(s)
and submitter(s) who actually submit data in the Submitter. The DRA dose
not disclose the submitter information to public.

<div class="attention">
All data in a submission are released at the same time. If you want to
release data at different time, please divide a submission.
</div>

{% include image.html url="books/hdra-submission01.jpg" caption="Enter metadata in the tool" class="w450" %}

#### Study  {#study}

<div class="attention">
When the Study and Sample tab contents are not displayed, please open them by using Edge or Firefox.
</div>

Submit a new project by clicking [New submission], or select a project
registered in the account.

Only one project can be submitted. To reference a project obtained in
the other account, please [contact DRA team](/contact-ddbj-e.html).

{% include image.html url="books/hdra-bp01.jpg" caption="Submit a new BioProject or select submitted one" class="w450" %}

To submit a BioProject, enter content from left to right tabs. The
second panel is for BioProject submission. Submitter information is
copied with that of DRA submission.

For BioProject metadata, please see the [BioProject
Handbook](/bioproject/submission-e.html#metadata).

{% include image.html url="books/hdra-bp02.jpg" caption="BioProject submission" class="w450" %}

To submit data corresponding to personal identification code to DRA/GEA/DDBJ, [your data submission application needs to be approved by NBDC before the submission](/policies-e.html#unrestricted-access). If your application was approved, write the NBDC application ID (for example, J-DS000001-001) in "Private comments to DDBJ staff" of BioProject.

{% include image.html url="books/bp-nbdc-approval.jpg" caption="NBDC data submission application ID" class="w450" %}


To submit genome assemblies to DDBJ, a unique [Locus tag
prefix](/ddbj/locus_tag-e.html) is necessary.

Locus tag prefix generation box will appear when [Project data
type="Genome Sequencing" or "Metagenome"] AND [Capture="Whole"] AND
[Objective="Sequence" or "Annotation" or "Assembly"]. Registration of
a unique locus tag prefix is required for studies that result in genome
assemblies.

The locus_tag prefix can contain only alpha-numeric characters and it
must be at least 3-12 characters long. It should start with a letter,
but numerals can be in the 2nd position or later in the string. (ex.
A1C). There should be no symbols, such as -_* in the prefix. The
locus_tag prefix is to be separated from the tag value by an underscore
‘_’, eg A1C_00001.

Please leave the prefix box empty, when a prefix is not necessary for
[WGS](/ddbj/wgs-e.html) only submission.

Prefix is managed by NCBI. When a project is submitted, our system tries
to reserve prefix to NCBI. When the prefix has already been reserved, an
error message will be displayed. Please enter a different prefix and
submit again.

<div class="attention">
When multiple prefixes are necessary, please [contact
us](/contact-ddbj-e.html).
</div>

{% include image.html url="books/hdra-bpltag.jpg" caption="Reserve locus tag prefix" class="w450" %}

Check the content in "OVERVIEW" and submit a project by clicking
[Submit BioProject].

{% include image.html url="books/hdra-bp03.jpg" caption="Submit BioProject" class="w450" %}

After submitting a project, submitted one is selected in Study.

{% include image.html url="books//hdra-bp04.jpg" caption="Submitted project is selected" class="w450" %}

#### Sample  {#sample}

<div class="attention">
When the Study and Sample tab contents are not displayed, please open them by using Edge or Firefox.
</div>

Submit new samples by clicking [New submission], or select samples
submitted in the account.

<div class="attention">
Upper limit is about 2,000 samples per submission.
</div>

To select a range of samples, first check a checkbox and click next box
with pressing the "Shift". Filter samples by entering text in the upper
box, and click [Select filtered BioSamples] to select all filtered
samples.

To reference samples obtained in the other account, please [contact us](/contact-ddbj-e.html).

{% include image.html url="books/hdra-bs01.jpg" caption="Submit new samples or select submitted ones" class="w450" %}

To submit a BioSample, enter content from left to right tabs. The second
panel is for BioSample submission. Submitter information is copied with
that of DRA submission.

Biological replicates are represented by separate
BioSamples. Regarding necessary number of sample for sequence
submission, please see the ["FAQ: How many samples do I need for my DRA
submission?"](/faq/en/samples-for-sra.html)

For BioSample metadata, please see the [BioSample
Handbook](/biosample/submission-e.html#metadata).

{% include image.html url="books/hdra-bs02.jpg" caption="BioSample submission" class="w450" %}

Select a sample type in the "SAMPLE TYPE". For genome samples, minimum
sample attributes are defined by
[MIxS](/biosample/submission-e.html#sample-attributes).

For the Sample type, please see the [BioSample
Handbook](/biosample/submission-e.html#Sample_type).

{% include image.html url="books/hdra-bs03.jpg" caption="Select a sample type" class="w450" %}

Download a template text file according to the selected sample type to
enter sample attributes.

A main sample submission step is to describe samples by required,
optional and user-defined attributes.

[BioSample attribute list](/biosample/attribute-e.html). User-defined
attributes can be added at rightmost column. Please see the "[Human Sample](/biosample/human-sample-e.html)" page about human samples.

[BioSample submission file examples](https://docs.google.com/spreadsheets/d/1VCCuSwvIRfp5-DT8cnvvAwWH4C7wbDFSjHQ_q3f3BII/edit#gid=1811256482)

A text file is separated by tab and can be opened and editted in
spreadsheet editor (e.g. Excel®). Attribute names are in a header line.
Attributes with "*" are required.

From second lines, enter one sample per line. Enter PSUB submission id
in [bioproject_id](/biosample/attribute-e.html?all=all#bioproject_id)
for project without PRJD accession numbers. For attributes without
measured values, enter "missing" or "not applicable".

{% include image.html url="books/hdra-bs04.jpg" caption="Download a text file for entering sample attributes" class="w450" %}

Upload the BioSample submission file by selecting the file and clicking
the Continue button. The validator checks the uploaded file and
feedbacks error and warning messages. Submitter can not submit the
BioSample until all errors are resolved.

For the validation rules and messages, please see [Validation rules
page](/biosample/validation-e.html).

{% include image.html url="books/hdra-bs04-2.jpg" caption="BioSample validation. In this example, an error for the future date in the collection_date and a warning for inconsistent countries between geo_loc_name and lat_lon of the sample [genome bacteria strain C] are displayed." class="w500" %}

Check content in the last "OVERVIEW" and submit samples. In the
"ATTRIBUTES" area, the submitted sample attribute file can be
downloaded.

{% include image.html url="books/hdra-bs05.jpg" caption="Submit BioSample" class="w450"%}

After submitting BioSamples, submitted BioSamples are selected in the
"Sample" tab.

{% include image.html url="books/hdra-bs06.jpg" caption="Submitted BioSamples are selected" class="w450"%}

#### Experiment  {#experiment}

Experiment and Run as same as selected BioSamples are automatically
created. Each BioSample,Experiment and Run are referenced. The
Experiment and Run are automatically generated when the Experiment tab
is initially displayed.

| BioProject | - BioSample (1) | - Experiment (1) | - Run (1) |
|            | - BioSample (2) | - Experiment (2) | - Run (2) |
|            | - BioSample (3) | - Experiment (3) | - Run (3) |

In this example, 3 Experiments are created and each Experiment reference
unique BioSample.

Add an Experiment by clicking the [Add new Experiment(s)] and delete
an Experiment by clicking the [Delete]. Experiment referenced by Run
cannot be deleted.

{% include image.html url="books/hdra-experiment01.jpg" caption="Experiment referencing selected BioSample, is automatically created" class="w450" %}

Experiments can be submitted in a tab-delimited text file. First save
and fix Aliases (e.g., test07-0040_Experiment_0001 - 0003) by clicking
the [Save]. Alias is used as a name until accession numbers are
issued.

Download content into a tab-delimited text file by clicking the
[Download TSV file].

{% include image.html url="books/hdra-experiment02.jpg" caption="Save, fix aliases and download as a tab-delimited text file" class="w450" %}

Metadata can be editted in spreadsheet software (e.g. Excel®).

If "Title" values are empty, titles are automatically constructed as
"[Sequencing Instrument Model] [paired end] sequencing of
[BioSample ID]" (e.g., "Illumina HiSeq 2000 paired end sequencing of
SAMD00025741"). Submitters can provide user-defined text in the "Title".

Reference samples in "BioSample Used" by "SSUB BioSample Submission ID"
: "Sample name" (example, SSUB003746 : Genome bacteria strain A). Spaces
around ":" are ignored.

{% include image.html url="books/hdra-experiment03.jpg" caption="Experiment template file" class="w450" %}

Save editted content in a tab-delimited text file and select and upload
it by clicking the [Upload TSV file].

{% include image.html url="books/hdra-experiment04.jpg" caption="Upload Experiment in a tab-delimited text file" class="w450" %}

<div class="attention">
Upload in tab-delimited text file and NOT in spreadsheet software
specific format.
</div>

#### Run  {#run}

Experiment and Run as same as selected BioSamples are automatically
created. Each Run references unique Experiment.

In this example, three Runs are created and each Run references unique
Experiment.

Add Run by clicking the [Add another Run(s)] and delete Run by
clicking the [Delete]. Run linked to files cannot be deleted.

{% include image.html url="books/hdra-run01.jpg" caption="Save and fix Aliases" class="w450" %}

After fixing aliases by clicking the [Save], run content can be
downloaded into a tab-delimited text file. To distinguish the data files
for Run, enter "Run" in the leftmost "Run/Analysis" column.

Click the [Select data files for Run] and link uploaded files to Run.

{% include image.html url="books/hdra-run02.jpg" caption="Move to next site to link files to Run" class="w450" %}

All files uploaded to the submission directory are shown. Associate a
file to a Run by selecting a Run alias in "Run/Analysis contains files".

Enter [File type](#File_Type) and [MD5 Checksum](#supplement-md5) for
files. File attributes can be entered by uploading a tab-delimited text
file.

<div class="attention">
Note that all data files listed in a Run will be merged into a single
SRA archive file, so files from different samples or replicates should
not be grouped in the same Run. Paired-end data files, conversely, MUST
be listed in a single run in order for the two files to be correctly
processed as paired-end.
</div>

<div class="attention">
For fastq files, select "fastq" for filetype irrespective of read length.
</div>

{% include image.html url="books/hdra-runfinish.jpg" caption="Enter file attributes and link files to Run" class="w450" %}

When an Analysis (optional) is unnecessary, submit metadata by clicking
the [Submit/Update DRA metadata].

{% include image.html url="books/hdra-submit01.jpg" caption="Submit DRA metadata" class="w450" %}

After submitting DRA metadata, start validation of data files. Click the
link "Validate uploaded data files to finish this submission".

{% include image.html url="books/hdra-submit02.jpg" caption="Go to data validation after submitting metadata" class="w450" %}

#### Analysis (optional)  {#analysis}

Create Analysis as many as required, enter content of each Analysis.
Unnecessary Analysis can be deleted by clicking the [Delete].

Click the [Select data files for Analysis] and link files to Analysis.

{% include image.html url="books/hdra-analysis01.jpg" caption="Enter Analysis content" class="w450" %}

Enter file attributes and associate them with Analysis. When submitting
the file attributes by uploading the tab-delimited text file, to
distinguish the data files for Analysis, enter "Analysis" in the
leftmost "Run/Analysis" column.

{% include image.html url="books/hdra-analysisfile01.jpg" caption="Enter file attributes and link files to Analysis" class="w450" %}

Submit DRA metadata by clicking the [Submit/Update DRA metadata] and
proceed to data validation process. Only md5 of analysis files are
checked during validation.

<div class="attention">
For large number of analysis, please submit them by using [Analysis metadata excel](/dra/analysis-e.html).
</div>

### Submit metadata by the excel {#metadata-excel}

Sometimes it is difficult to submit large-scale metadata (exceeds 100 Runs) by using the web tool whose response is too slow, 
please submit the metadata by the excel.

Before filling in the metadata excel, you need to finish followings.

* [Submit BioProject](#study)
* [Submit BioSample](#sample)
* [Create a new DRA Submission](#create-new-submission)
* [Upload sequencing data files](#upload-sequence-data)

Download [the DRA metadata excel](/assets/files/submission/dra_metadata.xlsx) and describe your metadata. [Example excel](/assets/files/submission/example-0001_dra_metadata.xlsx) 

Next, [upload XMLs generated from the excel](#upload-xml) or [send the excel by email attachment](#send-metadata-excel).

#### Upload XMLs generated from the excel {#upload-xml}

Please upload XMLs if you are familiar with command lines using container images.

You can submit metadata by uploading XMLs in the D-way submission page by using [the metadata excel](/assets/files/submission/dra_metadata.xlsx) and container images.   
Generate metadata XMLs according to the [GitHub](https://github.com/ddbj/submission-excel2xml) page.

Login [D-way](https://ddbj.nig.ac.jp/D-way) and move to the DRA submission page.  
Following is an example of uploading the Submission/Experiment/Run XMLs to the DRA submission "test07-0040".

<a href="/assets/images/books/hdra-xmlupload.jpg" title="Upload metadata XMLs" class="group1"><img src="/assets/images/books/hdra-xmlupload.jpg" alt="Upload metadata XMLs" title="Upload metadata XMLs" class="w500"></a>

** Web browser may time-out. However, submission processes are ongoing in backgroupd, so please leave it until the submission status becomes "metadata_submitted". If the status is unchanged even after a day, please contact us ([FAQ](/faq/en/dra-xml-timeout-e.html)). **   
After the submission status becomes "metadata_submitted", move on to the [data file validation](#validation-data-files) step.  
Avoid uploading XMLs many times after the time-out.   
Please [contact us](/contact-e.html) if you are not sure about the status of your submission.

#### Send the excel by email attachment  {#send-metadata-excel}

Send us the excel by email attachment if you are not familir with command lines.

Send your metadata excel with DRA submission ID to the following address.
<img src="/assets/images/center/trace-m.jpg">   
DRA curator generates XMLs and upload them instead of you.  
After uploading the XMLs, the curator send backs the metadata in a table file. 
Please check the file and proceed to the [data file validation step](#validation-data-files) if the file is correct.

### Validation of data files  {#validation-data-files}

Submitted data files are converted to the SRA files for archiving.
During this conversion process, MD5 value, file format and integrity
between files and metadata are validated.

In the “Data Files”, filenames in the Run and Analysis, MD5 values in
the Run and Analysis and those of uploaded files, are displayed.

Click the [Validate data files] and validate uploaded data files.

{% include image.html url="books/hdra-validate.jpg" caption="UStart validationo of data files" class="w450" %}

The files are validated in the following order.

[FAQ: How to deal with validation
errors?](/faq/en/data-validation-error)

#### MD5 Check  {#MD5_Check}

Consistency between the MD5 values in the metadata and of uploaded files
are checked. Inconsistency in the MD5 values cause errors. When MD5
errors occur, revise metadata and re-upload files.

#### Data Check  {#Data_Check}

Submitted data files are converted to the SRA files for archiving.
During this conversion process, MD5 value, file format and integrity
between files and metadata are validated. When errors occur, revise
metadata and re-upload files. Validation of large files takes time.

If no errors occur, submission status become "submission_validated",
and validated files are moved to separate directory.

The DRA staff review submissions with status "submission_validated".
Please do not touch submissions until the DRA staff contact submitters.

#### Revise a submission with "data_error"  {#Revise_a_submission_with_data_error}

Any errors in the validation process make the submission status to
"data_error". Revise metadata and/or re-upload data files after
stopping the validation by clicking the [Stop validation] button.
After revision, click the [Validate data files] button and start
validation again.

[FAQ: How to deal with validation errors?](/faq/en/data-validation-error)

{% include image.html url="books/hdra-error.jpg" caption="Stop validation" class="w450" %}

Submission status is backed to "metadata_submitted". Revise and
re-submit metadata or re-upload data files.

{% include image.html url="books/hdra-edit.jpg" caption="Revise submission" class="w450" %}

### Accession numbers  {#accession-numbers}

When both the metadata and sequence data are validated (Status
“submission_validated”), accession numbers with the prefix DR
(Submission (DRA)，Experiment (DRX)，Run (DRR)，Analysis (DRZ)) are
assigned ("acc_issued", "complete" or "private"). Accession numbers are
displayed in the “Component”.

### Limited-time access to archived fastq/SRA files  {#fastq-sra-files}

To allow submitter to download and check archived fastq/SRA files, the
files are copied to the following directories on the
ftp-private.ddbj.nig.ac.jp server. To save disk space, the copied files
are automatically deleted in one month.

<div class="attention">
Due to unexpected decrease of available disk space, copied fastq/SRA
files may be deleted within one month or the copy service may be
suspended. We will inform submitters on the website in advance as much
as possible, however, this annoucement could be immediately before the
deletion or service suspension.
</div>

  - (submitter's home)/report/dra/(DRA submission accession)/fastq/
  - (submitter's home)/report/dra/(DRA submission accession)/sra/

例

  - submitter/report/dra/DRA000001/fastq/DRR000001.fastq.bz2
  - submitter/report/dra/DRA000001/fastq/DRR000002.fastq.bz2
  - submitter/report/dra/DRA000001/fastq/DRR000002_1.fastq.bz2
  - submitter/report/dra/DRA000001/fastq/DRR000002_2.fastq.bz2
  - submitter/report/dra/DRA000001/sra/DRR000001.sra
  - submitter/report/dra/DRA000001/sra/DRR000002.sra

### Data release  {#data-release}

After the registered data is loaded into the database, the Status
becomes “complete (private)” and the submission is kept private until
one of the following conditions are met.

<div class="attention">
All data in a submission are released at the same time. If you want to
release data at different time, please divide a submission.
</div>

- A. Submitter requests to release their data.
- B. Submitter has published their accession number(s) and it has been
confirmed.  
We do not release the data when its accession number(s) has been
published wrongly by other than the submitter.  
"publish" means to disclose accession number(s) to the public
through paper, thesis, academic meeting, internet, press report etc.
- C. Specified hold-date has come.
- D. DDBJ/EMBL-Bank/GenBank records (e.g., [TSA](/ddbj/tsa-e.html),
[WGS](/ddbj/wgs-e.html), [CON](/ddbj/con-e.html)etc.) citing DRA Run
(DRR) accession number(s) have been made public.

Data are released without permission from submitters in the cases B, C
and D. In the case D, an entire DRA submission contains cited DRR Run(s)
is made public.

FAQ: [How are linked BioProject/BioSample/sequence data
released?](/faq/en/bp-bs-seq-release.html)

When the data is released, in a few days, the released data will become
searchable at [DDBJ Search](https://ddbj.nig.ac.jp/search)and the data
will be mirrored to the [NCBI
SRA](//www.ncbi.nlm.nih.gov/Traces/sra/sra.cgi).

The list of available fastq files at the DRA file server:
[fastqlist](https://ddbj.nig.ac.jp/public/ddbj_database/dra/meta/list/fastqlist)

FAQ: [How to request data release?](/faq/en/request-release-e.html)

## Update submission  {#update-submission}

### Update in each database  {#update-in-each-database}

|  Database  |  Update  |
|---|
|  [Annotated sequence database](/index-e.html)  |  [Request updates from web form](/ddbj/update-form-e.html)  |
|  Sequence Read Archive (DRA)  |  [Login D-way and update metadata](#change-hold-date)<br/>[To add or delete sequencing data, request updates from web form](/contact-ddbj-e.html)  |
|  BioProject/BioSample  |  [Request updates from web form](/contact-ddbj-e.html)  |

### Change hold date  {#change-hold-date} 

You can set the hold date for a maximum of 4 years and can change it. To
change the hold date, click the [Change] button in the Hold Date and
move to hold date change page.

<a href="/assets/images/books/hold_date.jpg" title="Change hold date" class="group1"><img src="/assets/images/books/hold_date.jpg" alt="Change hold date" title="Change hold date" class="w400"></a>

To immediately release the submission, click the [Release Now]. In the
middle of the night, the submission is released, data files will be made
available at [ftp](https://ddbj.nig.ac.jp/public/ddbj_database/dra/fastq/)and metadata will be indexed by the [DDBJ search system](https://ddbj.nig.ac.jp/search)in a few days.

### Update metadata  {#update-metadata}

Update metadata by clicking the [Enter / Update metadata] button. A
part of fields are blocked from editing. After editing your metadata,
please be sure to click the [Submit/Update DRA metadata] button and
reflect the updates to the DRA server.

### Add data files  {#add-data-files}

Data files cannot be directly added to the archived Run. In another DRA
submission, create new Experiment-Run objects referencing existing
BioProject and BioSample records to add data files.

Similar to Run, data files cannot be directly added to the archived
Analysis. To replace archived Analysis, please [contact to the DRA
team](/contact-ddbj-e.html).

Login [D-way](https://ddbj.nig.ac.jp/D-way/)and create a new submission
by clicking the [New submission]. Select the BioProject and BioSample
IDs to which data to be added. Next, add the DRA Experiment and Run
objects.

  - To add a new sample, share a BioProject ID and create a BioSample -
    Experiment - Run in a new DRA submission.
  - To add data files to existing sample, share BioProject and BioSample
    IDs and create an Experiment - Run in a new DRA submission.

Submit metadata and validate the appended data files. Accession numbers
will be issued to the appended Experiment/Run objects.

<div class="attention">
The BioProject ID remains same, but different DRA submission number is
assigned.
</div>

{% include image.html url="books/add_data_files.jpg" caption="Add data files" class="w400" %}

{% include image.html url="books/add_run.jpg" caption="Add data files to existing sample" class="w400" %}

To add data files to the existing DRA submission, please [contact us](/contact-ddbj-e.html).

### Withdraw archived objects  {#withdraw-archived-objects}

To withdrawing archived Experiment, Run and Analysis objects, please
[contact us](/contact-ddbj-e.html).

## MD5 checksum value {#md5} 

See "[MD5 checksum value](/checksum.html)" for how to obtain MD5 checksum values.

