---
layout: simple
title: 'Coping with the RNA World  -- The Revision of Feature Table Definition to Respond to the Demands of the Present Age -- '
category: news
db:
  - ddbj


date: 2007-12-06T00:00:00+09:00
retire_time: 2007-12-06T00:00:00+09:00
lang: en
---

<dd>Researchers who use DDBJ may mainly concern with genomic sequences, mRNA transcripts and protein-coding genes. Of course some researchers focusing molecular taxonomy may also be interested in rRNA sequences. Is it sufficient for the sequence database to prepare 4 types of features for mRNA, CDS, rRNA and, source (the origin of the sequence), then ?

    <p>In fact, there are 62 feature keys that are officially defined by DDBJ/EMBL/GenBank in the <a href="/ddbj/full_index-e.html">Feature Table Definition Document</a> (FT-Doc). This FT-Doc has been revised every April and October. Before the last revision in October 2007, there were 65 feature keys. It does not mean that we, DDBJ/EMBL/GenBank, abolished three old feature keys. We removed five old feature keys and created two new feature keys after very careful thinking. We do not only collect and provide sequence data submitted, but also make every effort to keep up with the progress of research and development in biology.<br>To discuss the revision of FT-Doc and some other issues, we have held the International Collaborators Meeting every year (ref: <a href="/activities/index-e.html#2007">The Report for the 20th International Collaborators Meeting</a>). </p>

    <p>One of the major topics at the meeting this year (2007) was "How to describe the diverse RNA transcripts?". Recently, many new RNA transcripts (miRNA, siRNA, and so on) were found and elucidated. Before the revision of FT-Doc, such new RNA genes were described by using the old feature key, misc_RNA that meant "other RNA transcript". It was actually a tentative resort for a rapidly developing research area. We thought it was the time to reflect the fruit of the research to the definition of feature keys.</p>

    <p>We, DDBJ/EMBL/GenBank, worked out the problem by addition of two RNA feature keys, ncRNA and tmRNA. The number of non-protein-coding RNA families is supposed to continue to increase, so, they will be classified into ncRNA feature. Also, three old feature keys, snRNA, snoRNA, and scRNA are merged into ncRNA by this month, December 2007. Some of old data with misc_RNA features will be modified with ncRNA or tmRNA, respectively.</p>

    <p>We call such modifications as fitting legacy data to new rules "retrofit". Every year, we apply new feature rules to the data schema by October, and retrofit the data by December. The retrofit in December 2007 is symbolic of our struggle against a paradigm shift related to RNA world.</p>

    <p>(The below is cited from <a href="/activities/index-e.html#2007">"The Report for the 20th International Collaborators Meeting"</a>)<br> </p>

    <ul><br>
        <li>New ncRNA feature <br>A variety of new types of RNA transcripts, "miRNA", "siRNA", and so on, have been introduced in recent years. Because the number of non protein coding RNA families is quite likely to continue to expand, a<br>new ncRNA feature that can flexibly accommodate them will be introduced.<br> <br>Furthermore, snRNA, snoRNA, and scRNA features are merged into ncRNA feature by December 2007.
<dd>
    <li>New /ncRNA_class qualifier<br> <br>The new feature, ncRNA, will utilize a new qualifier called /ncRNA_class, with <a href="http://www.insdc.org/page.php?page=rna_vocab" target="_blank">a controlled vocabulary</a> to indicate what type of non-protein-coding feature is being represented.<br>

        <dl><br>
            <dd>Format: /ncRNA_class="&lt;ncRNA_class_TYPE&gt;"<br>
            <dd>Example: /ncRNA_class="miRNA"<br> <br>&lt;ncRNA_class_TYPE&gt; should be selected from the following list;

                <table border="0">
                    <tr>
                        <td>"antisense_RNA"  </td>
                        <td>"autocatalytically_spliced_intron"  </td>
                        <td>"telomerase_RNA"</td>
                    </tr>
                    <tr>
                        <td>"hammerhead_ribozyme"  </td>
                        <td>"RNase_P_RNA",</td>
                        <td>"RNase_MRP_RNA"</td>
                    </tr>
                    <tr>
                        <td>"guide_RNA"</td>
                        <td>"rasiRNA"</td>
                        <td>"scRNA"</td>
                    </tr>
                    <tr>
                        <td>"siRNA"</td>
                        <td>"miRNA"</td>
                        <td>"piRNA"</td>
                    </tr>
                    <tr>
                        <td>"snoRNA"</td>
                        <td>"snRNA"</td>
                        <td>"SRP_RNA"</td>
                    </tr>
                    <tr>
                        <td>"vault_RNA"</td>
                        <td>"Y_RNA",</td>
                        <td> "other"</td>
                    </tr>
                </table>

                <p></p>
            </dd>
</dd>
</dl><br></li>
<li>New tmRNA feature<br>To support a class of RNA transcripts that have dual tRNA-like and mRNA-like behaviors, a new tmRNA feature will belegal. See <a href="http://psyche.uthct.edu/dbs/tmRDB/tmRDB.html" target="_blank">tmRDB</a> and <a href="http://www.indiana.edu/~tmrna/" target="_blank">tmRNA Website</a> that provide some backgroundinformation about the tmRNAs.<dd>
<li>New /tag_peptide qualifier<br>To indicate the nucleotide region encoding the proteolysis tag<br>peptide of tmRNA, a new qualifier, /tag_peptide, will be used for the<br>tmRNA feature.<br>

    <dl><br>
        <dd>Format: /tag_peptide=&lt;base_range&gt;<br>
        <dd>Example: /tag_peptide=90..122<br></dd>
        </dd>
    </dl><br>
</li>
</dd>
</li>
</dd>
</li>
</ul>
</dd>
